




<!DOCTYPE html>






    










<html lang="en"
      
        
      
      
        

        

        
        
            
        
        
        
        
        

      class="pb-page"
         data-request-id="9a020aaf9539530c-SJC"  

>

    
        <head data-pb-dropzone="head">
            








    
        
    
        
            
        
    
        
            
        
    
        
            
        
    
        
            
        
    
        
            
        
    
        
    
        
    
        
            
                
                
<title>AggNet: Advancing protein aggregation analysis through deep learning and protein language model - He - 2025 - Protein Science - Wiley Online Library</title>





                
            
        
    




    <script type="text/javascript" nonce="9a020aaf9539530c-SJC">
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('consent', 'default', {
        'functionality_storage': 'granted',
        'security_storage': 'granted',
        'analytics_storage': 'denied',
        'personalization_storage': 'denied',
        'ad_storage': 'denied',
        'ad_user_data': 'denied',
        'ad_personalization': 'denied',
        'wait_for_update': 500
    });
</script>

    





    <!-- OsanoScriptEnabled: true -->
    
        <link rel="preload" href="https://cmp.osano.com/AzZdRbSORDOOzF9W/67a788d8-9344-4c9b-a3c2-89b5073ee231/osano.js" as="script">
        <script src="https://cmp.osano.com/AzZdRbSORDOOzF9W/67a788d8-9344-4c9b-a3c2-89b5073ee231/osano.js" nonce="9a020aaf9539530c-SJC"></script>
        <script nonce="9a020aaf9539530c-SJC">
            window.Osano.cm.addEventListener("osano-cm-script-blocked", function (item) {
                console.log("Script blocked: " + item);
            });
            window.Osano.cm.addEventListener("osano-cm-cookie-blocked", function (item) {
                console.log("Cookie blocked: " + item);
            });
            window.Osano.cm.addEventListener("osano-cm-iframe-blocked", function (item) {
                console.log("iFrame blocked: " + item);
            });
        </script>
    


    














<script type="text/javascript" nonce="9a020aaf9539530c-SJC">
    var isProgrammaticAdsEnabled = false;
</script>




    <script nonce="9a020aaf9539530c-SJC">
        function isBrowserChrome() {
            return /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
        }
    </script>








    
        
    
        
            
        
    
        
            
                
                



    
        
    

                
            
        
    
        
            
        
    
        
            
        
    
        
            
        
    
        
    
        
    
        
            
        
    










    








    

    
    

    
    
    
    
        
            
            
            
                
                    <link rel="stylesheet" type="text/css" href="/wro/62b81afc18538b179f72ac9fd0c18fbc83a2b493~article-metrics-phase2.css">
                
            
        
    

<link rel="stylesheet" type="text/css" href="/wro/62b81afc18538b179f72ac9fd0c18fbc83a2b493~product.css">
    <link rel="stylesheet" href="/products/acropolis/pericles/releasedAssets/css/build-06a701d43c1ace699b3c.css"/><link rel="stylesheet" href="/products/acropolis/pericles/releasedAssets/css/print-06a701d43c1ace699b3c.css" media="print"/>




















    
        
    
        
            
                
                
                
                    
                    <script id="adobeDigitalData" registrationCompleted="false">window.adobeDataLayer = window.adobeDataLayer || [];
window.adobeDataLayer.push({"site":{"environment":"LIVE","hub":"pericles","locale":"en-US"},"user":{"login-status":"guest","admin-type":"non-admin","identities":[{"type":"RegionUser","uuid":"3c5060a1-6620-438f-862b-902a4d4cd4ac"},{"type":"ReferrerUser","uuid":"0951017f-fab2-4eba-9609-e6c1e41dad59"},{"type":"ReferrerUser","uuid":"5a2c1d70-1255-4a45-95f3-a3aa3a2771fb"},{"type":"InstitutionUser","uuid":"33de3730-74b5-477e-8afd-e85c471abafd","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#BYUHIP"}]},{"type":"SmartGroupUser","uuid":"8c72a733-117a-4bc9-a682-13475f13e6a3"},{"type":"InstitutionUser","uuid":"af4e79db-8de3-4d03-b99b-f63feb42e2ed","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"BRIGHA029S"}]},{"type":"ReferrerUser","uuid":"6401b279-b825-4f41-a389-04e29feb967f"},{"type":"InstitutionUser","uuid":"d2969670-fbe8-4d94-9065-3f519b375ff8","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#OATITLES"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"BasicGroup","uuid":"2f6750d5-f13f-4356-939b-048ac1423e64"},{"type":"InstitutionUser","uuid":"cf5635ed-5938-4983-af2e-14d04cbfb4cf","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#CIND"}]},{"type":"SmartGroupUser","uuid":"b238ab85-6bb4-49e4-84ca-a0d9f3ee5a35"},{"type":"SmartGroupUser","uuid":"89490c0a-0b70-4215-8ec1-6aa771253899"},{"type":"SmartGroupUser","uuid":"da88d808-3f7b-4bde-9210-10862b6d121b"},{"type":"InstitutionUser","uuid":"793ac739-ad4b-4ec4-b09e-1d797fd55c0b","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"DBTEMPE766"}]},{"type":"InstitutionUser","uuid":"64c30cd7-876d-4727-adec-5dfeabb41f48","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000118552"}]},{"type":"RegionUser","uuid":"afe7769c-0043-49da-9fcb-d7bbfa051bb4"},{"type":"InstitutionUser","uuid":"9bb5fb03-9360-4ac9-9c08-c1887d11759d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000207351"}]},{"type":"InstitutionUser","uuid":"f995f0a3-5b39-4226-8891-de37a12f5152","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL0000897"}]},{"type":"InstitutionUser","uuid":"472b002b-5cd2-4d45-a5cc-b2bb22863ecd","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#GWLA"}]},{"type":"SmartGroupUser","uuid":"b7acda8c-ce18-451b-a68d-e14b72f9b5b8"},{"type":"SmartGroupUser","uuid":"b2dbfe27-ecae-4bb6-bb0d-d75e5caa3762"}]},"content":{"item":{"title":"AggNet: Advancing protein aggregation analysis through deep learning and protein language model","doi":"10.1002/pro.70031","access":"full","type":"article","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biochemistry","topicUri":"ls36"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biochemistry (Chemical Biology)","topicUri":"chb0"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Structural Biology","topicUri":"ls38"},{"taxonomyUri":"global-subject-codes","topicLabel":"Proteins \u0026 Macromolecules","topicUri":"ls39"},{"taxonomyUri":"global-subject-codes","topicLabel":"Protein Science","topicUri":"chb4"},{"taxonomyUri":"global-subject-codes","topicLabel":"Cell \u0026 Molecular Biology","topicUri":"ls30"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Biology (Life Sciences)","topicUri":"ls31"},{"taxonomyUri":"fundref-funders","topicLabel":"King Abdullah University of Science and Technology","topicUri":"501100004052"}],"published-date":"2025-01-22","access-denial":"no","format-available":["Abstract","Cited by","First page image","Full text","PDF","References","Supplementary Material","AM pdf"],"format-viewed":"full text","auth-type":"license"},"group":{"doi":"10.1002/pro.v34.2","access":"full","type":"issue","volume":"34","issue":"2","published-date":"2025-02-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biochemistry","topicUri":"ls36"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biochemistry (Chemical Biology)","topicUri":"chb0"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Structural Biology","topicUri":"ls38"},{"taxonomyUri":"global-subject-codes","topicLabel":"Proteins \u0026 Macromolecules","topicUri":"ls39"},{"taxonomyUri":"global-subject-codes","topicLabel":"Protein Science","topicUri":"chb4"},{"taxonomyUri":"global-subject-codes","topicLabel":"Cell \u0026 Molecular Biology","topicUri":"ls30"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Biology (Life Sciences)","topicUri":"ls31"}]},"series":{"title":"Protein Science","doi":"10.1002/(ISSN)1469-896X","access":"full","type":"journal","code":"1469896x","eissn":"1469-896X","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biochemistry","topicUri":"ls36"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biochemistry (Chemical Biology)","topicUri":"chb0"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Structural Biology","topicUri":"ls38"},{"taxonomyUri":"global-subject-codes","topicLabel":"Proteins \u0026 Macromolecules","topicUri":"ls39"},{"taxonomyUri":"global-subject-codes","topicLabel":"Protein Science","topicUri":"chb4"},{"taxonomyUri":"global-subject-codes","topicLabel":"Cell \u0026 Molecular Biology","topicUri":"ls30"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Biology (Life Sciences)","topicUri":"ls31"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.chemistry_000033","topicUri":"wly.chemistry_000033"},{"taxonomyUri":"gpt-publisherid","topicLabel":"21618997260","topicUri":"21618997260"},{"taxonomyUri":"gpt-zonename","topicLabel":"protein-sci_PRO","topicUri":"protein-sci_pro"}]}},"page":{"brand":"wol","type":"article","primary-section":"journal","secondary-section":"article","tertiary-section":"full"}});</script>
                    
                
            
        
    
        
            
                
                
                
                
                
            
        
    
        
            
                
                
                
                    
                    <script type="text/javascript">var $DoubleClickSite = "wly.chemistry_000033";
var $DoubleClickZone = "protein-sci_pro";
var $WileyGooglePublisherID = 21618997260;
var $GooglePublisherCategory = "/21618997260/wly.chemistry_000033/protein-sci_pro";</script>
                    
                
            
        
    
        
            
                
                
                
                    
                    
                    
                
            
        
    
        
            
                
                
                
                    
                    <meta name="robots" content="noarchive"/><meta name="description" content="Protein aggregation is critical to various biological and pathological processes. Besides, it is also an important property in biotherapeutic development. However, experimental methods to profile pro..."/>

















    






    <meta property="og:title" content="AggNet: Advancing protein aggregation analysis through deep learning and protein language model"/>


    <meta property="og:type" content="Article"/>


    
        <meta property="og:url" content="https://onlinelibrary.wiley.com/doi/10.1002/pro.70031"/>
    
    



    
    
        
        

        
            <meta property="og:image" content="https://onlinelibrary.wiley.com/cms/asset/cc1752c7-12c8-4a22-a94b-2100c050a46a/pro.v34.2.cover.jpg?trick=1763412012013"/>
            <meta property="twitter:image" content="https://onlinelibrary.wiley.com/cms/asset/cc1752c7-12c8-4a22-a94b-2100c050a46a/pro.v34.2.cover.jpg?trick=1763412012013"/>
        
    


    <meta property="og:site_name" content="Wiley Online Library"/>


        

        <meta property="og:description" content="Protein aggregation is critical to various biological and pathological processes. Besides, it is also an important property in biotherapeutic development. However, experimental methods to profile pro..."/>


<meta name="viewport" content="width=device-width,initial-scale=1"><meta name="apple-itunes-app" content="app-id=782998971"/>
<meta name="publication_doi" content="10.1002/pro.70031"/>





















<meta name="submission-systems-domains" content="/pb-assets/utm_params_config/submission-systems-domains-1667213560157.txt" />

<style>
    .d-flex {
        display: flex;
    }
</style>

















    
    
        <script src="https://assets.adobedtm.com/59640f9fa510/591d3f748997/launch-1ff1f800248b.min.js" async></script>
    



    

    <meta name="citation_doi" content="10.1002/pro.70031"/>

    <meta name="dc.identifier" content="10.1002/pro.70031"/>




<meta name="citation_journal_title" content="Protein Science"><meta name="citation_title" content="AggNet: Advancing protein aggregation analysis through deep learning and protein language model"><meta name="citation_keywords" content="amyloid"><meta name="citation_keywords" content="APR"><meta name="citation_keywords" content="computational biology"><meta name="citation_keywords" content="machine learning"><meta name="citation_keywords" content="protein aggregation"><meta name="citation_keywords" content="protein engineering"><meta name="citation_volume" content="34"><meta name="citation_issue" content="2"><meta name="citation_firstpage" content="e70031"><meta name="citation_publication_date" content="2025/02/01"><meta name="citation_online_date" content="2025/01/22"><meta name="citation_issn" content="1469-896X"><meta name="citation_language" content="en"><meta name="citation_author" content="Wenjia He"><meta name="citation_author_institution" content="Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence for Smart Health (KCSH) King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence on Generative AI King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author" content="Xiaopeng Xu"><meta name="citation_author_institution" content="Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence for Smart Health (KCSH) King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence on Generative AI King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author" content="Haoyang Li"><meta name="citation_author_institution" content="Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence for Smart Health (KCSH) King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence on Generative AI King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author" content="Juexiao Zhou"><meta name="citation_author_institution" content="Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence for Smart Health (KCSH) King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence on Generative AI King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author" content="Xin Gao"><meta name="citation_author_institution" content="Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence for Smart Health (KCSH) King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_author_institution" content="Center of Excellence on Generative AI King Abdullah University of Science and Technology (KAUST) Thuwal Saudi Arabia"><meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.70031"><meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.70031"><meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1002/pro.70031"><meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd"><meta name="article_references" content=" He W,  Xu X,  Li H,  Zhou J,  Gao X. AggNet: Advancing protein aggregation analysis through deep learning and protein language model. Protein Science.  2025; 34(2):e70031. https://doi.org/10.1002/pro.70031"><meta name="Description" content="Abstract Protein aggregation is critical to various biological and pathological processes. Besides, it is also an important property in biotherapeutic development. However, experimental methods to ...">


                    
                
            
        
    
        
    
        
    
        
            
                
                
                
                
                
            
        
    




<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM"/>
<meta name="google-site-verification" content="hdWo7Ws1fVArpU876Aal2KaoZ7vokr9ZNLcISOBm9aI" />
<meta name="google-site-verification" content="f_2wKVmVSKiYSTFpns9vtqI0sVux_tz7Vwr8zOWwXME" />

<script async src="/pb-assets/catchpoint-1549680351760.js" nonce="9a020aaf9539530c-SJC"></script>
<meta name="referrer" content="origin-when-cross-origin"/>
<link rel="shortcut icon" href="/favicon.ico">

<script type="text/javascript" nonce="9a020aaf9539530c-SJC">
    var utmParamsEnabled = true;
</script>

<script type="text/javascript" nonce="9a020aaf9539530c-SJC">
    var _hmt = _hmt || [];
    (function () {
        var hm = document.createElement("script");
        hm.src = "https://hm.baidu.com/hm.js?953dddc9c7bea72022e3bd3ba7782e7b";
		hm.async = true;
        var s = document.getElementsByTagName("script")[0];
        s.parentNode.insertBefore(hm, s);
    })();
</script>

<script nonce="9a020aaf9539530c-SJC">
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
	
	// PERICLES-19717
    window.dvtag = window.dvtag || {};
    dvtag.cmd = dvtag.cmd || [];
	

    function loadScriptDynamically(scriptUrl, callback) {
        var gads = document.createElement('script'),
            node = document.getElementsByTagName('script') [0];
        gads.async = true;
        gads.src = scriptUrl;
        if (callback) {
            gads.onload = callback;
        }
        node.parentNode.insertBefore(gads, node);
    }

    loadScriptDynamically('https://securepubads.g.doubleclick.net/tag/js/gpt.js', function () {
        loadScriptDynamically('https://pub.doubleverify.com/dvtag/39215024/DV1871689/pub.js');
    });
</script>

<!-- DoubleVerify Unity Tag queue (PERICLES-19717) - START -->
<script nonce="9a020aaf9539530c-SJC">
    window.onDvtagReady = function (callback, timeout = 750) {
        const opt = { callback, timeout, timestamp: new Date().getTime() }
        dvtag.cmd.push(function () { dvtag.queueAdRequest(opt) })
        setTimeout(function () {
            const cb = opt.callback
            opt.callback = null
            if (cb) cb()
        }, timeout)
    }
</script>
<!-- DoubleVerify Unity Tag queue - END --> 

<script nonce="9a020aaf9539530c-SJC">
    googletag.cmd.push(function () {
        var sizes = {
            leaderboard: googletag.sizeMapping()
                .addSize([320, 100], [[300, 50], [320, 50]])
                .addSize([740, 100], [728, 90])
                .addSize([0, 0], 'fluid')
                .build(),
            mobileleaderboard: googletag.sizeMapping()
                .addSize([300, 100], [300, 50])
                .addSize([320, 100], [320, 50])
                .build(),
            mobilefooter: googletag.sizeMapping()
                .addSize([0, 0], [300, 50])
                .addSize([0, 0], [320, 50])
                .build(),
            rail: googletag.sizeMapping()
                .addSize([0, 0], [300, 250])
                .build(),
            interstitial: googletag.sizeMapping()
                .addSize([0, 0], [640, 480])
                .build()
        };

        gptPage = window.digitalData && digitalData.page && digitalData.page.page ? digitalData.page.page : "blank";
        gptSubPage = window.digitalData && digitalData.page && digitalData.page.subPage ? digitalData.page.subPage : "blank";

        function gptClean(gptText) {
            if (typeof (gptText) == "string") {
                gptText = gptText.replace(/\&/g, '%26');
                return gptText.replace(/\^|\*|#|\$|\"|\'|\=|\,|\!|\+|\~|\;|\(|\)|\<|\>|\[|\]|\?|\%|\:|\@|\;|\&|\||\{|\}|\\|/g, '');
            }
            return "blank";
        }

        googletag.pubads()
            .setTargeting('property', 'value-of-property')    // Change and duplicate this line for as many additional keywords to send
            .setTargeting('doi', gptClean('10.1002/pro.70031')) // Publication DOI
            .setTargeting('pbPage', gptClean(gptPage))
            .setTargeting('pbSubPage', gptClean(gptSubPage))
            .setTargeting('urlpath', gptClean(document.location.pathname));

        googletag.defineSlot($GooglePublisherCategory, 'fluid', 'advert-leaderboard')
            .defineSizeMapping(sizes.leaderboard)
            .addService(googletag.pubads())
            .setCollapseEmptyDiv(true);

        if(window.innerWidth >= 992) { // Remove MPU ad units from mobile view (LIT-621710)
            googletag.defineSlot($GooglePublisherCategory, 'fluid', 'advert-rail-1')
                .defineSizeMapping(sizes.rail)
                .setTargeting('Position', 'advert-rail-1')
                .addService(googletag.pubads())
                .setCollapseEmptyDiv(true);

            googletag.defineSlot($GooglePublisherCategory, 'fluid', 'advert-rail-2')
                .defineSizeMapping(sizes.rail)
                .setTargeting('Position', 'advert-rail-2')
                .addService(googletag.pubads())
                .setCollapseEmptyDiv(true);
        }

		onDvtagReady();	// DoubleVerify (PERICLES-19717)

        if(isProgrammaticAdsEnabled) {
            // PERICLES-17135
            googletag.pubads().disableInitialLoad();
        }
		
        googletag.enableServices();
    });

    if (isProgrammaticAdsEnabled) {
        googletag.cmd.push(function () {
            apstag.fetchBids({timeout: 2000}, function (bids) {
                apstag.setDisplayBids();
                onDvtagReady(function () {
                    googletag.pubads().refresh();
                })
            });
        });
    }
</script>

<!--PERICLES-19058-->

<script src="https://servedbydoceree.doceree.com/resources/p/doc_ad/84/281/dc.js" data-siteId="281" data-platId="1" data-pubId="84" nonce="9a020aaf9539530c-SJC"></script>

<script nonce="9a020aaf9539530c-SJC">
    /* User Details */
    if (Osano.cm.getConsent().MARKETING) {
        var $userConsent = "1";
    } else {
        var $userConsent = "0";
    }
    try{
        docereeAds.add("userDetails", {
            "userconsent": $userConsent
        })
    }
    catch(err){
        console.log(err)
    }
</script>

<link rel="stylesheet" type="text/css" href="/pb-assets/hub-assets/wol-common/wol-common-1762168501707.css">

<link rel="stylesheet" type="text/css" href="/pb-assets/hub-assets/pericles/hub-1760714733317.css">


<link rel="stylesheet" type="text/css" href="/pb-assets/css/iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/css/readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/css/dr-overrides-1753295165047.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1733168052867.css" -->

<style>
    /* PERICLES-15842: 22.1 Feature: Save and Publish Button */
    .pb-changes-dialog.pb-save-dialog .modal-footer .btn.dropdown-toggle {
        display: none;
    }
    .pb-changes-dialog.pb-save-dialog .modal-footer .btn.btn-primary{
        border-radius: 4px;
        margin-right: 5px
    }
</style>

<!-- For the temporary solution in LIT-648162 -->
<style>
    .pb-edit-view .owl-carousel .owl-stage-outer {
        height: auto !important;
    }

    .pb-edit-view .owl-carousel .owl-stage {
        transform: unset !important;
        width: unset !important;
    }

    .pb-edit-view .owl-carousel .owl-item {
        visibility: visible;
        width: 100% !important;
        height: unset !important;
    }

    .pb-edit-view .owl-carousel .slide-item > .pb-dropzone:not(.ui-sortable) {
        display: none;
    }

    .pb-edit-view .owl-carousel .owl-nav,
    .pb-edit-view .owl-carousel .owl-dots,
    .pb-edit-view .owl-carousel .owl-controls {
        display: none !important;
    }
</style>





    <!-- Get whether TNSI is in live or test mode -->
    
    <!-- Get the configured TNSI Javascript version number -->
    
    <!-- Get the configured TNSI MerchantID -->
    
    <!-- Add "TEST" to beginning of merchant ID when running in test mode -->
    

    
<!-- loading Javascript for using the persistence Service -->
    <script src="/wro/62b81afc18538b179f72ac9fd0c18fbc83a2b493~thiss-ds.js" nonce="9a020aaf9539530c-SJC"></script>

    <script type="text/javascript" nonce="9a020aaf9539530c-SJC">

        function getParameter(parameterName) {
            let result = null,
                tmp = [];
            location.search
                .substr(1)
                .split("&")
                .forEach(function (item) {
                    tmp = item.split("=");
                    if (tmp[0] === parameterName) result = tmp[1];
                });
            return result;
        }

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            queryParamsDelimiter: encodeURIComponent('?'),
            _persistenceService: null,
            _getPersistenceService: function () {
                if (!this._persistenceService) {
                    this._persistenceService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');
                }
                return this._persistenceService;
            },

            getWAYFLessUrl: function (entityId) {
                let redirectURL = getParameter('redirectUri');
                if(redirectURL == null){
                    redirectURL = encodeURIComponent(this.returnURL);
                }
                if(redirectURL.indexOf('saml_referrer') < 0){
                    redirectURL += encodeURIComponent(((redirectURL.indexOf(this.queryParamsDelimiter) !== -1) ? '&' : '?') + 'saml_referrer');
                }
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + redirectURL;
            },
            retrieveEntity: function () {
                var self = this;
                var ps = this._getPersistenceService();

                return new Promise(function (resolve, reject) {
                    ps.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var ps = this._getPersistenceService();

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    ps.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = this._getPersistenceService();

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId, entityTitle: entityTitle, eventGroupKey: '9a5fe4ea-ea34-4dd0-bc86-3513f306ec68'}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (let i in allSsoInstitutions) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>
    <script nonce="9a020aaf9539530c-SJC">
            (function() {

                if (isBrowserChrome()) {
                    const originTrialToken = 'AkNzJOW9KcqV6Qlbg96xP2jqRsvzla/CKZY+FSKbiPtVTSST6mhuoiD7RFVzdtCrn2wI4LU0tmr8jf521BokXgkAAACJeyJvcmlnaW4iOiJodHRwczovL29ubGluZWxpYnJhcnkud2lsZXkuY29tOjQ0MyIsImZlYXR1cmUiOiJEaXNhYmxlVGhpcmRQYXJ0eVN0b3JhZ2VQYXJ0aXRpb25pbmczIiwiZXhwaXJ5IjoxNzU3OTgwODAwLCJpc1N1YmRvbWFpbiI6dHJ1ZX0=';
                    const originTrialMetaTag = document.createElement('meta');
                    originTrialMetaTag.httpEquiv = 'origin-trial';
                    originTrialMetaTag.content = originTrialToken;
                    document.head.append(originTrialMetaTag);
                }
            })();
        </script>
    <script type="text/javascript" nonce="9a020aaf9539530c-SJC">
        (function() {
            function riskifiedBeaconLoad() {
                var store_domain = 'https://onlinelibrary.wiley.com';
                var session_id = '6FCD468F9DAD5F86F17E44CD41829B01';
                var url = ('https:' == document.location.protocol ? 'https://' : 'http://')
                    + "beacon.riskified.com?shop=" + store_domain + "&sid=" + session_id;
                var s = document.createElement('script');
                s.type = 'text/javascript';
                s.async = true;
                s.src = url;
                var x = document.getElementsByTagName('script')[0];
                x.parentNode.insertBefore(s, x);
            }
            if (window.attachEvent)
                window.attachEvent('onload', riskifiedBeaconLoad)
            else
                window.addEventListener('load', riskifiedBeaconLoad, false);
        })();
    </script>
<meta name="mathjax-enabled" content="true">
    <meta name="mathjax-version" content="3">
    <meta name="mathjax-lazy-load" content="true">

<meta name="pbContext" content=";journal:journal:1469896x;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;article:article:doi\:10.1002/pro.70031;wgroup:string:Publication Websites;issue:issue:doi\:10.1002/pro.v34.2;pageGroup:string:Publication Pages;subPage:string:Full Text;requestedJournal:journal:1469896x">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

        </head>
    

    



    
        
        
        
            <body
                    class="pb-ui website-pericles">
            






    
    
        <nav class="skip-links">
    <ul class="rlist">
        <li><button data-id="#article__content" tabindex="1">Skip to Article Content</button></li>
                <li><button data-id="#pane-pcw-detailscon" tabindex="1">Skip to Article Information</button></li>
            </ul>
</nav>








        <div id="pb-page-content" data-ng-non-bindable>
                
            <div data-pb-dropzone="main" data-pb-dropzone-name="Main">
                
                    
                        









    
    
        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader">
        



        
        



<header class="page-header">
    











<div  data-pb-dropzone="main" class="">
    
        
            



        
        

        
            









    
    
        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        



    

    

    
        
    


</div></div></div>

        </div>
    


        
            



        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









    
    
        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><div class="institution-box no-image"><div class="institution-info-wrapper"><a href="http://www.lib.byu.edu/" title="Harold B. Lee Library" target="_blank"><span class="institution__intro">Access By</span><span class="institution__name">Harold B. Lee Library</span></a></div></div></div>

        </div>
    





        
        <div class="pull-left">









    
    
        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="/" title=""><img id="mainLogo" alt="Wiley Online Library" src="/pb-assets/hub-assets/pericles/logo-header-1690978619437.png"/></a>

        </div>
    











    
    
        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="/" title=""><img id="mobileLogo" alt="Wiley Online Library" src="/pb-assets/hub-assets/pericles/mobilehublogo-1690978876347.png"/></a>

        </div>
    











    
    
        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><div class="institution-box no-image"><div class="institution-info-wrapper"><a href="http://www.lib.byu.edu/" title="Harold B. Lee Library" target="_blank"><span class="institution__name">Harold B. Lee Library</span></a></div></div></div>

        </div>
    

</div><div class="pull-right">









    
    
        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><button title="search" aria-label="Search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></button><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper"><div class="tab clearfix"><input type="search" placeholder="Search" value="" tabindex="13" aria-label="Search" class="fakeQSInput quickSearchFilter"/><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" tabindex="10" class="search-within"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" data-url="thisCategory" data-value="1469896x">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1" data-url="https://onlinelibrary.wiley.com">Anywhere</option></select><ul class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search" role="search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input type="search" aria-label="search text" role="combobox" aria-autocomplete="list" aria-controls="ui-id-1" aria-expanded="false" name="AllField" id="searchField0" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" tabindex="9" class="autocomplete actualQSInput quickSearchFilter"/><input type="hidden" name="SeriesKey" value="1469896x"/><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1469896x" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1469896x#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" class="tab__pane quick-search-pane__anywhere"><form action="https://onlinelibrary.wiley.com/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search" role="search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input type="search" aria-label="search text" role="combobox" aria-autocomplete="list" aria-controls="ui-id-1" aria-expanded="false" name="AllField" id="searchField1" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" tabindex="9" class="autocomplete actualQSInput quickSearchFilter"/><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li></ul></div></div></div></div>

        </div>
    











    
    
        <div data-widget-def="ux3-layout-widget" data-widget-id="5ef13ed1-f4ac-4b62-a426-07ec9576a586" class="cart-indicator__wrapper">
        



        
        <div class="pull-left">









    
    
        <div data-widget-def="eCommerceCartIndicatorWidget" data-widget-id="4a9f7c04-a273-4a25-91fb-8e7bd9970288" class="cart-indicator__wrapper">
        



        
        















<div class="eCommerceCartIndicatorWidget">
    
        
        
            
        
    
</div>

        </div>
    

</div><div class="pull-right">









    
    
        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    








    
        






    














    <div class="loginBar login-open">
        <button id="indivLogin" class="indivLogin" aria-expanded="false" uri="/" aria-label="Log in or Register">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </button>
    </div>
    <div class="navigation-login-dropdown-container hidden">
        <div class="navigation-login-dropdown">
            <ul>
                <li>
                    <i aria-hidden="true" class="icon-person"></i>
                    <a href="/action/showLogin?acdl-redirect=true&amp;uri=%2Fdoi%2F10.1002%2Fpro.70031" data-acdl-login--region="header" acdl-redirect="true" data-acdl-event="login-initiate" uri="/doi/10.1002/pro.70031" data-acdl-login--step="login initiate">
                        Individual login
                    </a>
                </li>
                
                    <li><i aria-hidden="true" class="icon-account_balance"></i>
                        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1002%2Fpro.70031">
                            
                            Institutional login
                        </a>
                    </li>
                
                
                
                <li>
                    <a href="/action/registration?redirectUri=%2Fdoi%2F10.1002%2Fpro.70031&amp;acdl-redirect=true" class="btn" redirectUri="/doi/10.1002/pro.70031" acdl-redirect="true" data-acdl-registration--step="registration initiated" data-acdl-event="registration-initiated" data-acdl-registration--cta-region="header">
                        REGISTER
                    </a>
                </li>
            </ul>
        </div>
    </div>


    
    



        </div>
    

</div>

        </div>
    

</div>
</div></div></div>

        
    
</div>

</header>

        </div>
    


                    
                        









    
    
        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet body-text">
        



        
        


<main id="main-content" tabindex="-1">
    <!--begin pagefulltext-->
    <div class="page-body pagefulltext">
        











<div  data-pb-dropzone="main" class="">
    
        
            



        
        <div>









    
    
        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        



    
        <div class="pb-las">
            <aside role="complementary">
   <div id="advert-leaderboard" class="advert advert-leaderboard">
      <script nonce="9a020aaf9539530c-SJC">
         console.time('gpt ready');
         googletag.cmd.push( function () {
            console.timeEnd('gpt ready');
            console.time('DV ready');
            onDvtagReady(function () {
               console.timeEnd('DV ready');
               console.time('advert-leaderboard ready');
               googletag.display( 'advert-leaderboard' );
               console.timeEnd('advert-leaderboard ready');
            });
         });
      </script>

      <!-- No JavaScript support is a call to an image -->
      <noscript>
         <a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1002%2Fpro.70031&pubId=88187701&placeholderId=1015&productId=101018" target="_blank">
            <img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
         </a>
      </noscript>
   </div>
</aside>
            
        </div>
    

    

    



        </div>
    





        
        <style type="text/css">li.menu-item[data-menu-label="Accepted Articles"] {
  display: none;
}</style><style type="text/css">li.menu-item[data-menu-label="Early View"] {
  display: none;
}
li.menu-item[data-menu-label="Recently Published"] {
  display: none;
}</style>
</div>

        
            









    
    
        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fpro.70031&amp;pbContext=%3Bjournal%3Ajournal%3A1469896x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fpro.70031%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fpro.v34.2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3A1469896x&amp;widgetKey=ux3-publicationContent-widget_5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9_3067_4931_148546_en&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fpro.70031&amp;pbContext=%3Bjournal%3Ajournal%3A1469896x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fpro.70031%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fpro.v34.2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3A1469896x&amp;widgetKey=ux3-publicationContent-widget_5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9_3067_4931_148546_en&amp;accordionHeadingWrapper=h2" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fpro.70031&amp;pbContext=%3Bjournal%3Ajournal%3A1469896x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fpro.70031%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fpro.v34.2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3A1469896x&amp;widgetKey=ux3-publicationContent-widget_5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9_3067_4931_148546_en&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata&amp;showDownloadCountPopup=true" data-mathjax="" data-getFTR-references-enabled="data-getFTR-references-enabled" data-getFTR-retractions-enabled="data-getFTR-retractions-enabled" data-getFTR-reference-url="/action/CitationEntitlement" data-getFTR-links-count='10' data-getFTR-request-source="FULL_TEXT" data-getFTR-doi="10.1002/pro.70031"><div class="row article-row"><div id="article__content" tabindex="-1" class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title"><div id="journal-banner-text" class="journal-banner-text"><a href="/journal/1469896x" title="Protein Science homepage">Protein Science</a></div><a href="/toc/1469896x/2025/34/2" class="volume-issue">Volume 34, Issue 2</a><span class="citation__eLocator"> e70031</span><a href="/journal/1469896x" title="Protein Science homepage" class="citation--logo"><img id="journal-banner-image" test="test-value" alt="Protein Science" src="/pb-assets/journal-banners/1469896x-1640185618860.jpg" class="journal-banner-image"></a><script nonce="9a020aaf9539530c-SJC">var image = new Image();

image.onerror = function() {
  // if the image does not exist
  //document.getElementById("journal-banner-text").style.marginBottom = "15px";
  document.getElementById("journal-banner-text").style.fontSize = "22px";
  document.getElementById("journal-banner-image").style.display = "none";
}

image.src = "/pb-assets/journal-banners/1469896x-1640185618860.jpg";</script></div>
   <div class="doi-access-container clearfix"><span class="primary-heading"><span class="primary-heading">TOOLS FOR PROTEIN SCIENCE</span></span><div class="doi-access-wrapper"><div class="full-access access-type"><i aria-hidden="true" class="icon-icon-lock_open"></i><div class="doi-access">Full Access</div></div></div></div>
   <h1 class="citation__title">AggNet: Advancing protein aggregation analysis through deep learning and protein language model</h1>
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-versions"></div>
   <div class="loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="comma__list"><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/He/Wenjia" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl" role="button"><span>Wenjia He</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" role="region" id="am1">
               <p class="author-name">Wenjia He</p>
               <ul class="rlist sm-account">
                  <li class="sm-account__item"><a target="_blank" class="sm-account__link" href="https://orcid.org/0000-0001-8161-4642"><i aria-hidden="true" class="icon-orcid"></i><span>orcid.org/0000-0001-8161-4642</span></a></li>
               </ul>
               <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Contribution: Conceptualization, Methodology, Writing - original draft, Visualization, Investigation, Validation, Data curation</p><a class="moreInfoLink" href="/authored-by/He/Wenjia">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Xu/Xiaopeng" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl" role="button"><span>Xiaopeng Xu</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" role="region" id="am2">
               <p class="author-name">Xiaopeng Xu</p>
               <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Contribution: Validation, Writing - review &amp; editing, Visualization, Supervision</p><a class="moreInfoLink" href="/authored-by/Xu/Xiaopeng">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Li/Haoyang" class="author-name accordion-tabbed__control" data-id="am3" data-db-target-for="am3" aria-controls="am3" aria-haspopup="true" id="am3_Ctrl" role="button"><span>Haoyang Li</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="am3" aria-labelledby="am3_Ctrl" role="region" id="am3">
               <p class="author-name">Haoyang Li</p>
               <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Contribution: Writing - review &amp; editing, Visualization, Validation</p><a class="moreInfoLink" href="/authored-by/Li/Haoyang">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Zhou/Juexiao" class="author-name accordion-tabbed__control" data-id="am4" data-db-target-for="am4" aria-controls="am4" aria-haspopup="true" id="am4_Ctrl" role="button"><span>Juexiao Zhou</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="am4" aria-labelledby="am4_Ctrl" role="region" id="am4">
               <p class="author-name">Juexiao Zhou</p>
               <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Contribution: Writing - review &amp; editing, Visualization, Validation</p><a class="moreInfoLink" href="/authored-by/Zhou/Juexiao">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Gao/Xin" class="author-name accordion-tabbed__control" data-id="am5" data-db-target-for="am5" aria-controls="am5" aria-haspopup="true" id="am5_Ctrl" role="button"><span>Xin Gao<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="am5" aria-labelledby="am5_Ctrl" role="region" id="am5">
               <p class="author-type mb-1">Corresponding Author</p>
               <p class="author-name">Xin Gao</p>
               <ul class="rlist sm-account">
                  <li class="sm-account__item"><a class="sm-account__link" title="Link to email address" href="/cdn-cgi/l/email-protection#b3cbdadd9dd4d2dcf3d8d2c6c0c79dd6d7c69dc0d2"><i aria-hidden="true" class="icon-ic_mail_solid"></i><span><span class="__cf_email__" data-cfemail="c1b9a8afefa6a0ae81aaa0b4b2b5efa4a5b4efb2a0">[email&#160;protected]</span></span></a></li>
               </ul>
               <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
               <p><b>Correspondence</b></p>
               <p>Xin Gao, Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal 23955-6900, Saudi Arabia.</p>
               <p>Email: <a class="corr-email" title="Link to email address" href="/cdn-cgi/l/email-protection#037b6a6d2d64626c4368627670772d6667762d7062"><span><span class="__cf_email__" data-cfemail="7e06171050191f113e151f0b0d0a501b1a0b500d1f">[email&#160;protected]</span></span></a></p>
               <p>Contribution: Resources, Project administration, Writing - review &amp; editing, Funding acquisition, Supervision</p><a class="moreInfoLink" href="/authored-by/Gao/Xin">Search for more papers by this author</a></div></span></div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs desktop-authors">
      <div id="sb-1" class="accordion">
         <div class="comma__list">
            <div class="accordion-tabbed"><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/He/Wenjia" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl" role="button"><span>Wenjia He</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" role="region" id="a1">
                     <p class="author-name">Wenjia He</p>
                     <ul class="rlist sm-account">
                        <li class="sm-account__item"><a target="_blank" class="sm-account__link" href="https://orcid.org/0000-0001-8161-4642"><i aria-hidden="true" class="icon-orcid"></i><span>orcid.org/0000-0001-8161-4642</span></a></li>
                     </ul>
                     <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Contribution: Conceptualization, Methodology, Writing - original draft, Visualization, Investigation, Validation, Data curation</p><a class="moreInfoLink" href="/authored-by/He/Wenjia">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Xu/Xiaopeng" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl" role="button"><span>Xiaopeng Xu</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" role="region" id="a2">
                     <p class="author-name">Xiaopeng Xu</p>
                     <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Contribution: Validation, Writing - review &amp; editing, Visualization, Supervision</p><a class="moreInfoLink" href="/authored-by/Xu/Xiaopeng">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Li/Haoyang" class="author-name accordion-tabbed__control" data-id="a3" data-db-target-for="a3" aria-controls="a3" aria-haspopup="true" id="a3_Ctrl" role="button"><span>Haoyang Li</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="a3" aria-labelledby="a3_Ctrl" role="region" id="a3">
                     <p class="author-name">Haoyang Li</p>
                     <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Contribution: Writing - review &amp; editing, Visualization, Validation</p><a class="moreInfoLink" href="/authored-by/Li/Haoyang">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Zhou/Juexiao" class="author-name accordion-tabbed__control" data-id="a4" data-db-target-for="a4" aria-controls="a4" aria-haspopup="true" id="a4_Ctrl" role="button"><span>Juexiao Zhou</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="a4" aria-labelledby="a4_Ctrl" role="region" id="a4">
                     <p class="author-name">Juexiao Zhou</p>
                     <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Contribution: Writing - review &amp; editing, Visualization, Validation</p><a class="moreInfoLink" href="/authored-by/Zhou/Juexiao">Search for more papers by this author</a></div></span><span class="accordion-tabbed__tab-mobile  accordion__closed"><a href="/authored-by/Gao/Xin" class="author-name accordion-tabbed__control" data-id="a5" data-db-target-for="a5" aria-controls="a5" aria-haspopup="true" id="a5_Ctrl" role="button"><span>Xin Gao<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><span class="comma-separator">,&nbsp;</span><div class="author-info accordion-tabbed__content" data-db-target-of="a5" aria-labelledby="a5_Ctrl" role="region" id="a5">
                     <p class="author-type mb-1">Corresponding Author</p>
                     <p class="author-name">Xin Gao</p>
                     <ul class="rlist sm-account">
                        <li class="sm-account__item"><a class="sm-account__link" title="Link to email address" href="/cdn-cgi/l/email-protection#4a322324642d2b250a212b3f393e642f2e3f64392b"><i aria-hidden="true" class="icon-ic_mail_solid"></i><span><span class="__cf_email__" data-cfemail="077f6e6929606668476c6672747329626372297466">[email&#160;protected]</span></span></a></li>
                     </ul>
                     <p>Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence for Smart Health (KCSH), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p>Center of Excellence on Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</p>
                     <p><b>Correspondence</b></p>
                     <p>Xin Gao, Computer Science Program, Computer, Electrical and Mathematical Sciences and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal 23955-6900, Saudi Arabia.</p>
                     <p>Email: <a class="corr-email" title="Link to email address" href="/cdn-cgi/l/email-protection#4a322324642d2b250a212b3f393e642f2e3f64392b"><span><span class="__cf_email__" data-cfemail="c4bcadaaeaa3a5ab84afa5b1b7b0eaa1a0b1eab7a5">[email&#160;protected]</span></span></a></p>
                     <p>Contribution: Resources, Project administration, Writing - review &amp; editing, Funding acquisition, Supervision</p><a class="moreInfoLink" href="/authored-by/Gao/Xin">Search for more papers by this author</a></div></span></div>
         </div>
      </div>
   </div>
   <div class="epub-sections"><div class="epub-section"><span class="epub-state">First published: </span><span class="epub-date">22 January 2025</span></div>
      <div class="epub-section"><a class="epub-doi" aria-label="Digital Object Identifier for AggNet: Advancing protein aggregation analysis through deep learning and protein language model link" href="https://doi.org/10.1002/pro.70031">https://doi.org/10.1002/pro.70031</a></div><div class="epub-section cited-by-count"><span>Citations: <a href="#citedby-section">2</a></span></div></div><span class="openUrl"><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1002/pro.70031&amp;type=tocOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3F" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span>
   <div class="epub-sections">
      <div class="article-header__references-container no-truncate" id="pro70031-note-0001" tabindex="0">
         <div class="header-note-content">
            <p><b>Review Editor:</b> Nir Ben-Tal</p>
         </div>
      </div>
   </div>
</div></div><div class="fixedCoolBar"><nav aria-label="Article navigation and tools" class="coolBar stickybar"><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="javascript:void(0)" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetClass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"></div></div></div><div class="coolBar__section coolBar--sections"><a id="sections_Ctrl" href="javascript:void(0)" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download PdfLink"><a href="/doi/epdf/10.1002/pro.70031" title="ePDF" class="coolBar__ctrl pdf-download"><img src="/specs/products/acropolis/pericles/releasedAssets/images/pdf-icon-169a2eb30e52100e76dfa5f4b66998e6.png" alt=""/>PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" aria-label="Tools" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul role="menu" class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1002%2Fpro.70031&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="/action/showCitFormats?doi=10.1002%2Fpro.70031" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="/personalize/addFavoritePublication?doi=10.1002%2Fpro.70031" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="/action/addCitationAlert?doi=10.1002%2Fpro.70031" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div class="relative"><a id="share-access__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-controls="share_Pop" role="button" aria-label="Share" class="share-access__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-controls="share_Pop" role="button" class="share__ctrl hidden"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share-access__ctrl" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-share-access="/action/authorizeSharing?doi=10.1002/pro.70031&amp;shareType=P2P&amp;format=PDF" class="share-access hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#shareAccess" class="share-access__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container share-access__modal"><div id="shareAccess" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close modal__cycleElement"><i aria-hidden="true" class="icon-close_thin"></i></button><p>Share full-text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://onlinelibrary.wiley.com/termsAndConditions" target="_blank" class="terms-and-conditions__cycleElement">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" type="checkbox" value="yes" required="" name="terms-and-conditions" data-ajax-link="/action/generateShareUrl?doi=10.1002/pro.70031&amp;shareType=P2P&amp;format=PDF" data-shareable-link=""/><span class="label-txt terms-and-conditions__cycleElement">I have read and accept the Wiley Online Library Terms and Conditions of Use</span></label></div></fieldset><hr class="separator"/><div class="shareable"><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis more-link__cycleElement">Learn more.</a></p><div class="shareable__box"><div class="shareable__text"><div class="shareable__field"><span id="shareable__text"></span><textarea tabindex="-1" class="shareable__text-area"></textarea></div></div><button type="submit" disabled="" class="btn shareable__btn shareable-btn__cycleElement">Copy URL</button></div><div class="error shareable__error hidden"></div></div></form></div></div></div></div>
<p>Share a link</p><span class="sr-only">Share on</span><ul class="share__list a2a a2a_kit a2a_default_style mb-0 a2a_kit_size_32 share__list--vertical"><li class="a2a_listitem_custom"><a role="link" title="Email" class="share__link a2a_button_email a2a_button_custom "><i aria-hidden="true" class="icon-Email"></i>Email</a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="share__link a2a_button_facebook a2a_button_custom "><i aria-hidden="true" class="icon-facebook"></i>Facebook</a></li><li class="a2a_listitem_custom"><a role="link" title="x" class="share__link a2a_button_x a2a_button_custom "><i aria-hidden="true" class="icon-x-platform"></i>x</a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="share__link a2a_button_linkedin a2a_button_custom "><i aria-hidden="true" class="icon-linkedin"></i>LinkedIn</a></li><li class="a2a_listitem_custom"><a role="link" title="Reddit" class="share__link a2a_button_reddit a2a_button_custom "><i aria-hidden="true" class="icon-reddit-alien"></i>Reddit</a></li><li class="a2a_listitem_custom"><a role="link" title="Wechat" class="share__link a2a_button_wechat a2a_button_custom "><i aria-hidden="true" class="icon-wechat"></i>Wechat</a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="share__link a2a_button_bluesky a2a_button_custom "><i aria-hidden="true" class="icon-bluesky"></i>Bluesky</a></li></ul></div></div>
</div></div></nav></div><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body "><article lang="en">
   
   <div class="abstract-group ">
      <section class="article-section article-section__abstract" lang="en" data-lang="en" lang-name="English" id="section-1-en">
         <h2 id="d1967776" class="article-section__header section__title main abstractlang_en main">Abstract</h2>
         <div class="article-section__content en main">
            
            <p>Protein aggregation is critical to various biological and pathological processes. Besides, it is also an important property in biotherapeutic development. However, experimental methods to profile protein aggregation are costly and labor-intensive, driving the need for more efficient computational alternatives. In this study, we introduce AggNet, a novel deep learning framework based on the protein language model ESM2 and AlphaFold2, which utilizes physicochemical, evolutionary, and structural information to discriminate amyloid and non-amyloid peptides and identify aggregation-prone regions (APRs) in diverse proteins. Benchmark comparisons show that AggNet outperforms existing methods and achieves state-of-the-art performance on protein aggregation prediction. Also, the predictive ability of AggNet is stable across proteins with different secondary structures. Feature analysis and visualizations prove that the model effectively captures peptides' physicochemical properties effectively, thereby offering enhanced interpretability. Further validation through a case study on MEDI1912 confirms AggNet's practical utility in analyzing protein aggregation and guiding mutation for aggregation mitigation. This study enhances computational tools for predicting protein aggregation and highlights the potential of AggNet in protein engineering. Finally, to improve the accessibility of AggNet, the source code can be accessed at: <a href="https://github.com/Hill-Wenka/AggNet" class="linkBehavior">https://github.com/Hill-Wenka/AggNet</a>.</p>
            </div>
      </section>
   </div>
   <div class="pb-dropzone" data-pb-dropzone="below-abstract-group"><!-- Empty dropzone --></div>
   
   <section class="article-section article-section__full">
      <section class="article-section__content" id="pro70031-sec-0001">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0001-title">1 INTRODUCTION</h2>
         
         <p>Protein aggregation is a complex biochemical process that often results in the formation of highly ordered amyloid fibrils or amorphous aggregates. This phenomenon is associated with various human diseases, poses challenges in therapeutic protein development, and inspires the creation of novel bio-inspired materials. Neurodegenerative diseases such as Alzheimer's disease (AD), Parkinson's disease (PD), and Huntington's disease (HD) are widely believed to be caused by the misfolding and aggregation of specific peptides or proteins (Koo et al.,&nbsp;<span><a href="#pro70031-bib-0016" id="#pro70031-bib-0016_R_d1967767e480" class="bibLink tab-link" data-tab="pane-pcw-references">1999</a></span>; Ross &amp; Poirier,&nbsp;<span><a href="#pro70031-bib-0033" id="#pro70031-bib-0033_R_d1967767e483" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>). Similarly, type II diabetes is linked to the aggregation of islet amyloid polypeptide (IAPP) (Kahn et al.,&nbsp;<span><a href="#pro70031-bib-0014" id="#pro70031-bib-0014_R_d1967767e486" class="bibLink tab-link" data-tab="pane-pcw-references">1999</a></span>). With the rapid advancement of protein-based pharmaceuticals like monoclonal antibodies and peptide drugs, protein aggregation has emerged as a significant hurdle in their production, storage, and purification (Lowe et al.,&nbsp;<span><a href="#pro70031-bib-0020" id="#pro70031-bib-0020_R_d1967767e489" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Perchiacca &amp; Tessier,&nbsp;<span><a href="#pro70031-bib-0027" id="#pro70031-bib-0027_R_d1967767e492" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>). High concentrations are often required for these biotherapeutics, but increased interactions between protein molecules can lead to severe aggregation, reducing therapeutic efficacy and potentially causing side effects (Lundahl et al.,&nbsp;<span><a href="#pro70031-bib-0021" id="#pro70031-bib-0021_R_d1967767e496" class="bibLink tab-link" data-tab="pane-pcw-references">2021</a></span>; Rahban et al.,&nbsp;<span><a href="#pro70031-bib-0030" id="#pro70031-bib-0030_R_d1967767e499" class="bibLink tab-link" data-tab="pane-pcw-references">2023</a></span>). Protein aggregation also inspires the development of self-assembling nanomaterials. Peptides with specifically designed sequences can self-assemble into nanofibers under certain pH conditions and temperatures (Wang et al.,&nbsp;<span><a href="#pro70031-bib-0039" id="#pro70031-bib-0039_R_d1967767e502" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>). These nanofibers serve as cell scaffolds to promote cell growth and tissue regeneration or function as drug delivery systems to control drug release (Matson &amp; Stupp,&nbsp;<span><a href="#pro70031-bib-0022" id="#pro70031-bib-0022_R_d1967767e505" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>). Such applications have spurred considerable interest in understanding and controlling protein aggregation.</p>
         
         <p>To effectively characterize the aggregation and stability of proteins, many biosensors are proposed to monitor and engineer proteins. For instance, Ebo et al. (<span><a href="#pro70031-bib-0006" id="#pro70031-bib-0006_R_d1967767e511" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>) developed a tripartite -lactamase enzyme assay (TPBLA) that correlates protein aggregation propensity with bacterial susceptibility of beta-lactam antibiotics. Similarly, Ren et al. (<span><a href="#pro70031-bib-0032" id="#pro70031-bib-0032_R_d1967767e514" class="bibLink tab-link" data-tab="pane-pcw-references">2021</a></span>) linked the stability of the protein of interest (POI) with the activity of the bacterial enzyme CysGA, which catalyzes the formation of endogenous fluorescent compounds. Nonetheless, these in vivo methods, along with traditional approaches like x-ray crystallography and nuclear magnetic resonance (NMR) spectroscopy (Housmans et al.,&nbsp;<span><a href="#pro70031-bib-0010" id="#pro70031-bib-0010_R_d1967767e517" class="bibLink tab-link" data-tab="pane-pcw-references">2023</a></span>), are notably resource-intensive and time-consuming. In contrast, in silico methods offer a rapid, cost-effective alternative for identifying aggregation-prone regions (APRs), which are essential for understanding aggregation mechanisms, developing mitigation strategies, and even suggesting new therapeutic strategies for cancer. A recent innovation by Janssen et al. (<span><a href="#pro70031-bib-0013" id="#pro70031-bib-0013_R_d1967767e520" class="bibLink tab-link" data-tab="pane-pcw-references">2023</a></span>) involved designing amyloid peptides (Pept-ins) that target the APRs of KRAS to induce its misfolding and aggregation, offering a novel approach to address previously undruggable targets through protein aggregation.</p>
         
         <p>Several computational tools have been proposed for protein aggregation prediction, including sequence-based and structure-based methods. Sequence-based models such as TANGO (Fernandez-Escamilla et al.,&nbsp;<span><a href="#pro70031-bib-0007" id="#pro70031-bib-0007_R_d1967767e526" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>), AGGRESCAN (Conchillo-Sol et al.,&nbsp;<span><a href="#pro70031-bib-0004" id="#pro70031-bib-0004_R_d1967767e529" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>), WALTZ (Maurer-Stroh et al.,&nbsp;<span><a href="#pro70031-bib-0023" id="#pro70031-bib-0023_R_d1967767e532" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>), ANuPP (Prabakaran et al.,&nbsp;<span><a href="#pro70031-bib-0029" id="#pro70031-bib-0029_R_d1967767e535" class="bibLink tab-link" data-tab="pane-pcw-references">2021</a></span>), and AggreProt (Planas-Iglesias et al.,&nbsp;<span><a href="#pro70031-bib-0028" id="#pro70031-bib-0028_R_d1967767e538" class="bibLink tab-link" data-tab="pane-pcw-references">2024</a></span>) estimate aggregation propensity solely based on the intrinsic amino acid properties and sequence patterns. For example, TANGO uses a thermodynamic approach to consider interactions between residuesincluding hydrophobic interactions, charge effects, polarity, and volume effectsto predict the tendency of specific sequence regions to form cross- structures. ANuPP, a more recent sequence-based method, clusters amyloidogenic hexapeptides based on physicochemical properties like hydrophobicity and charge, then builds individual logistic regression models for each cluster before ensemble prediction. However, due to the lack of spatial context, sequence-based models may produce false positives as they cannot determine whether a predicted APR is buried within the hydrophobic core of a protein. Conversely, structure-based methods like CORDAX (Louros, Orlando, et al.,&nbsp;<span><a href="#pro70031-bib-0019" id="#pro70031-bib-0019_R_d1967767e542" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>), SAP (Chennamsetty et al.,&nbsp;<span><a href="#pro70031-bib-0003" id="#pro70031-bib-0003_R_d1967767e545" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>), Aggrescan3D (Zambrano et al.,&nbsp;<span><a href="#pro70031-bib-0041" id="#pro70031-bib-0041_R_d1967767e548" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>), CamSol (Sormanni et al.,&nbsp;<span><a href="#pro70031-bib-0035" id="#pro70031-bib-0035_R_d1967767e551" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>), and AggScore (Sankar et al.,&nbsp;<span><a href="#pro70031-bib-0034" id="#pro70031-bib-0034_R_d1967767e554" class="bibLink tab-link" data-tab="pane-pcw-references">2018</a></span>) incorporate three-dimensional structural information, including atomic coordinates, surface hydrophobic patches, solvent accessibility, and the local microenvironment, to enhance predictive performance. Aggrescan3D, for instance, combines amino acid properties with solvent accessibility and spatial proximity, while CamSol adjusts intrinsic aggregation scores based on the solvent-accessible surface area of residues, facilitating more precise identification of aggregation hotspots, especially on protein surfaces. Despite their enhanced precision, these methods are limited by the availability of high-quality protein structural data, which is often scarce.</p>
         
         <p>Given the significant limitations of current models, necessitated by inadequate and unbalanced training data and the sparse availability of high-resolution protein structures, there is a pressing need for more advanced methodologies. Recent breakthroughs in computational biology, such as the development of large-scale protein language models, like ESM2 (Lin et al.,&nbsp;<span><a href="#pro70031-bib-0017" id="#pro70031-bib-0017_R_d1967767e560" class="bibLink tab-link" data-tab="pane-pcw-references">2023</a></span>), and cutting-edge protein structure prediction models represented by AlphaFold2 (Tunyasuvunakool et al.,&nbsp;<span><a href="#pro70031-bib-0037" id="#pro70031-bib-0037_R_d1967767e563" class="bibLink tab-link" data-tab="pane-pcw-references">2021</a></span>), have provided us potentially breakthrough solutions to solve the challenging aggregation prediction problem. These tools allow for the extraction of rich, contextual representations from amino acid sequences and have demonstrated remarkable accuracy in predicting three-dimensional protein structures directly from sequences. By integrating these advanced models, we propose AggNet, a comprehensive framework that leverages physiochemical, sequential, and structural information to predict protein aggregation, including amyloid peptide prediction and protein APR identification. For amyloid peptide prediction, AggNet utilizes the pre-trained protein language model ESM2 to extract informative representations and further fuse with traditional AAindex (Kawashima,&nbsp;<span><a href="#pro70031-bib-0015" id="#pro70031-bib-0015_R_d1967767e566" class="bibLink tab-link" data-tab="pane-pcw-references">2000</a></span>) features for amino acid sequence modeling. For APR identification, it employs the three-dimensional structures predicted by AlphaFold2 to incorporate spatial information. By combining the strengths of both sequence and structure-based approaches, AggNet aims to overcome their respective limitations and address the issue of data insufficiency. Our benchmark comparisons confirm that AggNet outperforms existing methods across various datasets. Additionally, a case study on MEDI1912 further verifies AggNet's effectiveness for protein engineering to rank different variants. Our findings suggest that AggNet is a useful tool for studying protein aggregation and can contribute significantly to the development of therapeutic proteins and novel biomaterials.</p>
         </section>
      <section class="article-section__content" id="pro70031-sec-0002">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0002-title">2 RESULTS</h2>
         
         <section class="article-section__sub-content" id="pro70031-sec-0003">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0003-title">2.1 The architecture of AggNet</h3>
            
            <p>The architecture of AggNet, illustrated in Figure&nbsp;<a href="#pro70031-fig-0001">1</a>, comprises two primary submodules: APNet and APRNet. APNet is tasked with predicting amyloid peptides, while APRNet focuses on profiling protein APRs. APNet initiates the processing of peptide inputs through one-hot encoding and AAindex feature encoding. These encoded features are subsequently integrated with a feature fusion module that synthesizes an informative fused embedding. This embedding is then input into a multilayer perceptron (MLP) to estimate the aggregation propensity (AP) score of a peptide. In contrast, APRNet processes proteins by employing both a sequential and a structural channel. The sequential channel dissects the primary sequence into hexapeptides, assesses them using a sliding window approach, and scores them via the trained APNet to predict their intrinsic aggregation scores. Concurrently, the structural channel folds the corresponding 3D structure using AlphaFold2 and extracts spatial information, including neighborhood interactions and relative solvent accessible surface areas (RSA). After data preparation, this sequential and structural information is aggregated at the residue level to profile the aggregation characteristics of specific proteins. AggNet alleviates data scarcity by leveraging ESM2 and addresses the lack of structural data through the use of AlphaFold2. Moreover, the novel feature fusion module integrates sequential and evolutionary information from ESM embeddings with physicochemical data from AAindex features, thereby enhancing the performance in both amyloid peptide classification and protein APR identification. More details can be found in the method section.</p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="pro70031-fig-0001"><a target="_blank" href="/cms/asset/fad3fa0e-20a1-41d4-91b0-3c434a403495/pro70031-fig-0001-m.jpg"><picture>
                        <source srcset="/cms/asset/fad3fa0e-20a1-41d4-91b0-3c434a403495/pro70031-fig-0001-m.jpg" media="(min-width: 1650px)"></source><img class="figure__image" src="/cms/asset/6083c5d8-32d9-4a3d-b33a-eed0d1b721d8/pro70031-fig-0001-m.png" data-lg-src="/cms/asset/fad3fa0e-20a1-41d4-91b0-3c434a403495/pro70031-fig-0001-m.jpg" alt="Details are in the caption following the image" title="Details are in the caption following the image" loading="lazy"></picture></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure__title">FIGURE 1<span style="font-weight:normal"></span></strong><div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="/action/downloadFigures?id=pro70031-fig-0001&amp;partId=&amp;doi=10.1002%2Fpro.70031" class="ppt-figure-link"><i aria-hidden="true" class="icon-Icon_Download"></i><span>PowerPoint</span></a></div>
                     </div>
                     <div class="figure__caption figure__caption-text">Schematic of the AggNet framework. It comprises two submodules: APNet for amyloid peptide classification and APRNet for protein APR identification.</div>
                  </figcaption>
               </figure>
            </section>
            </section>
         
         <section class="article-section__sub-content" id="pro70031-sec-0004">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0004-title">2.2 AggNet's discriminative power in amyloid and non-amyloid peptide classification</h3>
            
            <p>Discriminating between amyloid and non-amyloid peptides is a critical task in predicting protein aggregation, essential for understanding the formation of amyloid fibers associated with various neurodegenerative diseases. We first conduct a benchmark comparison using the Hex142 dataset (see the method section) to highlight the advantages of AggNet in classifying amyloid and non-amyloid peptides.</p>
            
            <p>First, we compare the AggNet with typical machine learning methods and AggNet exhibits superior performance over conventional machine learning algorithms including k-nearest neighbor (KNN), logistic regression (LR), Naive Bayes (NB), support vector machine (SVM), multilayer perceptron (MLP), in discriminating between amyloid and non-amyloid peptides. The comparative analysis, detailed in Tables&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S1</a> and <a class="suppLink scrollableLink" href="#support-information-section">S2</a>, highlights AggNet's advancement over both basic one-hot feature-based models and more complex AAindex feature-based models, demonstrating a significant improvement in both F1 score and Matthews correlation coefficient (MCC). Importantly, while the best performing traditional models like MLP (using one-hot features) and Logistic Regression (using AAindex features) achieve an AUC of up to 0.902, AggNet surpasses these with notably higher F1 scores and MCC values, exceeding 80% in F1 score, indicative of its balanced prediction capability.</p>
            
            <p>Furthermore, we compare the performance between AggNet and existing methods to show its superiority. The benchmark comparisons outlined in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#pro70031-tbl-0001">1</a> confirm AggNet's state-of-the-art performance in amyloid peptide classification across various metrics including accuracy (ACC), sensitivity (SP), specificity (Q), F1 score, MCC, and area under the curve (AUC). AggNet achieves an accuracy of 87.3%, sensitivity of 80.4%, specificity of 91.2%, F1 score of 82.0%, MCC of 0.723, and AUC of 0.913, which is notably higher than the next best method, ANuPP, by about 3.0% in AUC and 4.2% in F1 score. This superior performance, particularly in balancing precision and recall in an unbalanced training dataset, underscores the potential of protein language models in enhancing peptide representation for improved amyloid prediction. Additionally, we conducted a fivefold cross-validation using the widely recognized WALTZ-DB 2.0 dataset to further assess AggNet's performance relative to existing methods. Results detailed in Table&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a> reaffirm AggNet's superiority over contemporary approaches.</p>
            
            <div class="article-table-content" id="pro70031-tbl-0001">
               
               
               
               
               <header class="article-table-caption"><span class="table-caption__label">TABLE  1.
                    </span>Benchmark comparison of amyloid peptide classification on the Hex142 dataset.</header>
               
               
               
               <div class="article-table-content-wrapper" tabindex="0">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell right-bordered-cell left-aligned">Model</th>
                           
                           <th class="bottom-bordered-cell left-aligned">ACC (%)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">SE (%)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">SP (%)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Q (%)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">F1 (%)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">MCC</th>
                           
                           <th class="bottom-bordered-cell left-aligned">AUC</th>
                           </tr>
                        </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">TANGO</td>
                           
                           <td class="left-aligned">64.8</td>
                           
                           <td class="left-aligned">5.9</td>
                           
                           <td class="left-aligned">97.8</td>
                           
                           <td class="left-aligned">51.8</td>
                           
                           <td class="left-aligned">10.7</td>
                           
                           <td class="left-aligned">0.096</td>
                           
                           <td class="left-aligned">0.597</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">WALTZ</td>
                           
                           <td class="left-aligned">75.4</td>
                           
                           <td class="left-aligned">39.2</td>
                           
                           <td class="left-aligned">95.6</td>
                           
                           <td class="left-aligned">67.4</td>
                           
                           <td class="left-aligned">53.3</td>
                           
                           <td class="left-aligned">0.446</td>
                           
                           <td class="left-aligned">0.675</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">GAP</td>
                           
                           <td class="left-aligned">51.4</td>
                           
                           <td class="left-aligned">94.1</td>
                           
                           <td class="left-aligned">27.5</td>
                           
                           <td class="left-aligned">60.8</td>
                           
                           <td class="left-aligned">58.2</td>
                           
                           <td class="left-aligned">0.260</td>
                           
                           <td class="left-aligned">0.721</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">FishAmyloid</td>
                           
                           <td class="left-aligned">69.0</td>
                           
                           <td class="left-aligned">45.1</td>
                           
                           <td class="left-aligned">82.4</td>
                           
                           <td class="left-aligned">63.8</td>
                           
                           <td class="left-aligned">51.1</td>
                           
                           <td class="left-aligned">0.296</td>
                           
                           <td class="left-aligned">0.798</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Pasta2</td>
                           
                           <td class="left-aligned">75.4</td>
                           
                           <td class="left-aligned">37.3</td>
                           
                           <td class="left-aligned">96.7</td>
                           
                           <td class="left-aligned">67.0</td>
                           
                           <td class="left-aligned">52.1</td>
                           
                           <td class="left-aligned">0.450</td>
                           
                           <td class="left-aligned">0.855</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Aggrescan</td>
                           
                           <td class="left-aligned">79.6</td>
                           
                           <td class="left-aligned">68.6</td>
                           
                           <td class="left-aligned">85.7</td>
                           
                           <td class="left-aligned">77.2</td>
                           
                           <td class="left-aligned">70.7</td>
                           
                           <td class="left-aligned">0.551</td>
                           
                           <td class="left-aligned">0.855</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">ANuPP</td>
                           
                           <td class="left-aligned">83.1</td>
                           
                           <td class="left-aligned">82.4</td>
                           
                           <td class="left-aligned">83.5</td>
                           
                           <td class="left-aligned">82.9</td>
                           
                           <td class="left-aligned">77.8</td>
                           
                           <td class="left-aligned">0.645</td>
                           
                           <td class="left-aligned">0.883</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">AggNet</td>
                           
                           <td class="left-aligned">87.3</td>
                           
                           <td class="left-aligned">80.4</td>
                           
                           <td class="left-aligned">91.2</td>
                           
                           <td class="left-aligned">85.8</td>
                           
                           <td class="left-aligned">82.0</td>
                           
                           <td class="left-aligned">0.723</td>
                           
                           <td class="left-aligned">0.913</td>
                           </tr>
                        </tbody>
                     </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            </section>
         
         <section class="article-section__sub-content" id="pro70031-sec-0005">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0005-title">2.3 AggNet's proficiency in identifying APRs</h3>
            
            <p>Identifying APRs helps pinpoint the aggregation hotspots within specific proteins, enhances understanding of aggregation mechanisms, and informs strategies to mitigate aggregation. Similarly, we conduct a benchmark study to validate the superiority of AggNet in performing this task across diverse proteins.</p>
            
            <p>AggNet sets a new standard in identifying APRs of various proteins, outperforming existing models as shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#pro70031-tbl-0002">2</a>. It demonstrates substantial gains across all performance metrics compared with the second-best model, ANuPP. For instance, AggNet achieves a Segment Overlap (SOV) Overall score of 54.6 and an SOV Average score of 51.4, surpassing ANuPP by 8.8% and 5.5%, respectively. The significant performance drop when excluding structural information from AggNet's inputs highlights the critical role of spatial data in achieving these results.</p>
            
            <div class="article-table-content" id="pro70031-tbl-0002">
               
               
               
               
               <header class="article-table-caption"><span class="table-caption__label">TABLE  2.
                    </span>Benchmark comparison of protein APR identification on the Amy37 dataset.</header>
               
               
               
               <div class="article-table-content-wrapper" tabindex="0">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell right-bordered-cell left-aligned">Model</th>
                           
                           <th class="bottom-bordered-cell left-aligned">SOV APR</th>
                           
                           <th class="bottom-bordered-cell left-aligned">SOV non-APR</th>
                           
                           <th class="bottom-bordered-cell left-aligned">SOV overall</th>
                           
                           <th class="bottom-bordered-cell left-aligned">SOV average</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Total score<a class="noteLink scrollableLink" data-noteId="pro70031-note-0002" title="Link to note" id="pro70031-note-0002_34-controller" href="#pro70031-note-0002_34" aria-haspopup="false" aria-expanded="false" aria-label="Note"><sup>a</sup></a>
                              </th>
                           </tr>
                        </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Pasta2</td>
                           
                           <td class="left-aligned">13.2</td>
                           
                           <td class="left-aligned">24.9</td>
                           
                           <td class="left-aligned">23.2</td>
                           
                           <td class="left-aligned">19.1</td>
                           
                           <td class="left-aligned">42.3</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">WALTZ</td>
                           
                           <td class="left-aligned">44.4</td>
                           
                           <td class="left-aligned">28.9</td>
                           
                           <td class="left-aligned">28.7</td>
                           
                           <td class="left-aligned">36.6</td>
                           
                           <td class="left-aligned">65.3</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">FishAmyloid</td>
                           
                           <td class="left-aligned">14.5</td>
                           
                           <td class="left-aligned">45.2</td>
                           
                           <td class="left-aligned">37.5</td>
                           
                           <td class="left-aligned">29.9</td>
                           
                           <td class="left-aligned">67.4</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Aggrescan</td>
                           
                           <td class="left-aligned">34.3</td>
                           
                           <td class="left-aligned">36.5</td>
                           
                           <td class="left-aligned">32.4</td>
                           
                           <td class="left-aligned">35.4</td>
                           
                           <td class="left-aligned">67.8</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">TANGO</td>
                           
                           <td class="left-aligned">19.1</td>
                           
                           <td class="left-aligned">57.8</td>
                           
                           <td class="left-aligned">48.1</td>
                           
                           <td class="left-aligned">38.5</td>
                           
                           <td class="left-aligned">86.6</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">ANuPP</td>
                           
                           <td class="left-aligned">45.2</td>
                           
                           <td class="left-aligned">52.3</td>
                           
                           <td class="left-aligned">50.2</td>
                           
                           <td class="left-aligned">48.7</td>
                           
                           <td class="left-aligned">98.9</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">AggNet (w/o structure)</td>
                           
                           <td class="left-aligned">27.2</td>
                           
                           <td class="left-aligned">59.2</td>
                           
                           <td class="left-aligned">51.0</td>
                           
                           <td class="left-aligned">43.2</td>
                           
                           <td class="left-aligned">94.3</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">AggNet</td>
                           
                           <td class="left-aligned">48.1</td>
                           
                           <td class="left-aligned">54.6</td>
                           
                           <td class="left-aligned">54.6</td>
                           
                           <td class="left-aligned">51.4</td>
                           
                           <td class="left-aligned">106.0</td>
                           </tr>
                        </tbody>
                     </table>
               </div>
               
               
               
               <div class="article-section__table-footnotes">
                  <ul>
                     <li id="pro70031-note-0002" class="footNotePopup__item" title="Footnote 1"><span class="number"><sup>a</sup> </span>
                        
                        Total score=SOV overall + SOV average.
                        </li>
                  </ul>
               </div>
               <div class="article-section__table-source"></div>
            </div>
            
            <p>Moreover, AggNet's ability to identify APRs across diverse secondary structureshelices, strands, or coilsis visually depicted in Figure&nbsp;<a href="#pro70031-fig-0002">2</a>. Notable examples include the experimental APRs of Apolipoprotein C-II, predominantly -helical, and Gamma-crystallin D, primarily -strand, alongside Chorion class A protein PC292, which is mostly coiled. AggNet's predictions align closely with experimental data, albeit with slight deviations at the residue level, exemplifying its precision in localizing APRs across different protein structures. Specifically, the predicted APR of Apolipoprotein C-II is located in residues 6176 while the experimental one is in residues 6070 (Wilson et al.,&nbsp;<span><a href="#pro70031-bib-0040" id="#pro70031-bib-0040_R_d1967767e1204" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>). The predicted APR of Gamma-crystallin D is located in residues 91146 while the experimental one is in residues 80163 (Moran, Decatur, &amp; Zanni,&nbsp;<span><a href="#pro70031-bib-0025" id="#pro70031-bib-0025_R_d1967767e1207" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Moran, Woys, et al.,&nbsp;<span><a href="#pro70031-bib-0026" id="#pro70031-bib-0026_R_d1967767e1210" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>). The predicted APR of Chorion class A protein PC292 is located in residues 46103 while the experimental one is in residues 4896 (Iconomidou et al.,&nbsp;<span><a href="#pro70031-bib-0012" id="#pro70031-bib-0012_R_d1967767e1213" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>).</p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="pro70031-fig-0002"><a target="_blank" href="/cms/asset/3330ec07-84fe-4d75-923d-97f73fa324e8/pro70031-fig-0002-m.jpg"><picture>
                        <source srcset="/cms/asset/3330ec07-84fe-4d75-923d-97f73fa324e8/pro70031-fig-0002-m.jpg" media="(min-width: 1650px)"></source><img class="figure__image" src="/cms/asset/1adca8bb-a738-4213-8f5a-0eefd253658b/pro70031-fig-0002-m.png" data-lg-src="/cms/asset/3330ec07-84fe-4d75-923d-97f73fa324e8/pro70031-fig-0002-m.jpg" alt="Details are in the caption following the image" title="Details are in the caption following the image" loading="lazy"></picture></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure__title">FIGURE 2<span style="font-weight:normal"></span></strong><div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="/action/downloadFigures?id=pro70031-fig-0002&amp;partId=&amp;doi=10.1002%2Fpro.70031" class="ppt-figure-link"><i aria-hidden="true" class="icon-Icon_Download"></i><span>PowerPoint</span></a></div>
                     </div>
                     <div class="figure__caption figure__caption-text">Comparison of predicted and experimental APRs. APRs are indicated in red. (a) Experimental APR of Apolipoprotein C-II. (b) Predicted APR of Apolipoprotein C-II. (c) Experimental APR of Gamma-crystallin D. (d) Predicted APR of Gamma-crystallin D. (e) Experimental APR of Chorion class A protein PC292. (f) Predicted APR of Chorion class A protein PC292.</div>
                  </figcaption>
               </figure>
            </section>
            </section>
         
         <section class="article-section__sub-content" id="pro70031-sec-0006">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0006-title">2.4 Ablation study and feature analysis</h3>
            
            <p>The input of AggNet is one-hot encodings of original peptide sequences and the corresponding AAindex features. AggNet's sophisticated integration of ESM embeddings and AAindex features results in an informative fused embedding that transcends traditional one-hot encoding and simple feature-based approaches. This fusion captures both sequence evolution information from ESM embeddings and physiochemical data from AAindex features, significantly enhancing the model's discriminatory capabilities. To enhance our understanding of AggNet's prediction mechanisms and improve its interpretability, we conduct an ablation study and feature analysis on the learned fused embeddings.</p>
            
            <p>The results of the ablation study presented in Table&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S4</a> illustrate a significant decline in performance when the model is deprived of either ESM2 embeddings or AAindex features. Similarly, the removal of element-wise multiplication without a skip connection in the feature fusion process also leads to a comparable decrease in performance. These findings underscore the crucial impact of the feature fusion strategy on the final classification outcomes. By integrating our specifically designed feature strategy for ESM2 embeddings and AAindex features, the model achieves a more balanced predictive performance, particularly in the discrimination of amyloid peptides. This balance is vital for enhancing the model's accuracy and reliability in practical applications.</p>
            
            <p>Dimensionality reduction and visualization using t-SNE (Hinton &amp; Roweis,&nbsp;<span><a href="#pro70031-bib-0009" id="#pro70031-bib-0009_R_d1967767e1254" class="bibLink tab-link" data-tab="pane-pcw-references">2002</a></span>), illustrated in Figure&nbsp;<a href="#pro70031-fig-0003">3</a>, reveal clear distinctions between amyloid and non-amyloid peptides. This separation is especially evident among peptides with varying counts of hydrophobic residues, demonstrating AggNet's ability to accurately reflect the physiochemical properties of peptides. It is evident that peptides with a lower number of hydrophobic residues (02) tend to cluster together, in contrast to peptides with a higher count (56), which are grouped separately. Peptides with a moderate number of hydrophobic residues (34) are distributed between these two extremes. This pattern underscores the model's capability to discern and represent the physicochemical properties of peptides effectively. Such clear stratification enhances the interpretability of how AggNet discriminates between amyloid and non-amyloid peptides, showcasing its analytical strength.</p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="pro70031-fig-0003"><a target="_blank" href="/cms/asset/f49bab1c-4e1b-4f25-9ee1-c813f49d7396/pro70031-fig-0003-m.jpg"><picture>
                        <source srcset="/cms/asset/f49bab1c-4e1b-4f25-9ee1-c813f49d7396/pro70031-fig-0003-m.jpg" media="(min-width: 1650px)"></source><img class="figure__image" src="/cms/asset/f3765160-2761-4ab0-acd8-8d3daac04b83/pro70031-fig-0003-m.png" data-lg-src="/cms/asset/f49bab1c-4e1b-4f25-9ee1-c813f49d7396/pro70031-fig-0003-m.jpg" alt="Details are in the caption following the image" title="Details are in the caption following the image" loading="lazy"></picture></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure__title">FIGURE 3<span style="font-weight:normal"></span></strong><div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="/action/downloadFigures?id=pro70031-fig-0003&amp;partId=&amp;doi=10.1002%2Fpro.70031" class="ppt-figure-link"><i aria-hidden="true" class="icon-Icon_Download"></i><span>PowerPoint</span></a></div>
                     </div>
                     <div class="figure__caption figure__caption-text">The t-SNE visualization of peptide embeddings in AggNet. (a) Peptides are depicted as scatter points, color-coded by their labels: Amyloid or non-amyloid. (b) Peptides are color-coded based on the count of hydrophobic residues they contain.</div>
                  </figcaption>
               </figure>
            </section>
            </section>
         
         <section class="article-section__sub-content" id="pro70031-sec-0007">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0007-title">2.5 Case study: MEDI1912 antibody</h3>
            
            <p>To further validate AggNet's utility in protein engineering aimed at reducing aggregation, we analyzed variants of the MEDI1912 antibody using the tripartite -lactamase enzyme assay (TPBLA). This novel in vivo method correlates with traditional aggregation assays like HP-SEC and AC-SINS, as evidenced by the area under the bacterial growth curve measurements (Ebo et al.,&nbsp;<span><a href="#pro70031-bib-0006" id="#pro70031-bib-0006_R_d1967767e1292" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>).</p>
            
            <p>Analysis of MEDI1912 and its variantsspecifically, STT and M139using AggNet revealed distinct APR score distributions in their VH domains, shown in Figure&nbsp;<a href="#pro70031-fig-0004">4</a>. Notably, the STT variant, known for its reduced aggregation propensity (Dobson et al.,&nbsp;<span><a href="#pro70031-bib-0005" id="#pro70031-bib-0005_R_d1967767e1301" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>), and M139, the most effective variant evolved by TPBLA (Ebo et al.,&nbsp;<span><a href="#pro70031-bib-0006" id="#pro70031-bib-0006_R_d1967767e1304" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>), showed significant differences in the CDRH1 region compared with wild-type MEDI1912. Specifically, the predicted scores by AggNet show that WFL has a high peak in the CDR1, STT has a lower peak, while the M139 eliminates the peak. These findings align well with experimental outcomes, demonstrating AggNet's potential to predict how specific mutations influence aggregation tendencies.</p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="pro70031-fig-0004"><a target="_blank" href="/cms/asset/b4250d73-f0e2-433a-bbf7-2a50bab8bd19/pro70031-fig-0004-m.jpg"><picture>
                        <source srcset="/cms/asset/b4250d73-f0e2-433a-bbf7-2a50bab8bd19/pro70031-fig-0004-m.jpg" media="(min-width: 1650px)"></source><img class="figure__image" src="/cms/asset/b9828a66-1d1e-4349-9df2-60dc6b18b533/pro70031-fig-0004-m.png" data-lg-src="/cms/asset/b4250d73-f0e2-433a-bbf7-2a50bab8bd19/pro70031-fig-0004-m.jpg" alt="Details are in the caption following the image" title="Details are in the caption following the image" loading="lazy"></picture></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure__title">FIGURE 4<span style="font-weight:normal"></span></strong><div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="/action/downloadFigures?id=pro70031-fig-0004&amp;partId=&amp;doi=10.1002%2Fpro.70031" class="ppt-figure-link"><i aria-hidden="true" class="icon-Icon_Download"></i><span>PowerPoint</span></a></div>
                     </div>
                     <div class="figure__caption figure__caption-text">Aggregation profile of the VH Domain in WFL, STT, and M139 as predicted by AggNet. The CDR regions are highlighted in gray.</div>
                  </figcaption>
               </figure>
            </section>
            
            <p>Additionally, AggNet's ability to rank the aggregation propensity of MEDI1912 variants was rigorously tested. The model's predictions for the seven specific variants at specific sites correlate strongly with experimental in vivo growth (A.U.) values, with Pearson and Spearman correlation coefficients of 0.86 and 0.95, respectively. Consistent with the experimental results, WFL exhibits the highest aggregation propensity and receives the highest predicted score, while STT shows the converse. Hence, we verify the efficacy of AggNet to guide the mutation design for aggregation mitigation by scoring different protein variants of MEDI1912.</p>
            
            <p>Furthermore, we evaluated whether AggNet remains effective when applied to WFL variants harboring multiple mutations. A total of 162 variants evolved via TPBLA were analyzed, 115 of which contain more than three mutations (Ebo et al.,&nbsp;<span><a href="#pro70031-bib-0006" id="#pro70031-bib-0006_R_d1967767e1333" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>). The results are presented in Figure&nbsp;<a href="#pro70031-fig-0005">5b</a>, omitting the seven variants of WFL previously discussed. The scatter plot delineates a negative correlation between predicted aggregation scores and in vivo growth rates (A.U.), suggesting an intuitive inverse relationship. Although the specific predicted values lack precision, they generally mirror the trend observed in experimental data: higher aggregation propensities correlate with reduced experimental growth rates. Notably, AggNet demonstrates superior performance with a Pearson correlation coefficient of 0.32 and a Spearman correlation coefficient of 0.34, surpassing state-of-the-art methods Aggrescan3D and CamSol, as shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#pro70031-tbl-0003">3</a>. This comparison further underscores AggNet's advantage in ranking protein variants, efficiently capturing the impact of multiple mutations on the aggregation landscape without necessitating recalibration or additional data. Collectively, these findings corroborate the utility of AggNet in protein aggregation studies and its potential in guiding mutation design to reduce aggregation in therapeutic proteins.</p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="pro70031-fig-0005"><a target="_blank" href="/cms/asset/53ce9af2-c2b0-4196-8487-fca405c7c00b/pro70031-fig-0005-m.jpg"><picture>
                        <source srcset="/cms/asset/53ce9af2-c2b0-4196-8487-fca405c7c00b/pro70031-fig-0005-m.jpg" media="(min-width: 1650px)"></source><img class="figure__image" src="/cms/asset/e74ab358-6473-42cf-ac37-80bdf2fa9111/pro70031-fig-0005-m.png" data-lg-src="/cms/asset/53ce9af2-c2b0-4196-8487-fca405c7c00b/pro70031-fig-0005-m.jpg" alt="Details are in the caption following the image" title="Details are in the caption following the image" loading="lazy"></picture></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure__title">FIGURE 5<span style="font-weight:normal"></span></strong><div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="/action/downloadFigures?id=pro70031-fig-0005&amp;partId=&amp;doi=10.1002%2Fpro.70031" class="ppt-figure-link"><i aria-hidden="true" class="icon-Icon_Download"></i><span>PowerPoint</span></a></div>
                     </div>
                     <div class="figure__caption figure__caption-text">Performance of AggNet on the MEDI1912 dataset from TPBLA in vivo experiments. (a) Scatter plot of different WFL variants with mutations in specific sites. (b) Scatter plot of WFL variants evolved by TPBLA.</div>
                  </figcaption>
               </figure>
            </section>
            
            <div class="article-table-content" id="pro70031-tbl-0003">
               
               
               
               
               <header class="article-table-caption"><span class="table-caption__label">TABLE  3.
                    </span>Performance comparison on MEDI1912 dataset.</header>
               
               
               
               <div class="article-table-content-wrapper" tabindex="0">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell right-bordered-cell left-aligned">Model</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Pearson <i>r</i></th>
                           
                           <th class="bottom-bordered-cell left-aligned">Spearman <i>r</i></th>
                           </tr>
                        </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Aggrescan3D</td>
                           
                           <td class="left-aligned">0.07</td>
                           
                           <td class="left-aligned">0.11</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">CamSol</td>
                           
                           <td class="left-aligned">0.29</td>
                           
                           <td class="left-aligned">0.29</td>
                           </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">AggNet</td>
                           
                           <td class="left-aligned">0.32</td>
                           
                           <td class="left-aligned">0.34</td>
                           </tr>
                        </tbody>
                     </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            </section>
         </section>
      <section class="article-section__content" id="pro70031-sec-0008">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0008-title">3 DISCUSSION AND CONCLUSION</h2>
         
         <p>In this study, we introduced AggNet, a deep learning framework that leverages physicochemical, sequential, and structural information to address the challenge of protein aggregation, including amyloid peptide prediction and protein APR identification. To mitigate the issue of insufficient data and to derive more informative amino acid sequence embeddings, we utilized the protein language model ESM2 (Lin et al.,&nbsp;<span><a href="#pro70031-bib-0017" id="#pro70031-bib-0017_R_d1967767e1454" class="bibLink tab-link" data-tab="pane-pcw-references">2023</a></span>) for feature extraction. We further enhanced the integration of physicochemical features and evolutionary information from ESM2 through a feature fusion layer, facilitating the learning of fused embeddings.</p>
         
         <p>Performance comparisons in amyloid peptide classification and analyses of the fused embedding landscape affirm the efficacy of our feature fusion approach. Moreover, to augment the prediction accuracy in APR identification, we employed AlphaFold2 (Mirdita et al.,&nbsp;<span><a href="#pro70031-bib-0024" id="#pro70031-bib-0024_R_d1967767e1460" class="bibLink tab-link" data-tab="pane-pcw-references">2022</a></span>; Tunyasuvunakool et al.,&nbsp;<span><a href="#pro70031-bib-0037" id="#pro70031-bib-0037_R_d1967767e1463" class="bibLink tab-link" data-tab="pane-pcw-references">2021</a></span>) to model the corresponding structures and extract spatial information. By leveraging aggregation scores from neighboring residues and their RSA, AggNet achieved state-of-the-art performance, significantly outperforming existing methods. Visual comparisons of predicted and experimental APRs across various secondary structureshelices, strands, and coilsillustrate AggNet's precision in locating APRs.</p>
         
         <p>AggNet addresses several limitations of existing models in predicting APRs in proteins. Sequence-based models such as ANuPP often struggle to identify the hydrophobic core of proteins, leading to a high rate of false positive predictions. In contrast, AggNet not only incorporates sequential context to score each residue but also leverages spatial neighbor information, resulting in more reliable APR predictions and a reduced false positive rate. Similarly, while Aggrescan3D relies on a scoring formula based on intrinsic aggregation scores derived from prior experimental data, these scores are static for a given residue regardless of its context within the protein structure, thereby constraining performance. AggNet overcomes this limitation by dynamically computing the intrinsic aggregation score for each residue based on its sequential context, utilizing APNet, a deep learning-based framework. This approach enhances the flexibility and accuracy of the scoring process.</p>
         
         <p>Beyond benchmark comparisons, we employed the MEDI1912 antibody as a case study to simulate a real-world protein engineering task. This included an examination of how AggNet's predicted APR profiles influence experimental outcomes by comparing the APR score landscapes in the VH domains of WFL, STT, and M139 variants. We also assessed AggNet's ranking performance on WFL variants with three mutations and on TPBLA-evolved WFL variants with more than three mutations. The results confirmed that, even without recalibration or fine-tuning with additional data, AggNet provides reasonable predictions, underscoring its utility in protein engineering for aggregation mitigation.</p>
         
         <p>AggNet represents a valuable tool for protein aggregation research and has potential applications in protein engineering. The integration of deep learning technology and protein language models offers promising insights for this field. However, there are limitations to AggNet, such as its suboptimal blind test performance in downstream tasks, like protein engineering, where it does not directly offer mutational recommendations. Besides, predicting aggregation effects of multiple mutations is still challenging for AggNet. Advancing the development of more effective aggregation prediction models remains both challenging and essential, particularly for variants that exhibit multiple mutations. Continued innovation from the computer science and deep learning communities is expected to further advance the field, enhancing the predictive accuracy and utility of protein aggregation prediction.</p>
         </section>
      <section class="article-section__content" id="pro70031-sec-0009">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0009-title">4 MATERIALS AND METHODS</h2>
         
         <section class="article-section__sub-content" id="pro70031-sec-0010">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0010-title">4.1 Dataset preparation</h3>
            
            <p>For amyloid peptide identification, we have two schemes including train-test and cross-validation. For the train-test split, we used the Hex1421 dataset, which comprises 1421 experimentally validated amyloidogenic and non-amyloidogenic hexapeptides sourced from the CPAD 2.0 database (Rawat et al.,&nbsp;<span><a href="#pro70031-bib-0031" id="#pro70031-bib-0031_R_d1967767e1487" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>). This dataset is non-redundant, ensuring a broad representation of peptide varieties. We adhered to a 90%:10% division for training (Hex1279) and testing sets (Hex142), respectively, resulting in 1279 peptides for training and 142 for validation, maintaining consistency with the train-test split used in ANuPP. Detailed dataset characteristics are provided in Table&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S5</a>. For the additional fivefold cross-validation, a dataset comprising 1416 non-redundant peptides was curated from WALTZ-DB 2.0 (Louros, Konstantoulea, et al.,&nbsp;<span><a href="#pro70031-bib-0018" id="#pro70031-bib-0018_R_d1967767e1493" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>), which stands as one of the largest publicly accessible repositories of experimentally validated amyloidogenic peptides. This dataset includes 901 amyloid hexapeptides and 515 non-amyloid hexapeptides. Each fold is based on a 4:1 stratified sampling of amyloid and non-amyloid categories to avoid data bias.</p>
            
            <p>For protein APR identification, we adopted ANuPP's methodology, selecting 162 proteins from the AmyPro database (Varadi et al.,&nbsp;<span><a href="#pro70031-bib-0038" id="#pro70031-bib-0038_R_d1967767e1499" class="bibLink tab-link" data-tab="pane-pcw-references">2018</a></span>). Proteins with unclear or ambiguous APR annotations or those with APR residue fractions outside the 10%95% range were excluded. We applied CD-HIT (Huang et al.,&nbsp;<span><a href="#pro70031-bib-0011" id="#pro70031-bib-0011_R_d1967767e1502" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>) to cluster these sequences at 40% sequence identity to reduce redundancy, culminating in a distilled set of 54 non-redundant proteins. These were further divided based on the presence of hexapeptides into two subsets: Amy17, containing proteins with more than one hexapeptide overlapping with Hex1279, and Amy37, comprising proteins with none or one hexapeptide from Hex1279, serving respectively for calibration and assessment.</p>
            
            <p>In the context of a case study on MEDI1912, a targeted dataset was compiled, consisting of seven scFv variants of WFL, each displaying mutations at specified sites. The specific variants, derived via TPBLA (Ebo et al.,&nbsp;<span><a href="#pro70031-bib-0006" id="#pro70031-bib-0006_R_d1967767e1508" class="bibLink tab-link" data-tab="pane-pcw-references">2020</a></span>), include WFT, WTL, WTT, SFL, STL, SFT, and STT, where WFL denotes W35/F36/L64, STT signifies S35/T36/T64 mutations, and so on. Additionally, a more extensive dataset of scFv variants evolved through TPBLA, was acquired from the corresponding author for comprehensive analysis. The detailed statistical information of these mutants is summarized in Table&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S6</a>.</p>
            </section>
         
         <section class="article-section__sub-content" id="pro70031-sec-0011">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0011-title">4.2 Model architecture</h3>
            
            <section class="article-section__sub-content" id="pro70031-sec-0012">
               
               <h4 class="article-section__sub-title section3" id="pro70031-sec-0012-title">4.2.1 <i>APNet architecture</i></h4>
               
               <p>APNet integrates the ESM2 module, a feature engineering module, a feature fusion module (illustrated in Figure&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S1</a>), and a prediction module. Based on existing research and insights from biophysics, the amyloidogenic properties of peptides are primarily influenced by their intrinsic physicochemical characteristics. To capture these properties, we employ features derived from the AAindex database to represent each peptide. In addition to traditional feature representations like AAindex, recent studies have demonstrated that protein language models, such as ESM2, are capable of effectively learning sequential, evolutionary, and physicochemical features of amino acid sequences, achieving notable performance across various tasks. Therefore, we hypothesize that integrating these two complementary and diverse feature setsAAindex-derived features and representations from protein language modelscan enhance the model's ability to discern the discriminative boundary between amyloidogenic and non-amyloidogenic peptides. ESM2 extracts embeddings for each residue, producing a 1280-dimensional vector encapsulating evolutionary and sequential information. These are dimensionally reduced to 256 dimensions via a multilayer perceptron (MLP) (Taud &amp; Mas,&nbsp;<span><a href="#pro70031-bib-0036" id="#pro70031-bib-0036_R_d1967767e1531" class="bibLink tab-link" data-tab="pane-pcw-references">2018</a></span>) and aggregated through mean pooling to form a peptide-level representation, denoted as <span class="fallback__mathEquation" data-altimg="/cms/asset/a29f0c59-e7a5-4c9c-9531-80e4386eed03/pro70031-math-0001.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0001" display="inline" location="graphic/pro70031-math-0001.png">
                     <semantics>
                        <mrow>
                           <mi>x</mi>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ x $$</annotation>
                     </semantics></math>. In parallel, the feature engineering module initially narrows down the set of 3396 AAindex features to the 600 most informative ones using ANOVA. Subsequently, this refined dataset is processed through an additional MLP to transform the non-redundant physicochemical AAindex features into a 256-dimensional embedding, denoted as <span class="fallback__mathEquation" data-altimg="/cms/asset/76d7cbc8-eed2-4062-b391-699a832016c7/pro70031-math-0002.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0002" display="inline" location="graphic/pro70031-math-0002.png">
                     <semantics>
                        <mrow>
                           <mi>y</mi>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ y $$</annotation>
                     </semantics></math>. These embeddings, denoted as <span class="fallback__mathEquation" data-altimg="/cms/asset/9ec282b1-1b55-4f50-88ac-7f2abd7df7a0/pro70031-math-0003.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0003" display="inline" location="graphic/pro70031-math-0003.png">
                     <semantics>
                        <mrow>
                           <mi>x</mi>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ x $$</annotation>
                     </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/8da4ea0e-078a-4700-badb-a067268ffa31/pro70031-math-0004.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0004" display="inline" location="graphic/pro70031-math-0004.png">
                     <semantics>
                        <mrow>
                           <mi>y</mi>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ y $$</annotation>
                     </semantics></math>, are then combined in the feature fusion module to form a more expressive representation <span class="fallback__mathEquation" data-altimg="/cms/asset/e704bc6e-8a45-41ed-a44d-3399b59c328c/pro70031-math-0005.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0005" display="inline" location="graphic/pro70031-math-0005.png">
                     <semantics>
                        <mrow>
                           <mi>z</mi>
                           <mo>=</mo>
                           <mi>f</mi>
                           <mfenced close=")" open="(">
                              <mrow>
                                 <mi>x</mi>
                                 <mo>,</mo>
                                 <mi>y</mi>
                              </mrow>
                           </mfenced>
                           <mo>+</mo>
                           <mi>x</mi>
                           <mo>=</mo>
                           <mi>x</mi>
                           <mo>*</mo>
                           <mi>y</mi>
                           <mo>+</mo>
                           <mi>x</mi>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ z=f\left(x,y\right)+x={x}^{\ast }y+x $$</annotation>
                     </semantics></math>. Here, we utilize element-wise multiplication and skip connection (He et al.,&nbsp;<span><a href="#pro70031-bib-0008" id="#pro70031-bib-0008_R_d1967767e1586" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>) for feature expression capabilities enhancement and efficient learning, respectively. This fused representation is further processed by the final MLP in the aggregation prediction head to yield the final aggregation propensity score, with higher scores indicating a greater intrinsic propensity for aggregation.</p>
               </section>
            
            <section class="article-section__sub-content" id="pro70031-sec-0013">
               
               <h4 class="article-section__sub-title section3" id="pro70031-sec-0013-title">4.2.2 <i>APRNet architecture</i></h4>
               
               <p>APRNet comprises modules for sequence information extraction, structure prediction, and score aggregation. For sequence data, protein sequences are dissected into hexapeptides using sliding windows, each scored via APNet. The intrinsic aggregation score for each residue is calculated as the average score from all hexapeptides encompassing that residue. For instance, the aggregation score for residue with index 9 is the average of the predicted scores for hexapeptides spanning residues 49, 510, 611, 712, 813, and 914. For the structure part, AlphaFold2 predicts the protein's 3D structure, facilitating the computation of an adjacency matrix and relative solvent accessible surface area (RSA) for each residue to identify spatial neighborhoods within an 8 radius and determine exposure levels. The aggregation score for each residue is adjusted based on its neighbors' scores, their distances, and RSA, following the formula: <span class="fallback__mathEquation" data-altimg="/cms/asset/856b3cb0-7426-4b67-bf8a-183a4051665d/pro70031-math-0006.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0006" display="inline" location="graphic/pro70031-math-0006.png">
                     <semantics>
                        <mrow>
                           <msub>
                              <mi>E</mi>
                              <mi>i</mi>
                           </msub>
                           <mo>=</mo>
                           <munder>
                              <mo></mo>
                              <mi>j</mi>
                           </munder>
                           <msub>
                              <mi>s</mi>
                              <mi>j</mi>
                           </msub>
                           <mo></mo>
                           <msup>
                              <mi>e</mi>
                              <mrow>
                                 <mi></mi>
                                 <mo></mo>
                                 <msub>
                                    <mi>r</mi>
                                    <mi>j</mi>
                                 </msub>
                              </mrow>
                           </msup>
                           <mo></mo>
                           <msup>
                              <mi>e</mi>
                              <mrow>
                                 <mi></mi>
                                 <mo></mo>
                                 <msub>
                                    <mi>d</mi>
                                    <mi>j</mi>
                                 </msub>
                              </mrow>
                           </msup>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ {E}_i=\sum \limits_j{s}_j\bullet {e}^{\alpha \bullet {r}_j}\bullet {e}^{\beta \bullet {d}_j} $$</annotation>
                     </semantics></math>, where <i></i> and <i></i> are tunable parameters reflecting the increased aggregation contribution from surface-exposed residues and proximal neighbors (Brcenas et al.,&nbsp;<span><a href="#pro70031-bib-0002" id="#pro70031-bib-0002_R_d1967767e1636" class="bibLink tab-link" data-tab="pane-pcw-references">2024</a></span>). APRs are then delineated by identifying peak scores above a set threshold <span class="fallback__mathEquation" data-altimg="/cms/asset/8218ae10-fb68-4b1e-a330-a39c0ab81adb/pro70031-math-0007.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0007" display="inline" location="graphic/pro70031-math-0007.png">
                     <semantics>
                        <mrow>
                           <msub>
                              <mi>t</mi>
                              <mtext mathvariant="italic">peak</mtext>
                           </msub>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ {t}_{peak} $$</annotation>
                     </semantics></math> and extending these peaks in both directions to include neighboring residues scoring above a secondary threshold <span class="fallback__mathEquation" data-altimg="/cms/asset/700ae92f-271f-4133-a55f-84fae781d206/pro70031-math-0008.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0008" display="inline" location="graphic/pro70031-math-0008.png">
                     <semantics>
                        <mrow>
                           <msub>
                              <mi>t</mi>
                              <mtext mathvariant="italic">expand</mtext>
                           </msub>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ {t}_{expand} $$</annotation>
                     </semantics></math>. These parameters <span class="fallback__mathEquation" data-altimg="/cms/asset/164b7b83-ca77-4230-b923-b35a4cf69e89/pro70031-math-0009.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0009" display="inline" location="graphic/pro70031-math-0009.png">
                     <semantics>
                        <mrow>
                           <mfenced close="}" open="{" separators=",,,">
                              <mi></mi>
                              <mi></mi>
                              <msub>
                                 <mi>t</mi>
                                 <mtext mathvariant="italic">peak</mtext>
                              </msub>
                              <msub>
                                 <mi>t</mi>
                                 <mtext mathvariant="italic">expand</mtext>
                              </msub>
                           </mfenced>
                        </mrow>
                        <annotation encoding="application/x-tex">$$ \left\{\alpha, \beta, {t}_{peak},{t}_{expand}\right\} $$</annotation>
                     </semantics></math> are optimized using the Amy17 dataset (Prabakaran et al.,&nbsp;<span><a href="#pro70031-bib-0029" id="#pro70031-bib-0029_R_d1967767e1677" class="bibLink tab-link" data-tab="pane-pcw-references">2021</a></span>).</p>
               </section>
            
            <section class="article-section__sub-content" id="pro70031-sec-0014">
               
               <h4 class="article-section__sub-title section3" id="pro70031-sec-0014-title">4.2.3 <i>Training settings</i></h4>
               
               <p>The ESM2 module was kept frozen when training to prevent catastrophic forgetting caused by limited training data. This precaution ensures that the pre-trained model retains its learned features without adverse modifications. Alongside, we utilized the AdamW optimizer, favored for its adept management of weight decay and enhanced convergence characteristics, with an initial learning rate of 2e4. This rate was carefully chosen to accelerate training while maintaining the stability of the learning process. To address potential exploding gradient problems, we implemented gradient clipping with a norm threshold set at 1.0. APNet was trained for 25 epochs using batch sizes of 50 because we found that extending the training period led to negligible performance improvements. This batch size effectively balances computational resource utilization and the accuracy of gradient estimates, optimizing overall training efficiency. For hyper-parameters searching, we use the AutoML technique to determine the best one.</p>
               </section>
            </section>
         
         <section class="article-section__sub-content" id="pro70031-sec-0015">
            
            <h3 class="article-section__sub-title section2" id="pro70031-sec-0015-title">4.3 Evaluation metrics</h3>
            
            <div class="paragraph-element">For the amyloid peptide identification, common binary classification metrics are used to evaluate the performance of the proposed methods, including accuracy (ACC), sensitivity (SE), specificity (SP), Q score, F1 score, and Matthew correlation coefficient (MCC). The formulas of these metrics are described as follows:
               <div class="inline-equation" id="pro70031-disp-0001"><span class="inline-equation__construct"><span><!--FIGURE-->
                        <span class="fallback__mathEquation" data-altimg="/cms/asset/23a1cf11-363e-45a1-aa27-f72d4af8884a/pro70031-math-0010.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0010" display="block" location="graphic/pro70031-math-0010.png">
                           <semantics>
                              <mrow>
                                 <mfenced close="" open="{">
                                    <mtable columnalign="left">
                                       <mtr>
                                          <mtd>
                                             <mi>ACC</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mrow>
                                                   <mi>TP</mi>
                                                   <mo>+</mo>
                                                   <mi>TN</mi>
                                                </mrow>
                                                <mrow>
                                                   <mi>TP</mi>
                                                   <mo>+</mo>
                                                   <mi>TN</mi>
                                                   <mo>+</mo>
                                                   <mi>FP</mi>
                                                   <mo>+</mo>
                                                   <mi>FN</mi>
                                                </mrow>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd></mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi>SE</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mi>TP</mi>
                                                <mrow>
                                                   <mi>TP</mi>
                                                   <mo>+</mo>
                                                   <mi>FN</mi>
                                                </mrow>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd></mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi>SP</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mi>TN</mi>
                                                <mrow>
                                                   <mi>TN</mi>
                                                   <mo>+</mo>
                                                   <mi>FP</mi>
                                                </mrow>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd></mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi mathvariant="normal">Q</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mrow>
                                                   <mi>SE</mi>
                                                   <mo>+</mo>
                                                   <mi>SP</mi>
                                                </mrow>
                                                <mn>2</mn>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd></mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi mathvariant="normal">F</mi>
                                             <mn>1</mn>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mrow>
                                                   <mn>2</mn>
                                                   <mi>TP</mi>
                                                </mrow>
                                                <mrow>
                                                   <mn>2</mn>
                                                   <mi>TP</mi>
                                                   <mo>+</mo>
                                                   <mi>FN</mi>
                                                   <mo>+</mo>
                                                   <mi>FP</mi>
                                                </mrow>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd></mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi>MCC</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mrow>
                                                   <mi>TP</mi>
                                                   <mo></mo>
                                                   <mi>TN</mi>
                                                   <mo>+</mo>
                                                   <mi>FP</mi>
                                                   <mo></mo>
                                                   <mi>FN</mi>
                                                </mrow>
                                                <msqrt>
                                                   <mrow>
                                                      <mfenced close=")" open="(">
                                                         <mrow>
                                                            <mi>TP</mi>
                                                            <mo linebreak="goodbreak">+</mo>
                                                            <mi>FP</mi>
                                                         </mrow>
                                                      </mfenced>
                                                      <mfenced close=")" open="(">
                                                         <mrow>
                                                            <mi>TP</mi>
                                                            <mo linebreak="goodbreak">+</mo>
                                                            <mi>FN</mi>
                                                         </mrow>
                                                      </mfenced>
                                                      <mfenced close=")" open="(">
                                                         <mrow>
                                                            <mi>TN</mi>
                                                            <mo linebreak="goodbreak">+</mo>
                                                            <mi>FN</mi>
                                                         </mrow>
                                                      </mfenced>
                                                      <mfenced close=")" open="(">
                                                         <mrow>
                                                            <mi>TN</mi>
                                                            <mo linebreak="goodbreak">+</mo>
                                                            <mi>FP</mi>
                                                         </mrow>
                                                      </mfenced>
                                                   </mrow>
                                                </msqrt>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                    </mtable>
                                 </mfenced>
                              </mrow>
                              <annotation encoding="application/x-tex">$$ \left\{\begin{array}{l}\mathrm{ACC}=\frac{\mathrm{TP}+\mathrm{TN}}{\mathrm{TP}+\mathrm{TN}+\mathrm{FP}+\mathrm{FN}}\\ {}\\ {}\mathrm{SE}=\frac{\mathrm{TP}}{\mathrm{TP}+\mathrm{FN}}\\ {}\\ {}\mathrm{SP}=\frac{\mathrm{TN}}{\mathrm{TN}+\mathrm{FP}}\\ {}\\ {}\mathrm{Q}=\frac{\mathrm{SE}+\mathrm{SP}}{2}\\ {}\\ {}\mathrm{F}1=\frac{2\mathrm{TP}}{2\mathrm{TP}+\mathrm{FN}+\mathrm{FP}}\\ {}\\ {}\mathrm{MCC}=\frac{\mathrm{TP}\times \mathrm{TN}+\mathrm{FP}\times \mathrm{FN}}{\sqrt{\left(\mathrm{TP}+\mathrm{FP}\right)\left(\mathrm{TP}+\mathrm{FN}\right)\left(\mathrm{TN}+\mathrm{FN}\right)\left(\mathrm{TN}+\mathrm{FP}\right)}}\end{array}\right. $$</annotation>
                           </semantics></math>
                        </span></span><span class="inline-equation__label"></span></div>where TP is the number of true-positive samples, FP is the number of false-positive samples, TN is the number of true-negative samples, and FN is the number of false-negative samples. Besides, another metric called the area under the receiver operating characteristic curve (AUC) is used for the overall performance evaluation.</div>
            
            <div class="paragraph-element">For the protein APR identification, Segment OVerlap (SOV) (Zemla et al.,&nbsp;<span><a href="#pro70031-bib-0042" id="#pro70031-bib-0042_R_d1967767e1832" class="bibLink tab-link" data-tab="pane-pcw-references">1999</a></span>) scores evaluate the prediction accuracy based on the overlap between the predicted and actual segments instead of residues, which is more appropriate for segment prediction similar to secondary structure prediction. Four different SOV scores, SOV APR, SOV non-APR, SOV Overall, and SOV Average are used to evaluate the performance of predicting APRs in proteins. The formulation of these metrics is described below:
               <div class="inline-equation" id="pro70031-disp-0002"><span class="inline-equation__construct"><span><!--FIGURE-->
                        <span class="fallback__mathEquation" data-altimg="/cms/asset/9e88a56a-77fb-4d70-952f-9b9251d35ca0/pro70031-math-0011.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0011" display="block" location="graphic/pro70031-math-0011.png">
                           <semantics>
                              <mrow>
                                 <mfenced close="" open="{">
                                    <mtable columnalign="left">
                                       <mtr>
                                          <mtd>
                                             <mi>S</mi>
                                             <mfenced>
                                                <mi>i</mi>
                                             </mfenced>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfenced close="}" open="{">
                                                <mrow>
                                                   <mfenced>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>1</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>2</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                   </mfenced>
                                                   <mo>:</mo>
                                                   <mo></mo>
                                                   <msubsup>
                                                      <mi>s</mi>
                                                      <mn>2</mn>
                                                      <mi>i</mi>
                                                   </msubsup>
                                                </mrow>
                                                <mrow>
                                                   <msubsup>
                                                      <mi>s</mi>
                                                      <mn>1</mn>
                                                      <mi>i</mi>
                                                   </msubsup>
                                                   <mo></mo>
                                                   <msubsup>
                                                      <mi>s</mi>
                                                      <mn>2</mn>
                                                      <mi>i</mi>
                                                   </msubsup>
                                                   <mo></mo>
                                                   <mo></mo>
                                                </mrow>
                                             </mfenced>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <msup>
                                                <mi>S</mi>
                                                <mo></mo>
                                             </msup>
                                             <mfenced>
                                                <mi>i</mi>
                                             </mfenced>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfenced close="}" open="{">
                                                <mrow>
                                                   <mfenced>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>1</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>2</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                   </mfenced>
                                                   <mo>:</mo>
                                                   <mo></mo>
                                                   <msubsup>
                                                      <mi>s</mi>
                                                      <mn>2</mn>
                                                      <mi>i</mi>
                                                   </msubsup>
                                                </mrow>
                                                <mrow>
                                                   <msubsup>
                                                      <mi>s</mi>
                                                      <mn>1</mn>
                                                      <mi>i</mi>
                                                   </msubsup>
                                                   <mo></mo>
                                                   <msubsup>
                                                      <mi>s</mi>
                                                      <mn>2</mn>
                                                      <mi>i</mi>
                                                   </msubsup>
                                                   <mo linebreak="goodbreak">=</mo>
                                                   <mo></mo>
                                                </mrow>
                                             </mfenced>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi></mi>
                                             <mfenced>
                                                <msubsup>
                                                   <mi>s</mi>
                                                   <mn>1</mn>
                                                   <mi>i</mi>
                                                </msubsup>
                                                <msubsup>
                                                   <mi>s</mi>
                                                   <mn>2</mn>
                                                   <mi>i</mi>
                                                </msubsup>
                                             </mfenced>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mtable columnalign="left" displaystyle="true">
                                                <mtr>
                                                   <mtd>
                                                      <mi>min</mi>
                                                      <mrow>
                                                         <mo id="mo0-0" stretchy="true">{</mo>
                                                         <mfenced>
                                                            <mrow>
                                                               <mtext>maxov</mtext>
                                                               <mfenced>
                                                                  <msubsup>
                                                                     <mi>s</mi>
                                                                     <mn>1</mn>
                                                                     <mi>i</mi>
                                                                  </msubsup>
                                                                  <msubsup>
                                                                     <mi>s</mi>
                                                                     <mn>2</mn>
                                                                     <mi>i</mi>
                                                                  </msubsup>
                                                               </mfenced>
                                                               <mo linebreak="goodbreak"></mo>
                                                               <mtext>minov</mtext>
                                                               <mfenced>
                                                                  <msubsup>
                                                                     <mi>s</mi>
                                                                     <mn>1</mn>
                                                                     <mi>i</mi>
                                                                  </msubsup>
                                                                  <msubsup>
                                                                     <mi>s</mi>
                                                                     <mn>2</mn>
                                                                     <mi>i</mi>
                                                                  </msubsup>
                                                               </mfenced>
                                                            </mrow>
                                                         </mfenced>
                                                         <mo separator="true">,</mo>
                                                      </mrow>
                                                   </mtd>
                                                </mtr>
                                                <mtr>
                                                   <mtd>
                                                      <mrow>
                                                         <mtext>minov</mtext>
                                                         <mfenced>
                                                            <msubsup>
                                                               <mi>s</mi>
                                                               <mn>1</mn>
                                                               <mi>i</mi>
                                                            </msubsup>
                                                            <msubsup>
                                                               <mi>s</mi>
                                                               <mn>2</mn>
                                                               <mi>i</mi>
                                                            </msubsup>
                                                         </mfenced>
                                                         <mo separator="true">,</mo>
                                                         <mfenced close="" open="">
                                                            <mfrac>
                                                               <mrow>
                                                                  <mi>L</mi>
                                                                  <mfenced>
                                                                     <msubsup>
                                                                        <mi>s</mi>
                                                                        <mn>1</mn>
                                                                        <mi>i</mi>
                                                                     </msubsup>
                                                                  </mfenced>
                                                               </mrow>
                                                               <mn>2</mn>
                                                            </mfrac>
                                                         </mfenced>
                                                         <mo separator="true">,</mo>
                                                         <mfenced close="" open="">
                                                            <mfrac>
                                                               <mrow>
                                                                  <mi>L</mi>
                                                                  <mfenced>
                                                                     <msubsup>
                                                                        <mi>s</mi>
                                                                        <mn>2</mn>
                                                                        <mi>i</mi>
                                                                     </msubsup>
                                                                  </mfenced>
                                                               </mrow>
                                                               <mn>2</mn>
                                                            </mfrac>
                                                         </mfenced>
                                                         <mo stretchy="true">}</mo>
                                                      </mrow>
                                                   </mtd>
                                                </mtr>
                                             </mtable>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <msub>
                                                <mi>S</mi>
                                                <mi>i</mi>
                                             </msub>
                                             <mo linebreak="goodbreak">=</mo>
                                             <msub>
                                                <mo></mo>
                                                <mrow>
                                                   <mi>S</mi>
                                                   <mfenced>
                                                      <mi>i</mi>
                                                   </mfenced>
                                                </mrow>
                                             </msub>
                                             <mfrac>
                                                <mrow>
                                                   <mtext>minov</mtext>
                                                   <mfenced>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>1</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>2</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                   </mfenced>
                                                   <mo>+</mo>
                                                   <mi></mi>
                                                   <mfenced>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>1</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>2</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                   </mfenced>
                                                </mrow>
                                                <mrow>
                                                   <mtext>maxov</mtext>
                                                   <mfenced>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>1</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                      <msubsup>
                                                         <mi>s</mi>
                                                         <mn>2</mn>
                                                         <mi>i</mi>
                                                      </msubsup>
                                                   </mfenced>
                                                </mrow>
                                             </mfrac>
                                             <mo></mo>
                                             <mi>L</mi>
                                             <mfenced>
                                                <msubsup>
                                                   <mi>s</mi>
                                                   <mn>1</mn>
                                                   <mi>i</mi>
                                                </msubsup>
                                             </mfenced>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <msub>
                                                <mi>N</mi>
                                                <mi>i</mi>
                                             </msub>
                                             <mo linebreak="goodbreak">=</mo>
                                             <msub>
                                                <mo></mo>
                                                <mrow>
                                                   <mi>S</mi>
                                                   <mfenced>
                                                      <mi>i</mi>
                                                   </mfenced>
                                                </mrow>
                                             </msub>
                                             <mi>L</mi>
                                             <mfenced>
                                                <msubsup>
                                                   <mi>s</mi>
                                                   <mn>1</mn>
                                                   <mi>i</mi>
                                                </msubsup>
                                             </mfenced>
                                             <mo linebreak="goodbreak">+</mo>
                                             <msub>
                                                <mo></mo>
                                                <mrow>
                                                   <msup>
                                                      <mi>S</mi>
                                                      <mo></mo>
                                                   </msup>
                                                   <mfenced>
                                                      <mi>i</mi>
                                                   </mfenced>
                                                </mrow>
                                             </msub>
                                             <mi>L</mi>
                                             <mfenced>
                                                <msubsup>
                                                   <mi>s</mi>
                                                   <mn>1</mn>
                                                   <mi>i</mi>
                                                </msubsup>
                                             </mfenced>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi mathvariant="normal">SOV</mi>
                                             <mspace width="0.25em"></mspace>
                                             <mi mathvariant="normal">APR</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mi mathvariant="normal">SOV</mi>
                                             <mfenced>
                                                <mn>1</mn>
                                             </mfenced>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <msub>
                                                   <mi>S</mi>
                                                   <mn>1</mn>
                                                </msub>
                                                <msub>
                                                   <mi>N</mi>
                                                   <mn>1</mn>
                                                </msub>
                                             </mfrac>
                                             <mo linebreak="goodbreak"></mo>
                                             <mn>100</mn>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi mathvariant="normal">SOV</mi>
                                             <mspace width="0.25em"></mspace>
                                             <mtext>nonAPR</mtext>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mi mathvariant="normal">SOV</mi>
                                             <mfenced>
                                                <mn>0</mn>
                                             </mfenced>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <msub>
                                                   <mi>S</mi>
                                                   <mn>0</mn>
                                                </msub>
                                                <msub>
                                                   <mi>N</mi>
                                                   <mn>0</mn>
                                                </msub>
                                             </mfrac>
                                             <mo linebreak="goodbreak"></mo>
                                             <mn>100</mn>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi mathvariant="normal">SOV</mi>
                                             <mspace width="0.25em"></mspace>
                                             <mi mathvariant="normal">AVG</mi>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mrow>
                                                   <mi mathvariant="normal">SOV</mi>
                                                   <mfenced>
                                                      <mn>1</mn>
                                                   </mfenced>
                                                   <mo>+</mo>
                                                   <mi mathvariant="normal">SOV</mi>
                                                   <mfenced>
                                                      <mn>0</mn>
                                                   </mfenced>
                                                </mrow>
                                                <mn>2</mn>
                                             </mfrac>
                                          </mtd>
                                       </mtr>
                                       <mtr>
                                          <mtd>
                                             <mi mathvariant="normal">SOV</mi>
                                             <mspace width="0.25em"></mspace>
                                             <mtext>Overall</mtext>
                                             <mo linebreak="goodbreak">=</mo>
                                             <mfrac>
                                                <mrow>
                                                   <msub>
                                                      <mi>S</mi>
                                                      <mn>0</mn>
                                                   </msub>
                                                   <mo>+</mo>
                                                   <msub>
                                                      <mi>S</mi>
                                                      <mn>1</mn>
                                                   </msub>
                                                </mrow>
                                                <mrow>
                                                   <msub>
                                                      <mi>N</mi>
                                                      <mn>0</mn>
                                                   </msub>
                                                   <mo>+</mo>
                                                   <msub>
                                                      <mi>N</mi>
                                                      <mn>1</mn>
                                                   </msub>
                                                </mrow>
                                             </mfrac>
                                             <mo linebreak="goodbreak"></mo>
                                             <mn>100</mn>
                                             <mo>.</mo>
                                          </mtd>
                                       </mtr>
                                    </mtable>
                                 </mfenced>
                              </mrow>
                              <annotation encoding="application/x-tex">$$ \left\{\begin{array}{l}S(i)=\left\{\left({s}_1^i,{s}_2^i\right):\exists {s}_2^i,{s}_1^i\cap {s}_2^i\ne \varnothing \right\}\\ {}{S}^{\prime }(i)=\left\{\left({s}_1^i,{s}_2^i\right):\forall {s}_2^i,{s}_1^i\cap {s}_2^i=\varnothing \right\}\\ {}\delta \left({s}_1^i,{s}_2^i\right)={\displaystyle \begin{array}{l}\min \Big\{\left(\mathrm{maxov}\left({s}_1^i,{s}_2^i\right)-\mathrm{minov}\left({s}_1^i,{s}_2^i\right)\right),\\ {}\mathrm{minov}\left({s}_1^i,{s}_2^i\right),\left\lfloor \frac{L\left({s}_1^i\right)}{2}\right\rfloor, \left\lfloor \frac{L\left({s}_2^i\right)}{2}\right\rfloor \Big\}\end{array}}\\ {}{S}_i={\sum}_{S(i)}\frac{\mathrm{minov}\left({s}_1^i,{s}_2^i\right)+\delta \left({s}_1^i,{s}_2^i\right)}{\mathrm{maxov}\left({s}_1^i,{s}_2^i\right)}\bullet L\left({s}_1^i\right)\\ {}{N}_i={\sum}_{S(i)}L\left({s}_1^i\right)+{\sum}_{S^{\prime }(i)}L\left({s}_1^i\right)\\ {}\mathrm{SOV}\ \mathrm{APR}=\mathrm{SOV}(1)=\frac{S_1}{N_1}\times 100\\ {}\mathrm{SOV}\ \mathrm{nonAPR}=\mathrm{SOV}(0)=\frac{S_0}{N_0}\times 100\\ {}\mathrm{SOV}\ \mathrm{AVG}=\frac{\mathrm{SOV}(1)+\mathrm{SOV}(0)}{2}\\ {}\mathrm{SOV}\ \mathrm{Overall}=\frac{S_0+{S}_1}{N_0+{N}_1}\times 100.\end{array}\right. $$</annotation>
                           </semantics></math>
                        </span></span><span class="inline-equation__label"></span></div>where <span class="fallback__mathEquation" data-altimg="/cms/asset/d6a388bb-5dee-462a-aa3e-eb5afb2f238f/pro70031-math-0012.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0012" display="inline" location="graphic/pro70031-math-0012.png">
                  <semantics>
                     <mrow>
                        <msubsup>
                           <mi>s</mi>
                           <mn>1</mn>
                           <mi>i</mi>
                        </msubsup>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ {s}_1^i $$</annotation>
                  </semantics></math> is the segment of the ground truth label with state <span class="fallback__mathEquation" data-altimg="/cms/asset/7d890a2e-a076-43a1-98f6-4ba25545f3c8/pro70031-math-0013.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0013" display="inline" location="graphic/pro70031-math-0013.png">
                  <semantics>
                     <mrow>
                        <mi>i</mi>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ i $$</annotation>
                  </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/c779bec4-6e18-42e8-a47a-d30c206d47c4/pro70031-math-0014.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0014" display="inline" location="graphic/pro70031-math-0014.png">
                  <semantics>
                     <mrow>
                        <msubsup>
                           <mi>s</mi>
                           <mn>2</mn>
                           <mi>i</mi>
                        </msubsup>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ {s}_2^i $$</annotation>
                  </semantics></math> is the segment of the predicted label with state <span class="fallback__mathEquation" data-altimg="/cms/asset/b665ec6c-16ce-41d1-b358-19abba0a6e40/pro70031-math-0015.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0015" display="inline" location="graphic/pro70031-math-0015.png">
                  <semantics>
                     <mrow>
                        <mi>i</mi>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ i $$</annotation>
                  </semantics></math>. State <span class="fallback__mathEquation" data-altimg="/cms/asset/7409ac2f-5e7d-4ae9-bcb1-8f320c1b218a/pro70031-math-0016.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0016" display="inline" location="graphic/pro70031-math-0016.png">
                  <semantics>
                     <mrow>
                        <mi>i</mi>
                        <mo>=</mo>
                        <mn>1</mn>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ i=1 $$</annotation>
                  </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/d8d615ce-16f8-43b1-8ae0-4c9e7ee7ccae/pro70031-math-0017.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0017" display="inline" location="graphic/pro70031-math-0017.png">
                  <semantics>
                     <mrow>
                        <mi>i</mi>
                        <mo>=</mo>
                        <mn>0</mn>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ i=0 $$</annotation>
                  </semantics></math> denote the APR region and non-APR region, respectively. <span class="fallback__mathEquation" data-altimg="/cms/asset/42bfc40e-c948-4b21-9ed0-dd4b12d9071e/pro70031-math-0018.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0018" display="inline" location="graphic/pro70031-math-0018.png">
                  <semantics>
                     <mrow>
                        <mtext>maxov</mtext>
                        <mfenced close=")" open="(" separators=",">
                           <msubsup>
                              <mi>s</mi>
                              <mn>1</mn>
                              <mi>i</mi>
                           </msubsup>
                           <msubsup>
                              <mi>s</mi>
                              <mn>2</mn>
                              <mi>i</mi>
                           </msubsup>
                        </mfenced>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ \mathrm{maxov}\left({s}_1^i,{s}_2^i\right) $$</annotation>
                  </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/c57e59d9-d2a1-446d-8f67-8262714faaec/pro70031-math-0019.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0019" display="inline" location="graphic/pro70031-math-0019.png">
                  <semantics>
                     <mrow>
                        <mtext>minov</mtext>
                        <mfenced close=")" open="(" separators=",">
                           <msubsup>
                              <mi>s</mi>
                              <mn>1</mn>
                              <mi>i</mi>
                           </msubsup>
                           <msubsup>
                              <mi>s</mi>
                              <mn>2</mn>
                              <mi>i</mi>
                           </msubsup>
                        </mfenced>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ \mathrm{minov}\left({s}_1^i,{s}_2^i\right) $$</annotation>
                  </semantics></math> are the maximal and minimal overlap the length between <span class="fallback__mathEquation" data-altimg="/cms/asset/6e7e7f88-53d5-453e-849a-4d91cd997bb2/pro70031-math-0020.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0020" display="inline" location="graphic/pro70031-math-0020.png">
                  <semantics>
                     <mrow>
                        <msubsup>
                           <mi>s</mi>
                           <mn>1</mn>
                           <mi>i</mi>
                        </msubsup>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ {s}_1^i $$</annotation>
                  </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/4c9aaf16-81b9-4740-a080-501c492bbfdc/pro70031-math-0021.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0021" display="inline" location="graphic/pro70031-math-0021.png">
                  <semantics>
                     <mrow>
                        <msubsup>
                           <mi>s</mi>
                           <mn>2</mn>
                           <mi>i</mi>
                        </msubsup>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ {s}_2^i $$</annotation>
                  </semantics></math>. <span class="fallback__mathEquation" data-altimg="/cms/asset/392e3648-3192-41a1-b4ea-b5840365b81d/pro70031-math-0022.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0022" display="inline" location="graphic/pro70031-math-0022.png">
                  <semantics>
                     <mrow>
                        <mi>L</mi>
                        <mfenced close=")" open="(">
                           <msubsup>
                              <mi>s</mi>
                              <mn>1</mn>
                              <mi>i</mi>
                           </msubsup>
                        </mfenced>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ L\left({s}_1^i\right) $$</annotation>
                  </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/2b37e0d7-6f11-4714-82dc-e6777912ce3e/pro70031-math-0023.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0023" display="inline" location="graphic/pro70031-math-0023.png">
                  <semantics>
                     <mrow>
                        <mi>L</mi>
                        <mfenced close=")" open="(">
                           <msubsup>
                              <mi>s</mi>
                              <mn>2</mn>
                              <mi>i</mi>
                           </msubsup>
                        </mfenced>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ L\left({s}_2^i\right) $$</annotation>
                  </semantics></math> are the length of <span class="fallback__mathEquation" data-altimg="/cms/asset/95734652-1698-4ace-b065-91b9d737d670/pro70031-math-0024.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0024" display="inline" location="graphic/pro70031-math-0024.png">
                  <semantics>
                     <mrow>
                        <msubsup>
                           <mi>s</mi>
                           <mn>1</mn>
                           <mi>i</mi>
                        </msubsup>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ {s}_1^i $$</annotation>
                  </semantics></math> and <span class="fallback__mathEquation" data-altimg="/cms/asset/80d0955c-ea9c-40ef-8dec-15ffce9a6650/pro70031-math-0025.png"></span><math altimg="urn:x-wiley:09618368:media:pro70031:pro70031-math-0025" display="inline" location="graphic/pro70031-math-0025.png">
                  <semantics>
                     <mrow>
                        <msubsup>
                           <mi>s</mi>
                           <mn>2</mn>
                           <mi>i</mi>
                        </msubsup>
                     </mrow>
                     <annotation encoding="application/x-tex">$$ {s}_2^i $$</annotation>
                  </semantics></math>.</div>
            </section>
         </section>
      <section class="article-section__content" id="pro70031-sec-0018">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0018-title"> AUTHOR CONTRIBUTIONS</h2>
         
         <p><b>Wenjia He:</b> Conceptualization; methodology; writing  original draft; visualization; investigation; validation; data curation. <b>Xiaopeng Xu:</b> Validation; writing  review and editing; visualization; supervision. <b>Haoyang Li:</b> Writing  review and editing; visualization; validation. <b>Juexiao Zhou:</b> Writing  review and editing; visualization; validation. <b>Xin Gao:</b> Resources; project administration; writing  review and editing; funding acquisition; supervision.</p>
         </section>
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0016-title"> ACKNOWLEDGMENTS</h2>
         
         <p>The authors would like to thank Prof. David Brockwell for providing the experimental data of MEDI1912 from TPBLA. This publication is based upon work supported by the King Abdullah University of Science and Technology (KAUST) Office of Research Administration (ORA) under Award No REI/1/5234-01-01, REI/1/5414-01-01, REI/1/5289-01-01, REI/1/5404-01-01, REI/1/5992-01-01, URF/1/4663-01-01, Center of Excellence for Smart Health (KCSH), under award number 5932, and Center of Excellence on Generative AI, under award number 5940.</p>
         
         <ol></ol>
      </div>
      <section class="article-section__content" id="pro70031-sec-0017">
         
         <h2 class="article-section__title section__title section1" id="pro70031-sec-0017-title"> CONFLICT OF INTEREST STATEMENT</h2>
         
         <p>The authors declare no conflicts of interest.</p>
         </section>
      <section class="article-section article-section__supporting" data-suppl="/doi/suppl/10.1002/pro.70031?onlyLog=true">
         <div class="accordion article-accordion">
            <h2><a class="accordion__control" role="button" aria-controls="pro70031-supInfo-0001" aria-expanded="false">
                  <div tabindex="0" class="section__title" id="support-information-section"><span>Supporting Information</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div></a></h2>
            <div class="accordion__content" role="region" aria-labelledby="support-information-section" style="display: none;" id="pro70031-supInfo-0001" aria-hidden="false">
               <div class="support-info__table-wrapper article-table-content-wrapper">
                  
                  
                  <table class="support-info__table table article-section__table">
                     <thead>
                        <th id="article-filename" scope="col">Filename</th>
                        <th id="article-description" scope="col">Description</th>
                     </thead>
                     <tbody>
                        <tr id="pro70031-supitem-0001">
                           <td headers="article-filename"><a href="/action/downloadSupplement?doi=10.1002%2Fpro.70031&amp;file=pro70031-sup-0001-supinfo.pdf">pro70031-sup-0001-supinfo.pdf</a>PDF document, 249.9 KB</td>
                           <td headers="article-description"><p><b>Data S1.</b> Supporting Information.</p>
                              </td>
                        </tr>
                     </tbody>
                  </table>
                  <p class="disclaimer_text">Please note: The publisher is not responsible for the content or functionality of any supporting information supplied by the authors. Any queries (other than missing content) should be directed to the corresponding author for the article.</p>
               </div>
            </div>
         </div>
      </section>
      <section class="article-section article-section__references" id="article-references-section-1">
         <div class="accordion article-accordion">
            <h2>
               <div tabindex="0" role="button" class="accordion__control" aria-controls="idpro70031-bibl-0001" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fpro.70031&amp;pbContext=%3Bjournal%3Ajournal%3A1469896x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fpro.70031%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fpro.v34.2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3A1469896x&amp;widgetKey=ux3-publicationContent-widget_5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9_3067_4931_148546_en&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class="section__title" id="references-section-1">REFERENCES</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
            </h2>
            <div class="accordion__content" style="display: none;" role="region" aria-labelledby="references-section-1" id="pro70031-bibl-0001">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="pro70031-bib-0002"><span class="author">Brcenas O</span>, <span class="author">Kuriata A</span>, <span class="author">Zalewski M</span>, <span class="author">Iglesias V</span>, <span class="author">Pintado-Grima C</span>, <span class="author">Firlik G</span>, et al. <span class="articleTitle">Aggrescan4D: structure-informed analysis of pH-dependent protein aggregation</span>. <i>Nucleic Acids Res</i>. <span class="pubYear">2024</span>; <span class="vol">52</span>(<span class="citedIssue">W1</span>): <span class="pageFirst">W170</span><span class="pageLast">W175</span>. <a href="https://doi.org/10.1093/nar/gkae382" class="linkBehavior">https://doi.org/10.1093/nar/gkae382</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/nar/gkae382</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=38738618&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkae382&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A001219708800001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkae382&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D52%26publication_year%3D2024%26pages%3DW170-W175%26journal%3DNucleic%2BAcids%2BRes%26issue%3DW1%26author%3DO%2BB%25C3%25A1rcenas%26author%3DA%2BKuriata%26author%3DM%2BZalewski%26author%3DV%2BIglesias%26author%3DC%2BPintado%25E2%2580%2590Grima%26author%3DG%2BFirlik%26title%3DAggrescan4D%253A%2Bstructure%25E2%2580%2590informed%2Banalysis%2Bof%2BpH%25E2%2580%2590dependent%2Bprotein%2Baggregation&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fnar%2Fgkae382&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_2_1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkae382&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fnar%2Fgkae382&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fnar%252Fgkae382%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DO%26rft.aulast%3DB%25C3%25A1rcenas%26rft.aufirst%3DA%26rft.aulast%3DKuriata%26rft.aufirst%3DM%26rft.aulast%3DZalewski%26rft.aufirst%3DV%26rft.aulast%3DIglesias%26rft.aufirst%3DC%26rft.aulast%3DPintado%25E2%2580%2590Grima%26rft.aufirst%3DG%26rft.aulast%3DFirlik%26rft.atitle%3DAggrescan4D%253A%2520structure%25E2%2580%2590informed%2520analysis%2520of%2520pH%25E2%2580%2590dependent%2520protein%2520aggregation%26rft.jtitle%3DNucleic%2520Acids%2520Res%26rft.date%3D2024%26rft.volume%3D52%26rft.issue%3DW1%26rft.spage%3DW170%26rft.epage%3DW175" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0003"><span class="author">Chennamsetty N</span>, <span class="author">Voynov V</span>, <span class="author">Kayser V</span>, <span class="author">Helk B</span>, <span class="author">Trout BL</span>. <span class="articleTitle">Design of therapeutic proteins with enhanced stability</span>. <i>Proc Natl Acad Sci</i>. <span class="pubYear">2009</span>; <span class="vol">106</span>(<span class="citedIssue">29</span>): <span class="pageFirst">11937</span><span class="pageLast">11942</span>. <a href="https://doi.org/10.1073/pnas.0904191106" class="linkBehavior">https://doi.org/10.1073/pnas.0904191106</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1073/pnas.0904191106</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsF2ks70%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.0904191106&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=19571001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.0904191106&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000268178400024&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.0904191106&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D106%26publication_year%3D2009%26pages%3D11937-11942%26journal%3DProc%2BNatl%2BAcad%2BSci%26issue%3D29%26author%3DN%2BChennamsetty%26author%3DV%2BVoynov%26author%3DV%2BKayser%26author%3DB%2BHelk%26author%3DBL%2BTrout%26title%3DDesign%2Bof%2Btherapeutic%2Bproteins%2Bwith%2Benhanced%2Bstability&doi=10.1002%2Fpro.70031&doiOfLink=10.1073%2Fpnas.0904191106&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_3_1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0904191106&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1073%2Fpnas.0904191106&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1073%252Fpnas.0904191106%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DN%26rft.aulast%3DChennamsetty%26rft.aufirst%3DV%26rft.aulast%3DVoynov%26rft.aufirst%3DV%26rft.aulast%3DKayser%26rft.aufirst%3DB%26rft.aulast%3DHelk%26rft.aufirst%3DBL%26rft.aulast%3DTrout%26rft.atitle%3DDesign%2520of%2520therapeutic%2520proteins%2520with%2520enhanced%2520stability%26rft.jtitle%3DProc%2520Natl%2520Acad%2520Sci%26rft.date%3D2009%26rft.volume%3D106%26rft.issue%3D29%26rft.spage%3D11937%26rft.epage%3D11942" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0004"><span class="author">Conchillo-Sol O</span>, <span class="author">De Groot NS</span>, <span class="author">Avils FX</span>, <span class="author">Vendrell J</span>, <span class="author">Daura X</span>, <span class="author">Ventura S</span>. <span class="articleTitle">AGGRESCAN: a server for the prediction and evaluation of hot spots of aggregation in polypeptides</span>. <i>BMC Bioinform</i>. <span class="pubYear">2007</span>; <span class="vol">8</span>(<span class="citedIssue">1</span>):65. <a href="https://doi.org/10.1186/1471-2105-8-65" class="linkBehavior">https://doi.org/10.1186/1471-2105-8-65</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1186/1471-2105-8-65</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksFGqsrg%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1186%2F1471-2105-8-65&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=17324296&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1186%2F1471-2105-8-65&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=000245020900001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1186%2F1471-2105-8-65&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D8%26publication_year%3D2007%26journal%3DBMC%2BBioinform%26issue%3D1%26author%3DO%2BConchillo%25E2%2580%2590Sol%25C3%25A9%26author%3DNS%2BDe%2BGroot%26author%3DFX%2BAvil%25C3%25A9s%26author%3DJ%2BVendrell%26author%3DX%2BDaura%26author%3DS%2BVentura%26title%3DAGGRESCAN%253A%2Ba%2Bserver%2Bfor%2Bthe%2Bprediction%2Band%2Bevaluation%2Bof%2B%25E2%2580%259Chot%2Bspots%25E2%2580%259D%2Bof%2Baggregation%2Bin%2Bpolypeptides&doi=10.1002%2Fpro.70031&doiOfLink=10.1186%2F1471-2105-8-65&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_4_1&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-65&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1186%2F1471-2105-8-65&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1186%252F1471-2105-8-65%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DO%26rft.aulast%3DConchillo%25E2%2580%2590Sol%25C3%25A9%26rft.aufirst%3DNS%26rft.aulast%3DDe%2520Groot%26rft.aufirst%3DFX%26rft.aulast%3DAvil%25C3%25A9s%26rft.aufirst%3DJ%26rft.aulast%3DVendrell%26rft.aufirst%3DX%26rft.aulast%3DDaura%26rft.aufirst%3DS%26rft.aulast%3DVentura%26rft.atitle%3DAGGRESCAN%253A%2520a%2520server%2520for%2520the%2520prediction%2520and%2520evaluation%2520of%2520%25E2%2580%259Chot%2520spots%25E2%2580%259D%2520of%2520aggregation%2520in%2520polypeptides%26rft.jtitle%3DBMC%2520Bioinform%26rft.date%3D2007%26rft.volume%3D8%26rft.issue%3D1" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0005"><span class="author">Dobson J</span>, <span class="author">Kumar A</span>, <span class="author">Willis LF</span>, <span class="author">Tuma R</span>, <span class="author">Higazi DR</span>, <span class="author">Turner R</span>, et al. <span class="articleTitle">Inducing protein aggregation by extensional flow</span>. <i>Proc Natl Acad Sci</i>. <span class="pubYear">2017</span>; <span class="vol">114</span>(<span class="citedIssue">18</span>): <span class="pageFirst">4673</span><span class="pageLast">4678</span>. <a href="https://doi.org/10.1073/pnas.1702724114" class="linkBehavior">https://doi.org/10.1073/pnas.1702724114</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1073/pnas.1702724114</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGgtL4%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.1702724114&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=28416674&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.1702724114&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000400358000043&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.1702724114&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D114%26publication_year%3D2017%26pages%3D4673-4678%26journal%3DProc%2BNatl%2BAcad%2BSci%26issue%3D18%26author%3DJ%2BDobson%26author%3DA%2BKumar%26author%3DLF%2BWillis%26author%3DR%2BTuma%26author%3DDR%2BHigazi%26author%3DR%2BTurner%26title%3DInducing%2Bprotein%2Baggregation%2Bby%2Bextensional%2Bflow&doi=10.1002%2Fpro.70031&doiOfLink=10.1073%2Fpnas.1702724114&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_5_1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1702724114&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1073%2Fpnas.1702724114&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1073%252Fpnas.1702724114%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DJ%26rft.aulast%3DDobson%26rft.aufirst%3DA%26rft.aulast%3DKumar%26rft.aufirst%3DLF%26rft.aulast%3DWillis%26rft.aufirst%3DR%26rft.aulast%3DTuma%26rft.aufirst%3DDR%26rft.aulast%3DHigazi%26rft.aufirst%3DR%26rft.aulast%3DTurner%26rft.atitle%3DInducing%2520protein%2520aggregation%2520by%2520extensional%2520flow%26rft.jtitle%3DProc%2520Natl%2520Acad%2520Sci%26rft.date%3D2017%26rft.volume%3D114%26rft.issue%3D18%26rft.spage%3D4673%26rft.epage%3D4678" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0006"><span class="author">Ebo JS</span>, <span class="author">Saunders JC</span>, <span class="author">Devine PWA</span>, <span class="author">Gordon AM</span>, <span class="author">Warwick AS</span>, <span class="author">Schiffrin B</span>, et al. <span class="articleTitle">An in vivo platform to select and evolve aggregation-resistant proteins</span>. <i>Nat Commun</i>. <span class="pubYear">2020</span>; <span class="vol">11</span>(<span class="citedIssue">1</span>): <span class="pageFirst">1816</span>. <a href="https://doi.org/10.1038/s41467-020-15667-1" class="linkBehavior">https://doi.org/10.1038/s41467-020-15667-1</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/s41467-020-15667-1</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFOhtrs%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41467-020-15667-1&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=32286330&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41467-020-15667-1&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000528789000006&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41467-020-15667-1&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D11%26publication_year%3D2020%26pages%3D1816%26journal%3DNat%2BCommun%26issue%3D1%26author%3DJS%2BEbo%26author%3DJC%2BSaunders%26author%3DPWA%2BDevine%26author%3DAM%2BGordon%26author%3DAS%2BWarwick%26author%3DB%2BSchiffrin%26title%3DAn%2Bin%2Bvivo%2Bplatform%2Bto%2Bselect%2Band%2Bevolve%2Baggregation%25E2%2580%2590resistant%2Bproteins&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fs41467-020-15667-1&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_6_1&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-15667-1&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fs41467-020-15667-1&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fs41467-020-15667-1%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DJS%26rft.aulast%3DEbo%26rft.aufirst%3DJC%26rft.aulast%3DSaunders%26rft.aufirst%3DPWA%26rft.aulast%3DDevine%26rft.aufirst%3DAM%26rft.aulast%3DGordon%26rft.aufirst%3DAS%26rft.aulast%3DWarwick%26rft.aufirst%3DB%26rft.aulast%3DSchiffrin%26rft.atitle%3DAn%2520in%2520vivo%2520platform%2520to%2520select%2520and%2520evolve%2520aggregation%25E2%2580%2590resistant%2520proteins%26rft.jtitle%3DNat%2520Commun%26rft.date%3D2020%26rft.volume%3D11%26rft.issue%3D1%26rft.spage%3D1816" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0007"><span class="author">Fernandez-Escamilla A-M</span>, <span class="author">Rousseau F</span>, <span class="author">Schymkowitz J</span>, <span class="author">Serrano L</span>. <span class="articleTitle">Prediction of sequence-dependent and mutational effects on the aggregation of peptides and proteins</span>. <i>Nat Biotechnol</i>. <span class="pubYear">2004</span>; <span class="vol">22</span>(<span class="citedIssue">10</span>): <span class="pageFirst">1302</span><span class="pageLast">1306</span>. <a href="https://doi.org/10.1038/nbt1012" class="linkBehavior">https://doi.org/10.1038/nbt1012</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/nbt1012</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotFGqtb8%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnbt1012&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=15361882&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnbt1012&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000224326100033&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnbt1012&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D22%26publication_year%3D2004%26pages%3D1302-1306%26journal%3DNat%2BBiotechnol%26issue%3D10%26author%3DA%25E2%2580%2590M%2BFernandez%25E2%2580%2590Escamilla%26author%3DF%2BRousseau%26author%3DJ%2BSchymkowitz%26author%3DL%2BSerrano%26title%3DPrediction%2Bof%2Bsequence%25E2%2580%2590dependent%2Band%2Bmutational%2Beffects%2Bon%2Bthe%2Baggregation%2Bof%2Bpeptides%2Band%2Bproteins&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fnbt1012&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_7_1&amp;dbid=16384&amp;doi=10.1038%2Fnbt1012&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fnbt1012&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fnbt1012%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DA%25E2%2580%2590M%26rft.aulast%3DFernandez%25E2%2580%2590Escamilla%26rft.aufirst%3DF%26rft.aulast%3DRousseau%26rft.aufirst%3DJ%26rft.aulast%3DSchymkowitz%26rft.aufirst%3DL%26rft.aulast%3DSerrano%26rft.atitle%3DPrediction%2520of%2520sequence%25E2%2580%2590dependent%2520and%2520mutational%2520effects%2520on%2520the%2520aggregation%2520of%2520peptides%2520and%2520proteins%26rft.jtitle%3DNat%2520Biotechnol%26rft.date%3D2004%26rft.volume%3D22%26rft.issue%3D10%26rft.spage%3D1302%26rft.epage%3D1306" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0008"><span class="author">He K</span>, <span class="author">Zhang X</span>, <span class="author">Ren S</span>, <span class="author">Sun J</span>.  <span class="chapterTitle">Deep residual learning for image recognition</span>.  <span class="bookTitle">2016 IEEE conference on computer vision and pattern recognition (CVPR)</span>. <span class="publisherLocation">Las Vegas, NV</span>: IEEE; <span class="pubYear">2016</span>. p.  <span class="pageFirst">770</span><span class="pageLast">778</span>. <a href="https://doi.org/10.1109/CVPR.2016.90" class="linkBehavior">https://doi.org/10.1109/CVPR.2016.90</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1109/CVPR.2016.90</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26publication_year%3D2016%26pages%3D770-778%26author%3DK%2BHe%26author%3DX%2BZhang%26author%3DS%2BRen%26author%3DJ%2BSun%26title%3D2016%2BIEEE%2Bconference%2Bon%2Bcomputer%2Bvision%2Band%2Bpattern%2Brecognition%2B%2528CVPR%2529&doi=10.1002%2Fpro.70031&doiOfLink=10.1109%2FCVPR.2016.90&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_8_1&amp;dbid=16384&amp;doi=10.1109%2FCVPR.2016.90&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1109%2FCVPR.2016.90&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1109%252FCVPR.2016.90%26url_ver%3DZ39.88-2004%26rft.genre%3Dbook%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DK%26rft.aulast%3DHe%26rft.aufirst%3DX%26rft.aulast%3DZhang%26rft.aufirst%3DS%26rft.aulast%3DRen%26rft.aufirst%3DJ%26rft.aulast%3DSun%26rft.title%3DDeep%2520residual%2520learning%2520for%2520image%2520recognition%26rft.btitle%3D2016%2520IEEE%2520conference%2520on%2520computer%2520vision%2520and%2520pattern%2520recognition%2520%2528CVPR%2529%26rft.pub%3DIEEE%26rft.date%3D2016%26rft.spage%3D770%26rft.epage%3D778" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0009"><span class="author">Hinton G</span>, <span class="author">Roweis S</span>.  <span class="chapterTitle">Stochastic neighbor embedding</span>.  <span class="bookTitle">Advances in neural information processing systems 15</span>. <span class="publisherLocation">Cambridge</span>: MIT Press; <span class="pubYear">2002</span>.
                     <div class="extra-links getFTR"><span class="hidden data-doi"></span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26publication_year%3D2002%26author%3DG%2BHinton%26author%3DS%2BRoweis%26title%3DAdvances%2Bin%2Bneural%2Binformation%2Bprocessing%2Bsystems%2B15&doi=10.1002%2Fpro.70031&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_9_1&amp;dbid=16384&amp;doi=&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F%26url_ver%3DZ39.88-2004%26rft.genre%3Dbook%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DG%26rft.aulast%3DHinton%26rft.aufirst%3DS%26rft.aulast%3DRoweis%26rft.title%3DStochastic%2520neighbor%2520embedding%26rft.btitle%3DAdvances%2520in%2520neural%2520information%2520processing%2520systems%252015%26rft.pub%3DMIT%2520Press%26rft.date%3D2002" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0010"><span class="author">Housmans JAJ</span>, <span class="author">Wu G</span>, <span class="author">Schymkowitz J</span>, <span class="author">Rousseau F</span>. <span class="articleTitle">A guide to studying protein aggregation</span>. <i>FEBS J</i>. <span class="pubYear">2023</span>; <span class="vol">290</span>(<span class="citedIssue">3</span>): <span class="pageFirst">554</span><span class="pageLast">583</span>. <a href="https://doi.org/10.1111/febs.16312" class="linkBehavior">https://doi.org/10.1111/febs.16312</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1111/febs.16312</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3MXislejtrvO&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1111%2Ffebs.16312&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=34862849&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1111%2Ffebs.16312&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=000729568200001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1111%2Ffebs.16312&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D290%26publication_year%3D2023%26pages%3D554-583%26journal%3DFEBS%2BJ%26issue%3D3%26author%3DJAJ%2BHousmans%26author%3DG%2BWu%26author%3DJ%2BSchymkowitz%26author%3DF%2BRousseau%26title%3DA%2Bguide%2Bto%2Bstudying%2Bprotein%2Baggregation&doi=10.1002%2Fpro.70031&doiOfLink=10.1111%2Ffebs.16312&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_10_1&amp;dbid=16384&amp;doi=10.1111%2Ffebs.16312&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1111%2Ffebs.16312&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1111%252Ffebs.16312%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DJAJ%26rft.aulast%3DHousmans%26rft.aufirst%3DG%26rft.aulast%3DWu%26rft.aufirst%3DJ%26rft.aulast%3DSchymkowitz%26rft.aufirst%3DF%26rft.aulast%3DRousseau%26rft.atitle%3DA%2520guide%2520to%2520studying%2520protein%2520aggregation%26rft.jtitle%3DFEBS%2520J%26rft.date%3D2023%26rft.volume%3D290%26rft.issue%3D3%26rft.spage%3D554%26rft.epage%3D583" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0011"><span class="author">Huang Y</span>, <span class="author">Niu B</span>, <span class="author">Gao Y</span>, <span class="author">Fu L</span>, <span class="author">Li W</span>. <span class="articleTitle">CD-HIT suite: a web server for clustering and comparing biological sequences</span>. <i>Bioinformatics</i>. <span class="pubYear">2010</span>; <span class="vol">26</span>(<span class="citedIssue">5</span>): <span class="pageFirst">680</span><span class="pageLast">682</span>. <a href="https://doi.org/10.1093/bioinformatics/btq003" class="linkBehavior">https://doi.org/10.1093/bioinformatics/btq003</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/bioinformatics/btq003</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1WntLY%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fbioinformatics%2Fbtq003&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=20053844&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fbioinformatics%2Fbtq003&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000274973800015&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fbioinformatics%2Fbtq003&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D26%26publication_year%3D2010%26pages%3D680-682%26journal%3DBioinformatics%26issue%3D5%26author%3DY%2BHuang%26author%3DB%2BNiu%26author%3DY%2BGao%26author%3DL%2BFu%26author%3DW%2BLi%26title%3DCD%25E2%2580%2590HIT%2Bsuite%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bclustering%2Band%2Bcomparing%2Bbiological%2Bsequences&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fbioinformatics%2Fbtq003&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_11_1&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq003&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fbioinformatics%2Fbtq003&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fbioinformatics%252Fbtq003%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DY%26rft.aulast%3DHuang%26rft.aufirst%3DB%26rft.aulast%3DNiu%26rft.aufirst%3DY%26rft.aulast%3DGao%26rft.aufirst%3DL%26rft.aulast%3DFu%26rft.aufirst%3DW%26rft.aulast%3DLi%26rft.atitle%3DCD%25E2%2580%2590HIT%2520suite%253A%2520a%2520web%2520server%2520for%2520clustering%2520and%2520comparing%2520biological%2520sequences%26rft.jtitle%3DBioinformatics%26rft.date%3D2010%26rft.volume%3D26%26rft.issue%3D5%26rft.spage%3D680%26rft.epage%3D682" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0012"><span class="author">Iconomidou VA</span>, <span class="author">Chryssikos GD</span>, <span class="author">Gionis V</span>, <span class="author">Galanis AS</span>, <span class="author">Cordopatis P</span>, <span class="author">Hoenger A</span>, et al. <span class="articleTitle">Amyloid fibril formation propensity is inherent into the hexapeptide tandemly repeating sequence of the central domain of silkmoth chorion proteins of the A-family</span>. <i>J Struct Biol</i>. <span class="pubYear">2006</span>; <span class="vol">156</span>(<span class="citedIssue">3</span>): <span class="pageFirst">480</span><span class="pageLast">488</span>. <a href="https://doi.org/10.1016/j.jsb.2006.08.011" class="linkBehavior">https://doi.org/10.1016/j.jsb.2006.08.011</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1016/j.jsb.2006.08.011</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CgsbzE&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jsb.2006.08.011&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=17056273&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jsb.2006.08.011&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000242650700010&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jsb.2006.08.011&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D156%26publication_year%3D2006%26pages%3D480-488%26journal%3DJ%2BStruct%2BBiol%26issue%3D3%26author%3DVA%2BIconomidou%26author%3DGD%2BChryssikos%26author%3DV%2BGionis%26author%3DAS%2BGalanis%26author%3DP%2BCordopatis%26author%3DA%2BHoenger%26title%3DAmyloid%2Bfibril%2Bformation%2Bpropensity%2Bis%2Binherent%2Binto%2Bthe%2Bhexapeptide%2Btandemly%2Brepeating%2Bsequence%2Bof%2Bthe%2Bcentral%2Bdomain%2Bof%2Bsilkmoth%2Bchorion%2Bproteins%2Bof%2Bthe%2BA%25E2%2580%2590family&doi=10.1002%2Fpro.70031&doiOfLink=10.1016%2Fj.jsb.2006.08.011&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_12_1&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2006.08.011&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1016%2Fj.jsb.2006.08.011&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1016%252Fj.jsb.2006.08.011%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DVA%26rft.aulast%3DIconomidou%26rft.aufirst%3DGD%26rft.aulast%3DChryssikos%26rft.aufirst%3DV%26rft.aulast%3DGionis%26rft.aufirst%3DAS%26rft.aulast%3DGalanis%26rft.aufirst%3DP%26rft.aulast%3DCordopatis%26rft.aufirst%3DA%26rft.aulast%3DHoenger%26rft.atitle%3DAmyloid%2520fibril%2520formation%2520propensity%2520is%2520inherent%2520into%2520the%2520hexapeptide%2520tandemly%2520repeating%2520sequence%2520of%2520the%2520central%2520domain%2520of%2520silkmoth%2520chorion%2520proteins%2520of%2520the%2520A%25E2%2580%2590family%26rft.jtitle%3DJ%2520Struct%2520Biol%26rft.date%3D2006%26rft.volume%3D156%26rft.issue%3D3%26rft.spage%3D480%26rft.epage%3D488" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0013"><span class="author">Janssen K</span>, <span class="author">Claes F</span>, <span class="author">Van De Velde D</span>, <span class="author">Wehbi VL</span>, <span class="author">Houben B</span>, <span class="author">Lampi Y</span>, et al. <span class="articleTitle">Exploiting the intrinsic misfolding propensity of the KRAS oncoprotein</span>. <i>Proc Natl Acad Sci</i>. <span class="pubYear">2023</span>; <span class="vol">120</span>(<span class="citedIssue">9</span>):e2214921120. <a href="https://doi.org/10.1073/pnas.2214921120" class="linkBehavior">https://doi.org/10.1073/pnas.2214921120</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1073/pnas.2214921120</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3sXlvVGqtLc%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.2214921120&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=36812200&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.2214921120&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000943142000009&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.2214921120&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D120%26publication_year%3D2023%26journal%3DProc%2BNatl%2BAcad%2BSci%26issue%3D9%26author%3DK%2BJanssen%26author%3DF%2BClaes%26author%3DD%2BVan%2BDe%2BVelde%26author%3DVL%2BWehbi%26author%3DB%2BHouben%26author%3DY%2BLampi%26title%3DExploiting%2Bthe%2Bintrinsic%2Bmisfolding%2Bpropensity%2Bof%2Bthe%2BKRAS%2Boncoprotein&doi=10.1002%2Fpro.70031&doiOfLink=10.1073%2Fpnas.2214921120&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_13_1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2214921120&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1073%2Fpnas.2214921120&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1073%252Fpnas.2214921120%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DK%26rft.aulast%3DJanssen%26rft.aufirst%3DF%26rft.aulast%3DClaes%26rft.aufirst%3DD%26rft.aulast%3DVan%2520De%2520Velde%26rft.aufirst%3DVL%26rft.aulast%3DWehbi%26rft.aufirst%3DB%26rft.aulast%3DHouben%26rft.aufirst%3DY%26rft.aulast%3DLampi%26rft.atitle%3DExploiting%2520the%2520intrinsic%2520misfolding%2520propensity%2520of%2520the%2520KRAS%2520oncoprotein%26rft.jtitle%3DProc%2520Natl%2520Acad%2520Sci%26rft.date%3D2023%26rft.volume%3D120%26rft.issue%3D9" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0014"><span class="author">Kahn SE</span>, <span class="author">Andrikopoulos S</span>, <span class="author">Verchere CB</span>. <span class="articleTitle">Islet amyloid: a long-recognized but underappreciated pathological feature of type 2 diabetes</span>. <i>Diabetes</i>. <span class="pubYear">1999</span>; <span class="vol">48</span>(<span class="citedIssue">2</span>): <span class="pageFirst">241</span><span class="pageLast">253</span>. <a href="https://doi.org/10.2337/diabetes.48.2.241" class="linkBehavior">https://doi.org/10.2337/diabetes.48.2.241</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.2337/diabetes.48.2.241</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADyaK1MXpsFKguw%253D%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.2337%2Fdiabetes.48.2.241&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=10334297&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.2337%2Fdiabetes.48.2.241&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000078253600001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.2337%2Fdiabetes.48.2.241&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D48%26publication_year%3D1999%26pages%3D241-253%26journal%3DDiabetes%26issue%3D2%26author%3DSE%2BKahn%26author%3DS%2BAndrikopoulos%26author%3DCB%2BVerchere%26title%3DIslet%2Bamyloid%253A%2Ba%2Blong%25E2%2580%2590recognized%2Bbut%2Bunderappreciated%2Bpathological%2Bfeature%2Bof%2Btype%2B2%2Bdiabetes&doi=10.1002%2Fpro.70031&doiOfLink=10.2337%2Fdiabetes.48.2.241&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_14_1&amp;dbid=16384&amp;doi=10.2337%2Fdiabetes.48.2.241&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.2337%2Fdiabetes.48.2.241&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.2337%252Fdiabetes.48.2.241%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DSE%26rft.aulast%3DKahn%26rft.aufirst%3DS%26rft.aulast%3DAndrikopoulos%26rft.aufirst%3DCB%26rft.aulast%3DVerchere%26rft.atitle%3DIslet%2520amyloid%253A%2520a%2520long%25E2%2580%2590recognized%2520but%2520underappreciated%2520pathological%2520feature%2520of%2520type%25202%2520diabetes%26rft.jtitle%3DDiabetes%26rft.date%3D1999%26rft.volume%3D48%26rft.issue%3D2%26rft.spage%3D241%26rft.epage%3D253" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0015"><span class="author">Kawashima S</span>. <span class="articleTitle">AAindex: amino acid index database</span>. <i>Nucleic Acids Res</i>. <span class="pubYear">2000</span>; <span class="vol">28</span>(<span class="citedIssue">1</span>):374. <a href="https://doi.org/10.1093/nar/28.1.374" class="linkBehavior">https://doi.org/10.1093/nar/28.1.374</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/nar/28.1.374</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKju7s%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2F28.1.374&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=10592278&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2F28.1.374&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000084896300112&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2F28.1.374&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D28%26publication_year%3D2000%26journal%3DNucleic%2BAcids%2BRes%26issue%3D1%26author%3DS%2BKawashima%26title%3DAAindex%253A%2Bamino%2Bacid%2Bindex%2Bdatabase&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fnar%2F28.1.374&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_15_1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.374&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fnar%2F28.1.374&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fnar%252F28.1.374%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DS%26rft.aulast%3DKawashima%26rft.atitle%3DAAindex%253A%2520amino%2520acid%2520index%2520database%26rft.jtitle%3DNucleic%2520Acids%2520Res%26rft.date%3D2000%26rft.volume%3D28%26rft.issue%3D1" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0016"><span class="author">Koo EH</span>, <span class="author">Lansbury PT</span>, <span class="author">Kelly JW</span>. <span class="articleTitle">Amyloid diseases: abnormal protein aggregation in neurodegeneration</span>. <i>Proc Natl Acad Sci</i>. <span class="pubYear">1999</span>; <span class="vol">96</span>(<span class="citedIssue">18</span>): <span class="pageFirst">9989</span><span class="pageLast">9990</span>. <a href="https://doi.org/10.1073/pnas.96.18.9989" class="linkBehavior">https://doi.org/10.1073/pnas.96.18.9989</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1073/pnas.96.18.9989</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehurk%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.96.18.9989&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=10468546&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.96.18.9989&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000082424100010&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.96.18.9989&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D96%26publication_year%3D1999%26pages%3D9989-9990%26journal%3DProc%2BNatl%2BAcad%2BSci%26issue%3D18%26author%3DEH%2BKoo%26author%3DPT%2BLansbury%26author%3DJW%2BKelly%26title%3DAmyloid%2Bdiseases%253A%2Babnormal%2Bprotein%2Baggregation%2Bin%2Bneurodegeneration&doi=10.1002%2Fpro.70031&doiOfLink=10.1073%2Fpnas.96.18.9989&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_16_1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9989&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1073%2Fpnas.96.18.9989&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1073%252Fpnas.96.18.9989%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DEH%26rft.aulast%3DKoo%26rft.aufirst%3DPT%26rft.aulast%3DLansbury%26rft.aufirst%3DJW%26rft.aulast%3DKelly%26rft.atitle%3DAmyloid%2520diseases%253A%2520abnormal%2520protein%2520aggregation%2520in%2520neurodegeneration%26rft.jtitle%3DProc%2520Natl%2520Acad%2520Sci%26rft.date%3D1999%26rft.volume%3D96%26rft.issue%3D18%26rft.spage%3D9989%26rft.epage%3D9990" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0017"><span class="author">Lin Z</span>, <span class="author">Akin H</span>, <span class="author">Rao R</span>, <span class="author">Hie B</span>, <span class="author">Zhu Z</span>, <span class="author">Lu W</span>, et al. <span class="articleTitle">Evolutionary-scale prediction of atomic-level protein structure with a language model</span>. <i>Science</i>. <span class="pubYear">2023</span>; <span class="vol">379</span>(<span class="citedIssue">6637</span>): <span class="pageFirst">1123</span><span class="pageLast">1130</span>. <a href="https://doi.org/10.1126/science.ade2574" class="linkBehavior">https://doi.org/10.1126/science.ade2574</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1126/science.ade2574</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3sXls1ertrk%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1126%2Fscience.ade2574&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=36927031&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1126%2Fscience.ade2574&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000959028800006&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1126%2Fscience.ade2574&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D379%26publication_year%3D2023%26pages%3D1123-1130%26journal%3DScience%26issue%3D6637%26author%3DZ%2BLin%26author%3DH%2BAkin%26author%3DR%2BRao%26author%3DB%2BHie%26author%3DZ%2BZhu%26author%3DW%2BLu%26title%3DEvolutionary%25E2%2580%2590scale%2Bprediction%2Bof%2Batomic%25E2%2580%2590level%2Bprotein%2Bstructure%2Bwith%2Ba%2Blanguage%2Bmodel&doi=10.1002%2Fpro.70031&doiOfLink=10.1126%2Fscience.ade2574&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_17_1&amp;dbid=16384&amp;doi=10.1126%2Fscience.ade2574&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1126%2Fscience.ade2574&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1126%252Fscience.ade2574%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DZ%26rft.aulast%3DLin%26rft.aufirst%3DH%26rft.aulast%3DAkin%26rft.aufirst%3DR%26rft.aulast%3DRao%26rft.aufirst%3DB%26rft.aulast%3DHie%26rft.aufirst%3DZ%26rft.aulast%3DZhu%26rft.aufirst%3DW%26rft.aulast%3DLu%26rft.atitle%3DEvolutionary%25E2%2580%2590scale%2520prediction%2520of%2520atomic%25E2%2580%2590level%2520protein%2520structure%2520with%2520a%2520language%2520model%26rft.jtitle%3DScience%26rft.date%3D2023%26rft.volume%3D379%26rft.issue%3D6637%26rft.spage%3D1123%26rft.epage%3D1130" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0018"><span class="author">Louros N</span>, <span class="author">Konstantoulea K</span>, <span class="author">De Vleeschouwer M</span>, <span class="author">Ramakers M</span>, <span class="author">Schymkowitz J</span>, <span class="author">Rousseau F</span>. <span class="articleTitle">WALTZ-DB 2.0: an updated database containing structural information of experimentally determined amyloid-forming peptides</span>. <i>Nucleic Acids Res</i>. <span class="pubYear">2020</span>; <span class="vol">48</span>(<span class="citedIssue">D1</span>): <span class="pageFirst">D389</span><span class="pageLast">D393</span>. <a href="https://doi.org/10.1093/nar/gkz758" class="linkBehavior">https://doi.org/10.1093/nar/gkz758</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/nar/gkz758</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWltrjE&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkz758&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=31504823&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkz758&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000525956700054&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkz758&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D48%26publication_year%3D2020%26pages%3DD389-D393%26journal%3DNucleic%2BAcids%2BRes%26issue%3DD1%26author%3DN%2BLouros%26author%3DK%2BKonstantoulea%26author%3DM%2BDe%2BVleeschouwer%26author%3DM%2BRamakers%26author%3DJ%2BSchymkowitz%26author%3DF%2BRousseau%26title%3DWALTZ%25E2%2580%2590DB%2B2.0%253A%2Ban%2Bupdated%2Bdatabase%2Bcontaining%2Bstructural%2Binformation%2Bof%2Bexperimentally%2Bdetermined%2Bamyloid%25E2%2580%2590forming%2Bpeptides&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fnar%2Fgkz758&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_18_1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz758&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fnar%2Fgkz758&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fnar%252Fgkz758%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DN%26rft.aulast%3DLouros%26rft.aufirst%3DK%26rft.aulast%3DKonstantoulea%26rft.aufirst%3DM%26rft.aulast%3DDe%2520Vleeschouwer%26rft.aufirst%3DM%26rft.aulast%3DRamakers%26rft.aufirst%3DJ%26rft.aulast%3DSchymkowitz%26rft.aufirst%3DF%26rft.aulast%3DRousseau%26rft.atitle%3DWALTZ%25E2%2580%2590DB%25202.0%253A%2520an%2520updated%2520database%2520containing%2520structural%2520information%2520of%2520experimentally%2520determined%2520amyloid%25E2%2580%2590forming%2520peptides%26rft.jtitle%3DNucleic%2520Acids%2520Res%26rft.date%3D2020%26rft.volume%3D48%26rft.issue%3DD1%26rft.spage%3DD389%26rft.epage%3DD393" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0019"><span class="author">Louros N</span>, <span class="author">Orlando G</span>, <span class="author">De Vleeschouwer M</span>, <span class="author">Rousseau F</span>, <span class="author">Schymkowitz J</span>. <span class="articleTitle">Structure-based machine-guided mapping of amyloid sequence space reveals uncharted sequence clusters with higher solubilities</span>. <i>Nat Commun</i>. <span class="pubYear">2020</span>; <span class="vol">11</span>(<span class="citedIssue">1</span>): <span class="pageFirst">3314</span>. <a href="https://doi.org/10.1038/s41467-020-17207-3" class="linkBehavior">https://doi.org/10.1038/s41467-020-17207-3</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/s41467-020-17207-3</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlGms77N&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41467-020-17207-3&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=32620861&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41467-020-17207-3&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000546623500014&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41467-020-17207-3&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D11%26publication_year%3D2020%26pages%3D3314%26journal%3DNat%2BCommun%26issue%3D1%26author%3DN%2BLouros%26author%3DG%2BOrlando%26author%3DM%2BDe%2BVleeschouwer%26author%3DF%2BRousseau%26author%3DJ%2BSchymkowitz%26title%3DStructure%25E2%2580%2590based%2Bmachine%25E2%2580%2590guided%2Bmapping%2Bof%2Bamyloid%2Bsequence%2Bspace%2Breveals%2Buncharted%2Bsequence%2Bclusters%2Bwith%2Bhigher%2Bsolubilities&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fs41467-020-17207-3&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_19_1&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-17207-3&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fs41467-020-17207-3&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fs41467-020-17207-3%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DN%26rft.aulast%3DLouros%26rft.aufirst%3DG%26rft.aulast%3DOrlando%26rft.aufirst%3DM%26rft.aulast%3DDe%2520Vleeschouwer%26rft.aufirst%3DF%26rft.aulast%3DRousseau%26rft.aufirst%3DJ%26rft.aulast%3DSchymkowitz%26rft.atitle%3DStructure%25E2%2580%2590based%2520machine%25E2%2580%2590guided%2520mapping%2520of%2520amyloid%2520sequence%2520space%2520reveals%2520uncharted%2520sequence%2520clusters%2520with%2520higher%2520solubilities%26rft.jtitle%3DNat%2520Commun%26rft.date%3D2020%26rft.volume%3D11%26rft.issue%3D1%26rft.spage%3D3314" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0020"><span class="author">Lowe D</span>, <span class="author">Dudgeon K</span>, <span class="author">Rouet R</span>, <span class="author">Schofield P</span>, <span class="author">Jermutus L</span>, <span class="author">Christ D</span>. <span class="articleTitle">Aggregation, stability, and formulation of human antibody therapeutics</span>. <i>Adv Protein Chem Struct Biol</i>. <span class="pubYear">2011</span>; <span class="vol">84</span>: <span class="pageFirst">41</span><span class="pageLast">61</span>. <a href="https://doi.org/10.1016/B978-0-12-386483-3.00004-5" class="linkBehavior">https://doi.org/10.1016/B978-0-12-386483-3.00004-5</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1016/B978-0-12-386483-3.00004-5</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12rt73J&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2FB978-0-12-386483-3.00004-5&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=21846562&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2FB978-0-12-386483-3.00004-5&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000294690100002&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2FB978-0-12-386483-3.00004-5&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D84%26publication_year%3D2011%26pages%3D41-61%26journal%3DAdv%2BProtein%2BChem%2BStruct%2BBiol%26author%3DD%2BLowe%26author%3DK%2BDudgeon%26author%3DR%2BRouet%26author%3DP%2BSchofield%26author%3DL%2BJermutus%26author%3DD%2BChrist%26title%3DAggregation%252C%2Bstability%252C%2Band%2Bformulation%2Bof%2Bhuman%2Bantibody%2Btherapeutics&doi=10.1002%2Fpro.70031&doiOfLink=10.1016%2FB978-0-12-386483-3.00004-5&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_20_1&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-386483-3.00004-5&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1016%2FB978-0-12-386483-3.00004-5&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1016%252FB978-0-12-386483-3.00004-5%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DD%26rft.aulast%3DLowe%26rft.aufirst%3DK%26rft.aulast%3DDudgeon%26rft.aufirst%3DR%26rft.aulast%3DRouet%26rft.aufirst%3DP%26rft.aulast%3DSchofield%26rft.aufirst%3DL%26rft.aulast%3DJermutus%26rft.aufirst%3DD%26rft.aulast%3DChrist%26rft.atitle%3DAggregation%252C%2520stability%252C%2520and%2520formulation%2520of%2520human%2520antibody%2520therapeutics%26rft.jtitle%3DAdv%2520Protein%2520Chem%2520Struct%2520Biol%26rft.date%3D2011%26rft.volume%3D84%26rft.spage%3D41%26rft.epage%3D61" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0021"><span class="author">Lundahl MLE</span>, <span class="author">Fogli S</span>, <span class="author">Colavita PE</span>, <span class="author">Scanlan EM</span>. <span class="articleTitle">Aggregation of protein therapeutics enhances their immunogenicity: causes and mitigation strategies</span>. <i>RSC Chem Biol</i>. <span class="pubYear">2021</span>; <span class="vol">2</span>(<span class="citedIssue">4</span>): <span class="pageFirst">1004</span><span class="pageLast">1020</span>. <a href="https://doi.org/10.1039/D1CB00067E" class="linkBehavior">https://doi.org/10.1039/D1CB00067E</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1039/D1CB00067E</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtVajsrrO&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FD1CB00067E&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=34458822&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FD1CB00067E&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000649050400001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FD1CB00067E&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D2%26publication_year%3D2021%26pages%3D1004-1020%26journal%3DRSC%2BChem%2BBiol%26issue%3D4%26author%3DMLE%2BLundahl%26author%3DS%2BFogli%26author%3DPE%2BColavita%26author%3DEM%2BScanlan%26title%3DAggregation%2Bof%2Bprotein%2Btherapeutics%2Benhances%2Btheir%2Bimmunogenicity%253A%2Bcauses%2Band%2Bmitigation%2Bstrategies&doi=10.1002%2Fpro.70031&doiOfLink=10.1039%2FD1CB00067E&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_21_1&amp;dbid=16384&amp;doi=10.1039%2FD1CB00067E&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1039%2FD1CB00067E&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1039%252FD1CB00067E%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DMLE%26rft.aulast%3DLundahl%26rft.aufirst%3DS%26rft.aulast%3DFogli%26rft.aufirst%3DPE%26rft.aulast%3DColavita%26rft.aufirst%3DEM%26rft.aulast%3DScanlan%26rft.atitle%3DAggregation%2520of%2520protein%2520therapeutics%2520enhances%2520their%2520immunogenicity%253A%2520causes%2520and%2520mitigation%2520strategies%26rft.jtitle%3DRSC%2520Chem%2520Biol%26rft.date%3D2021%26rft.volume%3D2%26rft.issue%3D4%26rft.spage%3D1004%26rft.epage%3D1020" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0022"><span class="author">Matson JB</span>, <span class="author">Stupp SI</span>. <span class="articleTitle">Self-assembling peptide scaffolds for regenerative medicine</span>. <i>Chem Commun</i>. <span class="pubYear">2012</span>; <span class="vol">48</span>(<span class="citedIssue">1</span>): <span class="pageFirst">26</span><span class="pageLast">33</span>. <a href="https://doi.org/10.1039/C1CC15551B" class="linkBehavior">https://doi.org/10.1039/C1CC15551B</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1039/C1CC15551B</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFerur7N&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FC1CC15551B&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=22080255&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FC1CC15551B&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000297513300005&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FC1CC15551B&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D48%26publication_year%3D2012%26pages%3D26-33%26journal%3DChem%2BCommun%26issue%3D1%26author%3DJB%2BMatson%26author%3DSI%2BStupp%26title%3DSelf%25E2%2580%2590assembling%2Bpeptide%2Bscaffolds%2Bfor%2Bregenerative%2Bmedicine&doi=10.1002%2Fpro.70031&doiOfLink=10.1039%2FC1CC15551B&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_22_1&amp;dbid=16384&amp;doi=10.1039%2FC1CC15551B&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1039%2FC1CC15551B&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1039%252FC1CC15551B%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DJB%26rft.aulast%3DMatson%26rft.aufirst%3DSI%26rft.aulast%3DStupp%26rft.atitle%3DSelf%25E2%2580%2590assembling%2520peptide%2520scaffolds%2520for%2520regenerative%2520medicine%26rft.jtitle%3DChem%2520Commun%26rft.date%3D2012%26rft.volume%3D48%26rft.issue%3D1%26rft.spage%3D26%26rft.epage%3D33" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0023"><span class="author">Maurer-Stroh S</span>, <span class="author">Debulpaep M</span>, <span class="author">Kuemmerer N</span>, <span class="author">De La Paz ML</span>, <span class="author">Martins IC</span>, <span class="author">Reumers J</span>, et al. <span class="articleTitle">Exploring the sequence determinants of amyloid structure using position-specific scoring matrices</span>. <i>Nat Methods</i>. <span class="pubYear">2010</span>; <span class="vol">7</span>(<span class="citedIssue">3</span>): <span class="pageFirst">237</span><span class="pageLast">242</span>. <a href="https://doi.org/10.1038/nmeth.1432" class="linkBehavior">https://doi.org/10.1038/nmeth.1432</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/nmeth.1432</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGmsbw%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnmeth.1432&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=20154676&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnmeth.1432&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000275058200027&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnmeth.1432&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D7%26publication_year%3D2010%26pages%3D237-242%26journal%3DNat%2BMethods%26issue%3D3%26author%3DS%2BMaurer%25E2%2580%2590Stroh%26author%3DM%2BDebulpaep%26author%3DN%2BKuemmerer%26author%3DML%2BDe%2BLa%2BPaz%26author%3DIC%2BMartins%26author%3DJ%2BReumers%26title%3DExploring%2Bthe%2Bsequence%2Bdeterminants%2Bof%2Bamyloid%2Bstructure%2Busing%2Bposition%25E2%2580%2590specific%2Bscoring%2Bmatrices&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fnmeth.1432&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_23_1&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1432&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fnmeth.1432&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fnmeth.1432%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DS%26rft.aulast%3DMaurer%25E2%2580%2590Stroh%26rft.aufirst%3DM%26rft.aulast%3DDebulpaep%26rft.aufirst%3DN%26rft.aulast%3DKuemmerer%26rft.aufirst%3DML%26rft.aulast%3DDe%2520La%2520Paz%26rft.aufirst%3DIC%26rft.aulast%3DMartins%26rft.aufirst%3DJ%26rft.aulast%3DReumers%26rft.atitle%3DExploring%2520the%2520sequence%2520determinants%2520of%2520amyloid%2520structure%2520using%2520position%25E2%2580%2590specific%2520scoring%2520matrices%26rft.jtitle%3DNat%2520Methods%26rft.date%3D2010%26rft.volume%3D7%26rft.issue%3D3%26rft.spage%3D237%26rft.epage%3D242" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0024"><span class="author">Mirdita M</span>, <span class="author">Schtze K</span>, <span class="author">Moriwaki Y</span>, <span class="author">Heo L</span>, <span class="author">Ovchinnikov S</span>, <span class="author">Steinegger M</span>. <span class="articleTitle">ColabFold: making protein folding accessible to all</span>. <i>Nat Methods</i>. <span class="pubYear">2022</span>; <span class="vol">19</span>(<span class="citedIssue">6</span>): <span class="pageFirst">679</span><span class="pageLast">682</span>. <a href="https://doi.org/10.1038/s41592-022-01488-1" class="linkBehavior">https://doi.org/10.1038/s41592-022-01488-1</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/s41592-022-01488-1</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB38XhsVWjsLfL&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41592-022-01488-1&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=35637307&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41592-022-01488-1&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000802918900005&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41592-022-01488-1&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D19%26publication_year%3D2022%26pages%3D679-682%26journal%3DNat%2BMethods%26issue%3D6%26author%3DM%2BMirdita%26author%3DK%2BSch%25C3%25BCtze%26author%3DY%2BMoriwaki%26author%3DL%2BHeo%26author%3DS%2BOvchinnikov%26author%3DM%2BSteinegger%26title%3DColabFold%253A%2Bmaking%2Bprotein%2Bfolding%2Baccessible%2Bto%2Ball&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fs41592-022-01488-1&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_24_1&amp;dbid=16384&amp;doi=10.1038%2Fs41592-022-01488-1&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fs41592-022-01488-1&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fs41592-022-01488-1%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DM%26rft.aulast%3DMirdita%26rft.aufirst%3DK%26rft.aulast%3DSch%25C3%25BCtze%26rft.aufirst%3DY%26rft.aulast%3DMoriwaki%26rft.aufirst%3DL%26rft.aulast%3DHeo%26rft.aufirst%3DS%26rft.aulast%3DOvchinnikov%26rft.aufirst%3DM%26rft.aulast%3DSteinegger%26rft.atitle%3DColabFold%253A%2520making%2520protein%2520folding%2520accessible%2520to%2520all%26rft.jtitle%3DNat%2520Methods%26rft.date%3D2022%26rft.volume%3D19%26rft.issue%3D6%26rft.spage%3D679%26rft.epage%3D682" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0025"><span class="author">Moran SD</span>, <span class="author">Decatur SM</span>, <span class="author">Zanni MT</span>. <span class="articleTitle">Structural and sequence analysis of the human D-crystallin amyloid fibril core using 2D IR spectroscopy, segmental <sup>13</sup>C labeling, and mass spectrometry</span>. <i>J Am Chem Soc</i>. <span class="pubYear">2012</span>; <span class="vol">134</span>(<span class="citedIssue">44</span>): <span class="pageFirst">18410</span><span class="pageLast">18416</span>. <a href="https://doi.org/10.1021/ja307898g" class="linkBehavior">https://doi.org/10.1021/ja307898g</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1021/ja307898g</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs7zO&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1021%2Fja307898g&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=23082813&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1021%2Fja307898g&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D134%26publication_year%3D2012%26pages%3D18410-18416%26journal%3DJ%2BAm%2BChem%2BSoc%26issue%3D44%26author%3DSD%2BMoran%26author%3DSM%2BDecatur%26author%3DMT%2BZanni%26title%3DStructural%2Band%2Bsequence%2Banalysis%2Bof%2Bthe%2Bhuman%2B%25CE%25B3D%25E2%2580%2590crystallin%2Bamyloid%2Bfibril%2Bcore%2Busing%2B2D%2BIR%2Bspectroscopy%252C%2Bsegmental%2B13C%2Blabeling%252C%2Band%2Bmass%2Bspectrometry&doi=10.1002%2Fpro.70031&doiOfLink=10.1021%2Fja307898g&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_25_1&amp;dbid=16384&amp;doi=10.1021%2Fja307898g&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1021%2Fja307898g&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1021%252Fja307898g%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DSD%26rft.aulast%3DMoran%26rft.aufirst%3DSM%26rft.aulast%3DDecatur%26rft.aufirst%3DMT%26rft.aulast%3DZanni%26rft.atitle%3DStructural%2520and%2520sequence%2520analysis%2520of%2520the%2520human%2520%25CE%25B3D%25E2%2580%2590crystallin%2520amyloid%2520fibril%2520core%2520using%25202D%2520IR%2520spectroscopy%252C%2520segmental%252013C%2520labeling%252C%2520and%2520mass%2520spectrometry%26rft.jtitle%3DJ%2520Am%2520Chem%2520Soc%26rft.date%3D2012%26rft.volume%3D134%26rft.issue%3D44%26rft.spage%3D18410%26rft.epage%3D18416" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0026"><span class="author">Moran SD</span>, <span class="author">Woys AM</span>, <span class="author">Buchanan LE</span>, <span class="author">Bixby E</span>, <span class="author">Decatur SM</span>, <span class="author">Zanni MT</span>. <span class="articleTitle">Two-dimensional IR spectroscopy and segmental <sup>13</sup>C labeling reveals the domain structure of human D-crystallin amyloid fibrils</span>. <i>Proc Natl Acad Sci</i>. <span class="pubYear">2012</span>; <span class="vol">109</span>(<span class="citedIssue">9</span>): <span class="pageFirst">3329</span><span class="pageLast">3334</span>. <a href="https://doi.org/10.1073/pnas.1117704109" class="linkBehavior">https://doi.org/10.1073/pnas.1117704109</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1073/pnas.1117704109</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVyhtrk%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.1117704109&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=22328156&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.1117704109&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D109%26publication_year%3D2012%26pages%3D3329-3334%26journal%3DProc%2BNatl%2BAcad%2BSci%26issue%3D9%26author%3DSD%2BMoran%26author%3DAM%2BWoys%26author%3DLE%2BBuchanan%26author%3DE%2BBixby%26author%3DSM%2BDecatur%26author%3DMT%2BZanni%26title%3DTwo%25E2%2580%2590dimensional%2BIR%2Bspectroscopy%2Band%2Bsegmental%2B13C%2Blabeling%2Breveals%2Bthe%2Bdomain%2Bstructure%2Bof%2Bhuman%2B%25CE%25B3D%25E2%2580%2590crystallin%2Bamyloid%2Bfibrils&doi=10.1002%2Fpro.70031&doiOfLink=10.1073%2Fpnas.1117704109&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_26_1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1117704109&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1073%2Fpnas.1117704109&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1073%252Fpnas.1117704109%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DSD%26rft.aulast%3DMoran%26rft.aufirst%3DAM%26rft.aulast%3DWoys%26rft.aufirst%3DLE%26rft.aulast%3DBuchanan%26rft.aufirst%3DE%26rft.aulast%3DBixby%26rft.aufirst%3DSM%26rft.aulast%3DDecatur%26rft.aufirst%3DMT%26rft.aulast%3DZanni%26rft.atitle%3DTwo%25E2%2580%2590dimensional%2520IR%2520spectroscopy%2520and%2520segmental%252013C%2520labeling%2520reveals%2520the%2520domain%2520structure%2520of%2520human%2520%25CE%25B3D%25E2%2580%2590crystallin%2520amyloid%2520fibrils%26rft.jtitle%3DProc%2520Natl%2520Acad%2520Sci%26rft.date%3D2012%26rft.volume%3D109%26rft.issue%3D9%26rft.spage%3D3329%26rft.epage%3D3334" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0027"><span class="author">Perchiacca JM</span>, <span class="author">Tessier PM</span>. <span class="articleTitle">Engineering aggregation-resistant antibodies</span>. <i>Annu Rev Chem Biomol Eng</i>. <span class="pubYear">2012</span>; <span class="vol">3</span>(<span class="citedIssue">1</span>): <span class="pageFirst">263</span><span class="pageLast">286</span>. <a href="https://doi.org/10.1146/annurev-chembioeng-062011-081052" class="linkBehavior">https://doi.org/10.1146/annurev-chembioeng-062011-081052</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1146/annurev-chembioeng-062011-081052</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GmtbfO&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1146%2Fannurev-chembioeng-062011-081052&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=22468604&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1146%2Fannurev-chembioeng-062011-081052&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D3%26publication_year%3D2012%26pages%3D263-286%26journal%3DAnnu%2BRev%2BChem%2BBiomol%2BEng%26issue%3D1%26author%3DJM%2BPerchiacca%26author%3DPM%2BTessier%26title%3DEngineering%2Baggregation%25E2%2580%2590resistant%2Bantibodies&doi=10.1002%2Fpro.70031&doiOfLink=10.1146%2Fannurev-chembioeng-062011-081052&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_27_1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-chembioeng-062011-081052&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1146%2Fannurev-chembioeng-062011-081052&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1146%252Fannurev-chembioeng-062011-081052%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DJM%26rft.aulast%3DPerchiacca%26rft.aufirst%3DPM%26rft.aulast%3DTessier%26rft.atitle%3DEngineering%2520aggregation%25E2%2580%2590resistant%2520antibodies%26rft.jtitle%3DAnnu%2520Rev%2520Chem%2520Biomol%2520Eng%26rft.date%3D2012%26rft.volume%3D3%26rft.issue%3D1%26rft.spage%3D263%26rft.epage%3D286" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0028"><span class="author">Planas-Iglesias J</span>, <span class="author">Borko S</span>, <span class="author">Swiatkowski J</span>, <span class="author">Elias M</span>, <span class="author">Havlasek M</span>, <span class="author">Salamon O</span>, et al. <span class="articleTitle">AggreProt: a web server for predicting and engineering aggregation prone regions in proteins</span>. <i>Nucleic Acids Res</i>. <span class="pubYear">2024</span>; <span class="vol">52</span>(<span class="citedIssue">W1</span>): <span class="pageFirst">W159</span><span class="pageLast">W169</span>. <a href="https://doi.org/10.1093/nar/gkae420" class="linkBehavior">https://doi.org/10.1093/nar/gkae420</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/nar/gkae420</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=38801076&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkae420&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A001233323700001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkae420&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D52%26publication_year%3D2024%26pages%3DW159-W169%26journal%3DNucleic%2BAcids%2BRes%26issue%3DW1%26author%3DJ%2BPlanas%25E2%2580%2590Iglesias%26author%3DS%2BBorko%26author%3DJ%2BSwiatkowski%26author%3DM%2BElias%26author%3DM%2BHavlasek%26author%3DO%2BSalamon%26title%3DAggreProt%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bpredicting%2Band%2Bengineering%2Baggregation%2Bprone%2Bregions%2Bin%2Bproteins&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fnar%2Fgkae420&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_28_1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkae420&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fnar%2Fgkae420&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fnar%252Fgkae420%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DJ%26rft.aulast%3DPlanas%25E2%2580%2590Iglesias%26rft.aufirst%3DS%26rft.aulast%3DBorko%26rft.aufirst%3DJ%26rft.aulast%3DSwiatkowski%26rft.aufirst%3DM%26rft.aulast%3DElias%26rft.aufirst%3DM%26rft.aulast%3DHavlasek%26rft.aufirst%3DO%26rft.aulast%3DSalamon%26rft.atitle%3DAggreProt%253A%2520a%2520web%2520server%2520for%2520predicting%2520and%2520engineering%2520aggregation%2520prone%2520regions%2520in%2520proteins%26rft.jtitle%3DNucleic%2520Acids%2520Res%26rft.date%3D2024%26rft.volume%3D52%26rft.issue%3DW1%26rft.spage%3DW159%26rft.epage%3DW169" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0029"><span class="author">Prabakaran R</span>, <span class="author">Rawat P</span>, <span class="author">Kumar S</span>, <span class="author">Michael Gromiha M</span>. <span class="articleTitle">ANuPP: a versatile tool to predict aggregation nucleating regions in peptides and proteins</span>. <i>J Mol Biol</i>. <span class="pubYear">2021</span>; <span class="vol">433</span>(<span class="citedIssue">11</span>):166707. <a href="https://doi.org/10.1016/j.jmb.2020.11.006" class="linkBehavior">https://doi.org/10.1016/j.jmb.2020.11.006</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1016/j.jmb.2020.11.006</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVSgurfE&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2020.11.006&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=33972019&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2020.11.006&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000648520800008&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2020.11.006&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D433%26publication_year%3D2021%26journal%3DJ%2BMol%2BBiol%26issue%3D11%26author%3DR%2BPrabakaran%26author%3DP%2BRawat%26author%3DS%2BKumar%26author%3DM%2BMichael%2BGromiha%26title%3DANuPP%253A%2Ba%2Bversatile%2Btool%2Bto%2Bpredict%2Baggregation%2Bnucleating%2Bregions%2Bin%2Bpeptides%2Band%2Bproteins&doi=10.1002%2Fpro.70031&doiOfLink=10.1016%2Fj.jmb.2020.11.006&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_29_1&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2020.11.006&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1016%2Fj.jmb.2020.11.006&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1016%252Fj.jmb.2020.11.006%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DR%26rft.aulast%3DPrabakaran%26rft.aufirst%3DP%26rft.aulast%3DRawat%26rft.aufirst%3DS%26rft.aulast%3DKumar%26rft.aufirst%3DM%26rft.aulast%3DMichael%2520Gromiha%26rft.atitle%3DANuPP%253A%2520a%2520versatile%2520tool%2520to%2520predict%2520aggregation%2520nucleating%2520regions%2520in%2520peptides%2520and%2520proteins%26rft.jtitle%3DJ%2520Mol%2520Biol%26rft.date%3D2021%26rft.volume%3D433%26rft.issue%3D11" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0030"><span class="author">Rahban M</span>, <span class="author">Ahmad F</span>, <span class="author">Piatyszek MA</span>, <span class="author">Haertl T</span>, <span class="author">Saso L</span>, <span class="author">Saboury AA</span>. <span class="articleTitle">Stabilization challenges and aggregation in protein-based therapeutics in the pharmaceutical industry</span>. <i>RSC Adv</i>. <span class="pubYear">2023</span>; <span class="vol">13</span>(<span class="citedIssue">51</span>): <span class="pageFirst">35947</span><span class="pageLast">35963</span>. <a href="https://doi.org/10.1039/D3RA06476J" class="linkBehavior">https://doi.org/10.1039/D3RA06476J</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1039/D3RA06476J</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3sXisFOitbbO&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FD3RA06476J&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=38090079&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FD3RA06476J&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A001119536500001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2FD3RA06476J&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D13%26publication_year%3D2023%26pages%3D35947-35963%26journal%3DRSC%2BAdv%26issue%3D51%26author%3DM%2BRahban%26author%3DF%2BAhmad%26author%3DMA%2BPiatyszek%26author%3DT%2BHaertl%25C3%25A9%26author%3DL%2BSaso%26author%3DAA%2BSaboury%26title%3DStabilization%2Bchallenges%2Band%2Baggregation%2Bin%2Bprotein%25E2%2580%2590based%2Btherapeutics%2Bin%2Bthe%2Bpharmaceutical%2Bindustry&doi=10.1002%2Fpro.70031&doiOfLink=10.1039%2FD3RA06476J&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_30_1&amp;dbid=16384&amp;doi=10.1039%2FD3RA06476J&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1039%2FD3RA06476J&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1039%252FD3RA06476J%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DM%26rft.aulast%3DRahban%26rft.aufirst%3DF%26rft.aulast%3DAhmad%26rft.aufirst%3DMA%26rft.aulast%3DPiatyszek%26rft.aufirst%3DT%26rft.aulast%3DHaertl%25C3%25A9%26rft.aufirst%3DL%26rft.aulast%3DSaso%26rft.aufirst%3DAA%26rft.aulast%3DSaboury%26rft.atitle%3DStabilization%2520challenges%2520and%2520aggregation%2520in%2520protein%25E2%2580%2590based%2520therapeutics%2520in%2520the%2520pharmaceutical%2520industry%26rft.jtitle%3DRSC%2520Adv%26rft.date%3D2023%26rft.volume%3D13%26rft.issue%3D51%26rft.spage%3D35947%26rft.epage%3D35963" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0031"><span class="author">Rawat P</span>, <span class="author">Prabakaran R</span>, <span class="author">Sakthivel R</span>, <span class="author">Mary Thangakani A</span>, <span class="author">Kumar S</span>, <span class="author">Gromiha MM</span>. <span class="articleTitle">CPAD 2.0: a repository of curated experimental data on aggregating proteins and peptides</span>. <i>Amyloid</i>. <span class="pubYear">2020</span>; <span class="vol">27</span>(<span class="citedIssue">2</span>): <span class="pageFirst">128</span><span class="pageLast">133</span>. <a href="https://doi.org/10.1080/13506129.2020.1715363" class="linkBehavior">https://doi.org/10.1080/13506129.2020.1715363</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1080/13506129.2020.1715363</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWisbc%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1080%2F13506129.2020.1715363&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=31979981&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1080%2F13506129.2020.1715363&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000509171500001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1080%2F13506129.2020.1715363&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D27%26publication_year%3D2020%26pages%3D128-133%26journal%3DAmyloid%26issue%3D2%26author%3DP%2BRawat%26author%3DR%2BPrabakaran%26author%3DR%2BSakthivel%26author%3DA%2BMary%2BThangakani%26author%3DS%2BKumar%26author%3DMM%2BGromiha%26title%3DCPAD%2B2.0%253A%2Ba%2Brepository%2Bof%2Bcurated%2Bexperimental%2Bdata%2Bon%2Baggregating%2Bproteins%2Band%2Bpeptides&doi=10.1002%2Fpro.70031&doiOfLink=10.1080%2F13506129.2020.1715363&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_31_1&amp;dbid=16384&amp;doi=10.1080%2F13506129.2020.1715363&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1080%2F13506129.2020.1715363&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1080%252F13506129.2020.1715363%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DP%26rft.aulast%3DRawat%26rft.aufirst%3DR%26rft.aulast%3DPrabakaran%26rft.aufirst%3DR%26rft.aulast%3DSakthivel%26rft.aufirst%3DA%26rft.aulast%3DMary%2520Thangakani%26rft.aufirst%3DS%26rft.aulast%3DKumar%26rft.aufirst%3DMM%26rft.aulast%3DGromiha%26rft.atitle%3DCPAD%25202.0%253A%2520a%2520repository%2520of%2520curated%2520experimental%2520data%2520on%2520aggregating%2520proteins%2520and%2520peptides%26rft.jtitle%3DAmyloid%26rft.date%3D2020%26rft.volume%3D27%26rft.issue%3D2%26rft.spage%3D128%26rft.epage%3D133" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0032"><span class="author">Ren C</span>, <span class="author">Wen X</span>, <span class="author">Mencius J</span>, <span class="author">Quan S</span>. <span class="articleTitle">An enzyme-based biosensor for monitoring and engineering protein stability in vivo</span>. <i>Proc Natl Acad Sci</i>. <span class="pubYear">2021</span>; <span class="vol">118</span>(<span class="citedIssue">13</span>):e2101618118. <a href="https://doi.org/10.1073/pnas.2101618118" class="linkBehavior">https://doi.org/10.1073/pnas.2101618118</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1073/pnas.2101618118</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnvVamtLw%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.2101618118&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=33753520&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.2101618118&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000637394200090&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1073%2Fpnas.2101618118&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D118%26publication_year%3D2021%26journal%3DProc%2BNatl%2BAcad%2BSci%26issue%3D13%26author%3DC%2BRen%26author%3DX%2BWen%26author%3DJ%2BMencius%26author%3DS%2BQuan%26title%3DAn%2Benzyme%25E2%2580%2590based%2Bbiosensor%2Bfor%2Bmonitoring%2Band%2Bengineering%2Bprotein%2Bstability%2Bin%2Bvivo&doi=10.1002%2Fpro.70031&doiOfLink=10.1073%2Fpnas.2101618118&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_32_1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2101618118&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1073%2Fpnas.2101618118&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1073%252Fpnas.2101618118%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DC%26rft.aulast%3DRen%26rft.aufirst%3DX%26rft.aulast%3DWen%26rft.aufirst%3DJ%26rft.aulast%3DMencius%26rft.aufirst%3DS%26rft.aulast%3DQuan%26rft.atitle%3DAn%2520enzyme%25E2%2580%2590based%2520biosensor%2520for%2520monitoring%2520and%2520engineering%2520protein%2520stability%2520in%2520vivo%26rft.jtitle%3DProc%2520Natl%2520Acad%2520Sci%26rft.date%3D2021%26rft.volume%3D118%26rft.issue%3D13" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0033"><span class="author">Ross CA</span>, <span class="author">Poirier MA</span>. <span class="articleTitle">Protein aggregation and neurodegenerative disease</span>. <i>Nat Med</i>. <span class="pubYear">2004</span>; <span class="vol">10</span>(<span class="citedIssue">S7</span>): <span class="pageFirst">S10</span><span class="pageLast">S17</span>. <a href="https://doi.org/10.1038/nm1066" class="linkBehavior">https://doi.org/10.1038/nm1066</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/nm1066</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=15272267&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnm1066&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=000223202100003&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fnm1066&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D10%26publication_year%3D2004%26pages%3DS10-S17%26journal%3DNat%2BMed%26issue%3DS7%26author%3DCA%2BRoss%26author%3DMA%2BPoirier%26title%3DProtein%2Baggregation%2Band%2Bneurodegenerative%2Bdisease&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fnm1066&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_33_1&amp;dbid=16384&amp;doi=10.1038%2Fnm1066&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fnm1066&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fnm1066%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DCA%26rft.aulast%3DRoss%26rft.aufirst%3DMA%26rft.aulast%3DPoirier%26rft.atitle%3DProtein%2520aggregation%2520and%2520neurodegenerative%2520disease%26rft.jtitle%3DNat%2520Med%26rft.date%3D2004%26rft.volume%3D10%26rft.issue%3DS7%26rft.spage%3DS10%26rft.epage%3DS17" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0034"><span class="author">Sankar K</span>, <span class="author">Krystek SR</span>, <span class="author">Carl SM</span>, <span class="author">Day T</span>, <span class="author">Maier JKX</span>. <span class="articleTitle">AggScore: prediction of aggregation-prone regions in proteins based on the distribution of surface patches</span>. <i>Proteins: Struct, Funct, Bioinf</i>. <span class="pubYear">2018</span>; <span class="vol">86</span>(<span class="citedIssue">11</span>): <span class="pageFirst">1147</span><span class="pageLast">1156</span>. <a href="https://doi.org/10.1002/prot.25594" class="linkBehavior">https://doi.org/10.1002/prot.25594</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1002/prot.25594</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSitLfP&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1002%2Fprot.25594&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=30168197&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1002%2Fprot.25594&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000451044000003&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1002%2Fprot.25594&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D86%26publication_year%3D2018%26pages%3D1147-1156%26journal%3DProteins%253A%2BStruct%252C%2BFunct%252C%2BBioinf%26issue%3D11%26author%3DK%2BSankar%26author%3DSR%2BKrystek%26author%3DSM%2BCarl%26author%3DT%2BDay%26author%3DJKX%2BMaier%26title%3DAggScore%253A%2Bprediction%2Bof%2Baggregation%25E2%2580%2590prone%2Bregions%2Bin%2Bproteins%2Bbased%2Bon%2Bthe%2Bdistribution%2Bof%2Bsurface%2Bpatches&doi=10.1002%2Fpro.70031&doiOfLink=10.1002%2Fprot.25594&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_34_1&amp;dbid=16384&amp;doi=10.1002%2Fprot.25594&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1002%2Fprot.25594&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1002%252Fprot.25594%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DK%26rft.aulast%3DSankar%26rft.aufirst%3DSR%26rft.aulast%3DKrystek%26rft.aufirst%3DSM%26rft.aulast%3DCarl%26rft.aufirst%3DT%26rft.aulast%3DDay%26rft.aufirst%3DJKX%26rft.aulast%3DMaier%26rft.atitle%3DAggScore%253A%2520prediction%2520of%2520aggregation%25E2%2580%2590prone%2520regions%2520in%2520proteins%2520based%2520on%2520the%2520distribution%2520of%2520surface%2520patches%26rft.jtitle%3DProteins%253A%2520Struct%252C%2520Funct%252C%2520Bioinf%26rft.date%3D2018%26rft.volume%3D86%26rft.issue%3D11%26rft.spage%3D1147%26rft.epage%3D1156" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0035"><span class="author">Sormanni P</span>, <span class="author">Aprile FA</span>, <span class="author">Vendruscolo M</span>. <span class="articleTitle">The CamSol method of rational design of protein mutants with enhanced solubility</span>. <i>J Mol Biol</i>. <span class="pubYear">2015</span>; <span class="vol">427</span>(<span class="citedIssue">2</span>): <span class="pageFirst">478</span><span class="pageLast">490</span>. <a href="https://doi.org/10.1016/j.jmb.2014.09.026" class="linkBehavior">https://doi.org/10.1016/j.jmb.2014.09.026</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1016/j.jmb.2014.09.026</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOktbfN&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2014.09.026&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=25451785&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2014.09.026&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000348888200024&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2014.09.026&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D427%26publication_year%3D2015%26pages%3D478-490%26journal%3DJ%2BMol%2BBiol%26issue%3D2%26author%3DP%2BSormanni%26author%3DFA%2BAprile%26author%3DM%2BVendruscolo%26title%3DThe%2BCamSol%2Bmethod%2Bof%2Brational%2Bdesign%2Bof%2Bprotein%2Bmutants%2Bwith%2Benhanced%2Bsolubility&doi=10.1002%2Fpro.70031&doiOfLink=10.1016%2Fj.jmb.2014.09.026&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_35_1&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.09.026&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1016%2Fj.jmb.2014.09.026&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1016%252Fj.jmb.2014.09.026%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DP%26rft.aulast%3DSormanni%26rft.aufirst%3DFA%26rft.aulast%3DAprile%26rft.aufirst%3DM%26rft.aulast%3DVendruscolo%26rft.atitle%3DThe%2520CamSol%2520method%2520of%2520rational%2520design%2520of%2520protein%2520mutants%2520with%2520enhanced%2520solubility%26rft.jtitle%3DJ%2520Mol%2520Biol%26rft.date%3D2015%26rft.volume%3D427%26rft.issue%3D2%26rft.spage%3D478%26rft.epage%3D490" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0036"><span class="author">Taud H</span>, <span class="author">Mas JF</span>.  <span class="chapterTitle">Multilayer perceptron (MLP)</span>. In:  <span class="editor">MT Camacho Olmedo</span>,  <span class="editor">M Paegelow</span>,  <span class="editor">J-F Mas</span>,  <span class="editor">F Escobar</span>, editors.  <span class="bookTitle">Geomatic approaches for modeling land change scenarios</span>. <span class="publisherLocation">Cham</span>: Springer International Publishing; <span class="pubYear">2018</span>. p.  <span class="pageFirst">451</span><span class="pageLast">455</span>. <a href="https://doi.org/10.1007/978-3-319-60801-3_27" class="linkBehavior">https://doi.org/10.1007/978-3-319-60801-3_27</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1007/978-3-319-60801-3_27</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26publication_year%3D2018%26pages%3D451-455%26author%3DH%2BTaud%26author%3DJF%2BMas%26title%3DGeomatic%2Bapproaches%2Bfor%2Bmodeling%2Bland%2Bchange%2Bscenarios&doi=10.1002%2Fpro.70031&doiOfLink=10.1007%2F978-3-319-60801-3_27&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_36_1&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-60801-3_27&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1007%2F978-3-319-60801-3_27&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1007%252F978-3-319-60801-3_27%26url_ver%3DZ39.88-2004%26rft.genre%3Dbook%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DH%26rft.aulast%3DTaud%26rft.aufirst%3DJF%26rft.aulast%3DMas%26rft.title%3DMultilayer%2520perceptron%2520%2528MLP%2529%26rft.aufirst%3DMT%26rft.aulast%3DCamacho%2520Olmedo%26rft.aufirst%3DM%26rft.aulast%3DPaegelow%26rft.aufirst%3DJ%25E2%2580%2590F%26rft.aulast%3DMas%26rft.aufirst%3DF%26rft.aulast%3DEscobar%26rft.btitle%3DGeomatic%2520approaches%2520for%2520modeling%2520land%2520change%2520scenarios%26rft.pub%3DSpringer%2520International%2520Publishing%26rft.date%3D2018%26rft.spage%3D451%26rft.epage%3D455" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0037"><span class="author">Tunyasuvunakool K</span>, <span class="author">Adler J</span>, <span class="author">Wu Z</span>, <span class="author">Green T</span>, <span class="author">Zielinski M</span>, <span class="author">dek A</span>, et al. <span class="articleTitle">Highly accurate protein structure prediction for the human proteome</span>. <i>Nature</i>. <span class="pubYear">2021</span>; <span class="vol">596</span>(<span class="citedIssue">7873</span>): <span class="pageFirst">590</span><span class="pageLast">596</span>. <a href="https://doi.org/10.1038/s41586-021-03828-1" class="linkBehavior">https://doi.org/10.1038/s41586-021-03828-1</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1038/s41586-021-03828-1</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhvVaktrrO&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41586-021-03828-1&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=34293799&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41586-021-03828-1&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000686116900001&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1038%2Fs41586-021-03828-1&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D596%26publication_year%3D2021%26pages%3D590-596%26journal%3DNature%26issue%3D7873%26author%3DK%2BTunyasuvunakool%26author%3DJ%2BAdler%26author%3DZ%2BWu%26author%3DT%2BGreen%26author%3DM%2BZielinski%26author%3DA%2B%25C5%25BD%25C3%25ADdek%26title%3DHighly%2Baccurate%2Bprotein%2Bstructure%2Bprediction%2Bfor%2Bthe%2Bhuman%2Bproteome&doi=10.1002%2Fpro.70031&doiOfLink=10.1038%2Fs41586-021-03828-1&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_37_1&amp;dbid=16384&amp;doi=10.1038%2Fs41586-021-03828-1&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1038%2Fs41586-021-03828-1&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1038%252Fs41586-021-03828-1%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DK%26rft.aulast%3DTunyasuvunakool%26rft.aufirst%3DJ%26rft.aulast%3DAdler%26rft.aufirst%3DZ%26rft.aulast%3DWu%26rft.aufirst%3DT%26rft.aulast%3DGreen%26rft.aufirst%3DM%26rft.aulast%3DZielinski%26rft.aufirst%3DA%26rft.aulast%3D%25C5%25BD%25C3%25ADdek%26rft.atitle%3DHighly%2520accurate%2520protein%2520structure%2520prediction%2520for%2520the%2520human%2520proteome%26rft.jtitle%3DNature%26rft.date%3D2021%26rft.volume%3D596%26rft.issue%3D7873%26rft.spage%3D590%26rft.epage%3D596" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0038"><span class="author">Varadi M</span>, <span class="author">De Baets G</span>, <span class="author">Vranken WF</span>, <span class="author">Tompa P</span>, <span class="author">Pancsa R</span>. <span class="articleTitle">AmyPro: a database of proteins with validated amyloidogenic regions</span>. <i>Nucleic Acids Res</i>. <span class="pubYear">2018</span>; <span class="vol">46</span>(<span class="citedIssue">D1</span>): <span class="pageFirst">D387</span><span class="pageLast">D392</span>. <a href="https://doi.org/10.1093/nar/gkx950" class="linkBehavior">https://doi.org/10.1093/nar/gkx950</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/nar/gkx950</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisb3I&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkx950&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=29040693&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkx950&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000419550700059&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkx950&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D46%26publication_year%3D2018%26pages%3DD387-D392%26journal%3DNucleic%2BAcids%2BRes%26issue%3DD1%26author%3DM%2BVaradi%26author%3DG%2BDe%2BBaets%26author%3DWF%2BVranken%26author%3DP%2BTompa%26author%3DR%2BPancsa%26title%3DAmyPro%253A%2Ba%2Bdatabase%2Bof%2Bproteins%2Bwith%2Bvalidated%2Bamyloidogenic%2Bregions&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fnar%2Fgkx950&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_38_1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx950&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fnar%2Fgkx950&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fnar%252Fgkx950%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DM%26rft.aulast%3DVaradi%26rft.aufirst%3DG%26rft.aulast%3DDe%2520Baets%26rft.aufirst%3DWF%26rft.aulast%3DVranken%26rft.aufirst%3DP%26rft.aulast%3DTompa%26rft.aufirst%3DR%26rft.aulast%3DPancsa%26rft.atitle%3DAmyPro%253A%2520a%2520database%2520of%2520proteins%2520with%2520validated%2520amyloidogenic%2520regions%26rft.jtitle%3DNucleic%2520Acids%2520Res%26rft.date%3D2018%26rft.volume%3D46%26rft.issue%3DD1%26rft.spage%3DD387%26rft.epage%3DD392" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0039"><span class="author">Wang X</span>, <span class="author">Horii A</span>, <span class="author">Zhang S</span>. <span class="articleTitle">Designer functionalized self-assembling peptide nanofiber scaffolds for growth, migration, and tubulogenesis of human umbilical vein endothelial cells</span>. <i>Soft Matter</i>. <span class="pubYear">2008</span>; <span class="vol">4</span>(<span class="citedIssue">12</span>): <span class="pageFirst">2388</span>. <a href="https://doi.org/10.1039/b807155a" class="linkBehavior">https://doi.org/10.1039/b807155a</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1039/b807155a</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGrtrvL&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2Fb807155a&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000261726400012&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1039%2Fb807155a&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D4%26publication_year%3D2008%26pages%3D2388%26journal%3DSoft%2BMatter%26issue%3D12%26author%3DX%2BWang%26author%3DA%2BHorii%26author%3DS%2BZhang%26title%3DDesigner%2Bfunctionalized%2Bself%25E2%2580%2590assembling%2Bpeptide%2Bnanofiber%2Bscaffolds%2Bfor%2Bgrowth%252C%2Bmigration%252C%2Band%2Btubulogenesis%2Bof%2Bhuman%2Bumbilical%2Bvein%2Bendothelial%2Bcells&doi=10.1002%2Fpro.70031&doiOfLink=10.1039%2Fb807155a&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_39_1&amp;dbid=16384&amp;doi=10.1039%2Fb807155a&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1039%2Fb807155a&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1039%252Fb807155a%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DX%26rft.aulast%3DWang%26rft.aufirst%3DA%26rft.aulast%3DHorii%26rft.aufirst%3DS%26rft.aulast%3DZhang%26rft.atitle%3DDesigner%2520functionalized%2520self%25E2%2580%2590assembling%2520peptide%2520nanofiber%2520scaffolds%2520for%2520growth%252C%2520migration%252C%2520and%2520tubulogenesis%2520of%2520human%2520umbilical%2520vein%2520endothelial%2520cells%26rft.jtitle%3DSoft%2520Matter%26rft.date%3D2008%26rft.volume%3D4%26rft.issue%3D12%26rft.spage%3D2388" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0040"><span class="author">Wilson LM</span>, <span class="author">Mok Y-F</span>, <span class="author">Binger KJ</span>, <span class="author">Griffin MDW</span>, <span class="author">Mertens HDT</span>, <span class="author">Lin F</span>, et al. <span class="articleTitle">A structural Core within apolipoprotein C-II amyloid fibrils identified using hydrogen exchange and proteolysis</span>. <i>J Mol Biol</i>. <span class="pubYear">2007</span>; <span class="vol">366</span>(<span class="citedIssue">5</span>): <span class="pageFirst">1639</span><span class="pageLast">1651</span>. <a href="https://doi.org/10.1016/j.jmb.2006.12.040" class="linkBehavior">https://doi.org/10.1016/j.jmb.2006.12.040</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1016/j.jmb.2006.12.040</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Clsb8%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2006.12.040&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=17217959&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2006.12.040&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000244621100023&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1016%2Fj.jmb.2006.12.040&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D366%26publication_year%3D2007%26pages%3D1639-1651%26journal%3DJ%2BMol%2BBiol%26issue%3D5%26author%3DLM%2BWilson%26author%3DY%25E2%2580%2590F%2BMok%26author%3DKJ%2BBinger%26author%3DMDW%2BGriffin%26author%3DHDT%2BMertens%26author%3DF%2BLin%26title%3DA%2Bstructural%2BCore%2Bwithin%2Bapolipoprotein%2BC%25E2%2580%2590II%2Bamyloid%2Bfibrils%2Bidentified%2Busing%2Bhydrogen%2Bexchange%2Band%2Bproteolysis&doi=10.1002%2Fpro.70031&doiOfLink=10.1016%2Fj.jmb.2006.12.040&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_40_1&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.12.040&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1016%2Fj.jmb.2006.12.040&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1016%252Fj.jmb.2006.12.040%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DLM%26rft.aulast%3DWilson%26rft.aufirst%3DY%25E2%2580%2590F%26rft.aulast%3DMok%26rft.aufirst%3DKJ%26rft.aulast%3DBinger%26rft.aufirst%3DMDW%26rft.aulast%3DGriffin%26rft.aufirst%3DHDT%26rft.aulast%3DMertens%26rft.aufirst%3DF%26rft.aulast%3DLin%26rft.atitle%3DA%2520structural%2520Core%2520within%2520apolipoprotein%2520C%25E2%2580%2590II%2520amyloid%2520fibrils%2520identified%2520using%2520hydrogen%2520exchange%2520and%2520proteolysis%26rft.jtitle%3DJ%2520Mol%2520Biol%26rft.date%3D2007%26rft.volume%3D366%26rft.issue%3D5%26rft.spage%3D1639%26rft.epage%3D1651" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0041"><span class="author">Zambrano R</span>, <span class="author">Jamroz M</span>, <span class="author">Szczasiuk A</span>, <span class="author">Pujols J</span>, <span class="author">Kmiecik S</span>, <span class="author">Ventura S</span>. <span class="articleTitle">AGGRESCAN3D (A3D): server for prediction of aggregation properties of protein structures</span>. <i>Nucleic Acids Res</i>. <span class="pubYear">2015</span>; <span class="vol">43</span>(<span class="citedIssue">W1</span>): <span class="pageFirst">W306</span><span class="pageLast">W313</span>. <a href="https://doi.org/10.1093/nar/gkv359" class="linkBehavior">https://doi.org/10.1093/nar/gkv359</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1093/nar/gkv359</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbjK&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkv359&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=25883144&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkv359&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000359772700048&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1093%2Fnar%2Fgkv359&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D43%26publication_year%3D2015%26pages%3DW306-W313%26journal%3DNucleic%2BAcids%2BRes%26issue%3DW1%26author%3DR%2BZambrano%26author%3DM%2BJamroz%26author%3DA%2BSzczasiuk%26author%3DJ%2BPujols%26author%3DS%2BKmiecik%26author%3DS%2BVentura%26title%3DAGGRESCAN3D%2B%2528A3D%2529%253A%2Bserver%2Bfor%2Bprediction%2Bof%2Baggregation%2Bproperties%2Bof%2Bprotein%2Bstructures&doi=10.1002%2Fpro.70031&doiOfLink=10.1093%2Fnar%2Fgkv359&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_41_1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv359&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1093%2Fnar%2Fgkv359&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1093%252Fnar%252Fgkv359%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DR%26rft.aulast%3DZambrano%26rft.aufirst%3DM%26rft.aulast%3DJamroz%26rft.aufirst%3DA%26rft.aulast%3DSzczasiuk%26rft.aufirst%3DJ%26rft.aulast%3DPujols%26rft.aufirst%3DS%26rft.aulast%3DKmiecik%26rft.aufirst%3DS%26rft.aulast%3DVentura%26rft.atitle%3DAGGRESCAN3D%2520%2528A3D%2529%253A%2520server%2520for%2520prediction%2520of%2520aggregation%2520properties%2520of%2520protein%2520structures%26rft.jtitle%3DNucleic%2520Acids%2520Res%26rft.date%3D2015%26rft.volume%3D43%26rft.issue%3DW1%26rft.spage%3DW306%26rft.epage%3DW313" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  
                  <li data-bib-id="pro70031-bib-0042"><span class="author">Zemla A</span>, <span class="author">Venclovas C</span>, <span class="author">Fidelis K</span>, <span class="author">Rost B</span>. <span class="articleTitle">A modified definition of Sov, a segment-based measure for protein secondary structure prediction assessment</span>. <i>Proteins: Struct, Funct, Bioinf</i>. <span class="pubYear">1999</span>; <span class="vol">34</span>(<span class="citedIssue">2</span>): <span class="pageFirst">220</span><span class="pageLast">223</span>. <a href="https://doi.org/10.1002/(SICI)1097-0134(19990201)34:2%3C220::AID-PROT7%3E3.0.CO;2-K" class="linkBehavior">https://doi.org/10.1002/(SICI)1097-0134(19990201)34:2&lt;220::AID-PROT7&gt;3.0.CO;2-K</a>
                     <div class="extra-links getFTR"><span class="hidden data-doi">10.1002/(SICI)1097-0134(19990201)34:2<220::AID-PROT7>3.0.CO;2-K</span><div class="getFTR__content getFTR__placeholder"></div>
                        <span><a href="/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fpro.70031&amp;key=1%3ACAS%3A528%3ADyaK1MXlsFChsA%253D%253D&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1002%2F%28SICI%291097-0134%2819990201%2934%3A2%3C220%3A%3AAID-PROT7%3E3.0.CO%3B2-K&amp;linkType=COI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="CAS, opens in a new tab">CAS <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fpro.70031&amp;key=10022357&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1002%2F%28SICI%291097-0134%2819990201%2934%3A2%3C220%3A%3AAID-PROT7%3E3.0.CO%3B2-K&amp;linkType=PMID&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="PubMed, opens in a new tab">PubMed <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a href="/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fpro.70031&amp;key=WOS%3A000077832400007&amp;getFTLinkType=true&amp;doiForPubOfPage=10.1002%2Fpro.70031&amp;refDoi=10.1002%2F%28SICI%291097-0134%2819990201%2934%3A2%3C220%3A%3AAID-PROT7%3E3.0.CO%3B2-K&amp;linkType=ISI&amp;linkSource=FULL_TEXT&amp;linkLocation=Reference" class="visitable" target="_blank" aria-label="Web of Science, opens in a new tab">Web of Science <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span><a class="google-scholar visitable" href="/action/getFTRLinkout?url=http%3A%2F%2Fscholar.google.com%2Fscholar_lookup%3Fhl%3Den%26volume%3D34%26publication_year%3D1999%26pages%3D220-223%26journal%3DProteins%253A%2BStruct%252C%2BFunct%252C%2BBioinf%26issue%3D2%26author%3DA%2BZemla%26author%3DC%2BVenclovas%26author%3DK%2BFidelis%26author%3DB%2BRost%26title%3DA%2Bmodified%2Bdefinition%2Bof%2BSov%252C%2Ba%2Bsegment%25E2%2580%2590based%2Bmeasure%2Bfor%2Bprotein%2Bsecondary%2Bstructure%2Bprediction%2Bassessment&doi=10.1002%2Fpro.70031&doiOfLink=10.1002%2F%28SICI%291097-0134%2819990201%2934%3A2%3C220%3A%3AAID-PROT7%3E3.0.CO%3B2-K&linkType=gs&linkLocation=Reference&linkSource=FULL_TEXT"aria-label="Google Scholar, opens in a new tab" target="_blank">Google Scholar<i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span><span class="openUrl"><a href="/servlet/linkout?suffix=e_1_2_8_42_1&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819990201%2934%3A2%3C220%3A%3AAID-PROT7%3E3.0.CO%3B2-K&getFTLinkType=true&doiForPubOfPage=10.1002%2Fpro.70031&refDoi=10.1002%2F%28SICI%291097-0134%2819990201%2934%3A2%3C220%3A%3AAID-PROT7%3E3.0.CO%3B2-K&linkType=openurl&linkSource=FULL_TEXT&linkLocation=Reference&amp;FULL_TEXT&amp;type=refOpenUrl&amp;url=http%3A%2F%2Fwww.lib.byu.edu%2Fjfinder.pl%3Frft_id%3Dinfo%3Adoi%2F10.1002%252F%2528SICI%25291097-0134%252819990201%252934%253A2%253C220%253A%253AAID-PROT7%253E3.0.CO%253B2-K%26url_ver%3DZ39.88-2004%26rft.genre%3Darticle%26rfr_id%3Dinfo%3Asid%2Fwiley%26rft.aufirst%3DA%26rft.aulast%3DZemla%26rft.aufirst%3DC%26rft.aulast%3DVenclovas%26rft.aufirst%3DK%26rft.aulast%3DFidelis%26rft.aufirst%3DB%26rft.aulast%3DRost%26rft.atitle%3DA%2520modified%2520definition%2520of%2520Sov%252C%2520a%2520segment%25E2%2580%2590based%2520measure%2520for%2520protein%2520secondary%2520structure%2520prediction%2520assessment%26rft.jtitle%3DProteins%253A%2520Struct%252C%2520Funct%252C%2520Bioinf%26rft.date%3D1999%26rft.volume%3D34%26rft.issue%3D2%26rft.spage%3D220%26rft.epage%3D223" title="OpenURL Brigham Young University" class="sfxLink visitable" target="_blank" aria-label="Find @ Brigham Young University, opens in a new tab">Find @ Brigham Young University <i aria-hidden="true" class="icon icon-sponsored-external-link ml-1"></i></a></span></div></li>
                  </ul>
            </div>
         </div>
      </section>
   </section>
   <div class="noteList-container">
      <ul class="custom-style-list noteList"></ul>
   </div>
   <div class="pb-dropzone" data-pb-dropzone="citedBy-dropzone"><section id="cited-by" class="article-section article-section__citedBy cited-by"><div class="accordion article-accordion"><h2><div tabindex="0" role="button" aria-controls="cited-by__content" aria-expanded="false" data-getFTR-citedby-enabled="data-getFTR-citedby-enabled" data-getFTR-citedby-url="/action/CitedByEntitlement" data-getFTR-links-count='10' data-getFTR-request-source="FULL_TEXT" class="accordion__control"><span id="citedby-section" class="section__title">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div></h2><div id="cited-by__content" data-source="/pb/widgets/citedBy?page=0&amp;widgetId=f6d59956-1f70-4e88-8811-a42ff3d9d1fa&amp;pbContext=%3Bjournal%3Ajournal%3A1469896x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fpro.70031%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fpro.v34.2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3A1469896x" class="accordion__content cited-by__content"><div class="sciteMetricsDropzone"></div></div></div></section></div>
   </article></div>



        
        <!-- TrendMD Placeholder -->
<div id="trendmd-suggestions"></div>
</div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="/toc/1469896x/2025/34/2" title="View Volume 34, Issue 2"><img src="/cms/asset/cc1752c7-12c8-4a22-a94b-2100c050a46a/pro.v34.2.cover.jpg" alt=""/></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="/toc/1469896x/2025/34/2" title="View Volume 34, Issue 2"><span>Volume</span><span class="val">34</span>, <span>Issue</span><span class="val">2</span></a></p><p>February 2025</p><p>e70031</p>



        
        <div class="publication-cover-extra"><label>This article also appears in:</label><ul class="rlist"><li><a href="/doi/toc/10.1002/(ISSN)1469-896X.tools-protein-science2025">Tools for Protein Science 2025</a></li></ul></div>
</div>
</div>



        
        



    
        <div class="pb-las">
            <div id="advert-rail-1" class="advert advert-rail">
	<script nonce="9a020aaf9539530c-SJC">
		googletag.cmd.push(function () {
			onDvtagReady(function () {
				googletag.display( 'advert-rail-1' );
			});
		});
	</script>

	<!-- No JavaScript support is a call to an image -->
	<noscript>
		<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1002%2Fpro.70031&pubId=88187701&placeholderId=1014&productId=101018" target="_blank">
			<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
		</a>
	</noscript>
</div>
            
        </div>
    

    

    


<div class="tabs__wrapper"><div data-default-tab="related" class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko article__tab  tab--flex  tabs--xs "><ul data-mobile-toggle="slide" role="tablist" aria-label="Article Information" class="rlist tab__nav w-slide--list"><li role="presentation" class="tab__nav__item"><a id="pane-pcw-figurescon" href="#pane-pcw-figures" data-slide-clone="self" aria-controls="pane-pcw-figures" role="tab" data-toggle="tab" data-slide-target="#pane-pcw-figures" class="tab__nav__item__link article-row-right__tabLinks w-slide__btn figures-tab"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation" class="tab__nav__item"><a id="pane-pcw-referencescon" href="#pane-pcw-references" data-slide-clone="self" aria-controls="pane-pcw-references" role="tab" data-toggle="tab" data-slide-target="#pane-pcw-references" class="tab__nav__item__link article-row-right__tabLinks w-slide__btn references-tab"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation" class="tab__nav__item"><a id="pane-pcw-relatedcon" href="#pane-pcw-related" data-slide-clone="self" aria-controls="pane-pcw-related" role="tab" data-toggle="tab" data-slide-target="#pane-pcw-related" class="tab__nav__item__link article-row-right__tabLinks w-slide__btn related-tab"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation" class="tab__nav__item"><a id="pane-pcw-detailscon" href="#pane-pcw-details" data-slide-clone="self" aria-controls="pane-pcw-details" role="tab" data-toggle="tab" data-slide-target="#pane-pcw-details" class="tab__nav__item__link article-row-right__tabLinks w-slide__btn details-tab"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul><div role="presentation" class="tab__content"><div id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tabpanel" data-empty-text="" class="tab__pane article-row-right__panes empty"></div><div id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tabpanel" data-empty-text="" class="tab__pane article-row-right__panes empty"></div><div id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tabpanel" data-empty-text="" class="tab__pane article-row-right__panes"></div><div id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tabpanel" data-empty-text="" class="tab__pane article-row-right__panes empty"><div class="contentTab"></div></div><div class="tab__spinner"><div class="tab__spinner--box"><img src="/specs/products/acropolis/pericles/releasedAssets/images/spinner-1ffd60b3aabe5b09bc98c48345208fd9.gif" id="spinner" alt=""/></div></div></div></div></div><div class="articleAdvert fixed-element">



        
        



    
        <div class="pb-las">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script nonce="9a020aaf9539530c-SJC">
			googletag.cmd.push( function () {
				onDvtagReady(function () {
					googletag.display( 'advert-rail-2' );
				});
			});

		</script>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1002%2Fpro.70031&pubId=88187701&placeholderId=1013&productId=101018" target="_blank">
				<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	</div>
</aside>
            
        </div>
    

    

    


</div></div></div></article><script nonce="9a020aaf9539530c-SJC">var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" aria-label="Figure Viewer" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a href="#" data-ux3-role="controller" role="button" title="close" aria-label="close" class="figure-viewer__ctrl__close figure-viewer__cycleElement"><span class="icon-close_thin"><span class="sr-only">Close Figure Viewer</span></span></a></div><div class="figure-viewer__top__left"><button title="zoom out" aria-label="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out figure-viewer__cycleElement"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input type="range" title="zoom control" aria-label="zoom control" class="zoom-range figure-viewer__cycleElement"/></span><button title="zoom in" aria-label="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in figure-viewer__cycleElement"></button></div></div><div class="direction-control hidden"><span direction="up" tabindex="0" role="button" aria-label="move figure up" class="move-up figure-viewer__cycleElement"></span><span direction="right" tabindex="0" role="button" aria-label="move figure right" class="move-right figure-viewer__cycleElement"></span><span direction="down" tabindex="0" role="button" aria-label="move figure down" class="move-down figure-viewer__cycleElement"></span><span direction="left" tabindex="0" role="button" aria-label="move figure left" class="move-left figure-viewer__cycleElement"></span></div><div class="figure-viewer__reg__center"><div class="figure-viewer__cent__left"><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><a href="#" role="button" title="previous figure" aria-label="previous" class="figure-viewer__ctrl__prev figure-viewer__cycleElement"><span class="icon-arrow_l"><span class="sr-only">Previous Figure</span></span></a><a href="#" role="button" title="next figure" aria-label="next" class="figure-viewer__ctrl__next figure-viewer__cycleElement"><span class="icon-arrow_r"><span class="sr-only">Next Figure</span></span></a><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" aria-expanded="false" aria-label="Caption" class="figure-viewer__ctrl__caption figure-viewer__cycleElement"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text hidden"></span></div><div tabindex="0" role="complementary" class="figure-viewer__hold__figcap figure-viewer__cycleElement hidden"></div></div></div></div>




        
        <div style="text-align: right!important;"><a style="font-weight: 300!important; font-size: 90%;" href="/doi/pdf/10.1002/pro.70031">Download PDF</a></div>
</div>
</div></div></div>

        </section>
    


        
    
</div>

    </div>
    <!--end pagefulltext-->
</main>

        </div>
    


                    
                        









    
    
        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        


<footer class="page-footer">
    











<div  data-pb-dropzone="main" class="">
    
        
            



        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>










    
    
        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h2 class="sr-only">
                Additional links
            </h2>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>About Wiley Online Library</h3>
    <ul class="rlist">
        <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
        <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
        <li><a href="https://onlinelibrary.wiley.com/cookies">About Cookies</a></li>
        <li><button class="mange-cookies-btn" onclick="Osano.cm.showDrawer('osano-cm-dom-info-dialog-open'); return false;">Manage Cookies</button></li>
        <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
        <li><a href="https://onlinelibrary.wiley.com/publishing-policies">Wiley Research DE&amp;I Statement and Publishing Policies</a></li>
    </ul>
</div>
<style>
    .mange-cookies-btn{
        background: none;
        color: inherit;
        border: none;
        padding: 0;
        cursor: pointer;
        font-weight: 600;
    }
    .mange-cookies-btn:hover,
    .mange-cookies-btn:focus,
    .mange-cookies-btn:active {
        text-decoration: none;
        outline: 0;
        background-color: transparent;
    }

    .mange-cookies-btn:focus {
        outline: 5px auto -webkit-focus-ring-color;
    }

    .osano-cm-widget {
        display: none;
    }
</style>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help & Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
      <li><a href="https://www.wiley.com/customer-success/wiley-online-library-training-hub">Training and Support</a></li>
      <li><a href="https://onlinelibrary.wiley.com/dmca-notification-policy">DMCA &amp; Reporting Piracy</a></li>
      <li><a href="https://onlinelibrary.wiley.com/sitemap">Sitemap</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers & Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://www.wiley.com/network">The Wiley Network</a></li>
      <li><a href="https://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    











    
    
        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









    
    
        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        
<div class="DST-copyright">
        <p>Copyright &copy; 1999-2025 <a href="https://www.wiley.com">John Wiley &amp; Sons, Inc</a> or related companies. All rights reserved, including rights for text and data mining and training of artificial intelligence technologies or similar technologies.<p>


</div>

<style type="text/css">
        
        .DST-copyright {
                max-width: 340px;
                float: right;
        }

        @media (max-width: 768px) {
                .DST-copyright {
                        max-width: none;
                        float: none;
                        margin-bottom: 20px;
                }
        }

}

</style>

        </div>
    





        
        <div class="footer-logo pull-left">
        
<a href="https://www.wiley.com" title="Wiley Home Page" aria-label="Wiley"><img alt="Wiley Home Page" src="/pb-assets/tmp-images/footer-logo-wiley-1510029248417.png"></a>

</div>
</div></div></div>

        </div>
    

</div>

        
    
</div>

</footer>

        </div>
    


                    
                        



        
        


















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="loginPopupHead">



    <div class="content">
        <a href="#" role="button" aria-label="Cancel Login" class="close">
            <i class="icon-tools_close" aria-hidden="true"></i>
        </a>
        <div class="modal__header">
            

<div class="header_logo">
    <img src="/pb-assets/hub-assets/pericles/logo-header-1690978619437.png" alt="">
</div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2 id="loginPopupHead">Log in to Wiley Online Library</h2>

            
































<div class="login-form">
    

    
        
        
    
    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input type="hidden" name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" />
        
            <input type="hidden" name="popup" value="true" />
        
        

        
        <input type="hidden" name="loginUri" value="/doi/10.1002/pro.70031" />
        <input type="hidden" name="remoteLoginUri" value="" />

        
            





<input type="hidden" name="redirectUri" value="/doi/10.1002/pro.70031" />

            
        
        
        
            
            
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                
                
                    <label for="username">Email or Customer ID</label>
                
            
        </div>
        
        
            
            
                <input id="username" class="login" type="text" name="login" value="" size="15" placeholder="Enter your email"autocorrect="off"spellcheck="false"autocapitalize="off"required="true" />
            
        
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                
























    
    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        
        
            
            
                <input id="password" class="password" type="password" name="password" value="" autocomplete="off" placeholder="Enter your password"autocorrect="off"spellcheck="false"autocapitalize="off"required="true" />
            
        
        <span class="password-eye-icon icon-eye hidden" role="button" tabindex="0" aria-label="Password visibility" aria-pressed="false"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
            
            
                <div class="align-end">
                    
                    
                        
                            
                            <span class="submit ">
                                <input class="button btn submit primary no-margin-bottom accessSubmit" type="submit" name="submitButton" value="Log In" />
                            </span>
                        
                        
                    
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="/action/registration?acdl-redirect=true" class="linkBtn float-left" acdl-redirect="true" data-acdl-registration--step="registration initiated" data-acdl-event="registration-initiated" data-acdl-registration--cta-region="login modal">
                        
                        NEW USER <span aria-hidden="true">></span>
                    </a>
                    
                        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1002%2Fpro.70031" class="linkBtn float-right instituation-login">
                            
                            INSTITUTIONAL LOGIN <span aria-hidden="true">></span>
                        </a>
                    
                </div>
            

        </div>
    </div>
</div>

                    
                        



        
        






<div class="popup change-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="changePasswordPopupHead">
    <div class="content">
        <a href="#" class="close cancel" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
        <div class="modal__header">
            

<div class="header_logo">
    <img src="/pb-assets/hub-assets/pericles/logo-header-1690978619437.png" alt="">
</div>

            <h2 id="changePasswordPopupHead">Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input type="hidden" name="submit" value="submit" />
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" type="password" name="old" value="" autocomplete="off" />
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" type="password" name="new" value="" autocomplete="off" />
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            















    
        
        
            
        
    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="4">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
<div id="pswd_info" class="pass-strength-popup js__pswd_info" style="display: none;">
    <h4 id="length">
        
            
                Your password must have 10 characters or more:
            
            
        
    </h4>
    <ul>
        <li id="letter" class="invalid">
            <span>a lower case character,&nbsp;</span>
        </li>
        <li id="capital" class="invalid">
            <span>an upper case character,&nbsp;</span>
        </li>
        <li id="special" class="invalid">
            <span>a special character&nbsp;</span>
        </li>
        <li id="number" class="invalid">
            <span>or a digit</span>
        </li>
    </ul>
    <span class="strength">Too Short</span>
</div>
                        
                    </div>
                    <input class="button primary submit" type="submit" value="Submit" disabled="disabled" /></form>
                <a href="#" class="fakeFormLastElement"></a>
            </div>
            <div class="success-template hidden">
                <h2>Password Changed Successfully</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>


                    
                        



        
        





<div class="popup registration-popup hidden"  role="dialog" tabindex="-1" aria-labelledby="createAccountPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2 id="createAccountPopupHead">Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1002/pro.70031" />
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" type="text" name="email" value="" size="15" />
                </div>
                <div class="submit">
                    <input class="button submit primary" type="submit" value="Register" />
                </div></form>
            <div class="center">
                <a href="/action/showLogin" class="show-login">
                    
                    Returning user
                </a>
            </div>
        </div>
    </div>
</div>

                    
                        



        
        



<div class="popup top-drawer request-reset-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="resetPasswordPopupHead">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>

            

<div class="header_logo">
    <img src="/pb-assets/hub-assets/pericles/logo-header-1690978619437.png" alt="">
</div>

            <h2 id="resetPasswordPopupHead">Forgot your password?</h2>
            <p class="sub">Enter your email address below. </p>

        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input type="hidden" name="requestResetPassword" value="true" />
    
        
        
            <div class="input-group">
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                
                
                    <label for="email">Email</label>
                
            
        </div>
        
        
            
            
                <input id="email" class="email" type="text" name="email" value="" size="15" placeholder="Enter your email"autocorrect="off"spellcheck="false"autocapitalize="off" />
            
        
        
    </div>

            </div>
        
    


    
        
            <div class="password-recaptcha-ajax"></div>
        
        
    

    <div class="message error"></div>

    <div class="form-btn">
        <input class="button btn primary submit" type="submit" name="submit" value="RESET PASSWORD" disabled="disabled" />
    </div></form>
            

            <div class="success-template hidden">

                <p>Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>

            </div>
        </div>
    </div>
</div>


                    
                        



        
        




<div class="popup top-drawer request-username-drawer username-popup hidden"  role="dialog" tabindex="-1" aria-labelledby="requestUsernamePopupHead">
    <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2 id="requestUsernamePopupHead">Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input type="hidden" name="requestUsername" value="requestUsername" />
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" type="text" name="email" value="" size="15" />
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" type="submit" name="submit" value="Submit" disabled="disabled" />
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>

                    
                        



        
        <script type="text/javascript" nonce="9a020aaf9539530c-SJC">
    window.addEventListener('load', () => {
        document.querySelectorAll('.admin-bar-popup-menu ul li')?.forEach(function (li) {
            const anchor = li.querySelector('a');
            if (anchor && anchor.textContent.trim() === 'Edit Access Denial') {
                li.style.display = 'none';
            }
        });
    });
</script>

                    
                        



        
        <!--  -->  <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 

                    
                        



        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>

                    
                
            </div>
        </div>
        <script src="/products/acropolis/pericles/releasedAssets/js/build.lazyload.bundle-e1dc5868ad4d2bbdc547.js" nonce="9a020aaf9539530c-SJC"></script><script src="/products/acropolis/pericles/releasedAssets/js/main.bundle-37c1ea93346f6a82126a.js" nonce="9a020aaf9539530c-SJC"></script><script type="text/javascript" src="/wro/62b81afc18538b179f72ac9fd0c18fbc83a2b493~product.js" nonce="9a020aaf9539530c-SJC"></script>

<script nonce="9a020aaf9539530c-SJC">var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>
















    <script type="text/javascript" nonce="9a020aaf9539530c-SJC">
        $(document).ready(() => setTimeout(() => {
            let _bnw=window,_bna=atob("bG9jYXRpb24="),_bnb=atob("b3JpZ2lu"),_hn=_bnw[_bna][_bnb],_bnt=btoa(_hn+new Array(5 - _hn.length % 4).join(" "));
            $.get("/resource/lodash?t="+_bnt);
        },4000));
    </script>












    

    
    

    
    
    
    
        
            
            
            
                
                    <script type="text/javascript" src="/wro/62b81afc18538b179f72ac9fd0c18fbc83a2b493~article-metrics-phase2.js" nonce="9a020aaf9539530c-SJC"></script>
                
            
        
    


<!-- iframe.jsp -->

<iframe class="connect-teaser" src="https://wiley.scienceconnect.io/teaser.html" frameborder="0" allowfullscreen></iframe>




    <script nonce="9a020aaf9539530c-SJC">
        fetch("https://wiley.met.vgwort.de/na/pw-base64-dmd6bS42MDY1MC0xMC4xMDAyLXByby43MDAzMQ==", {mode: 'no-cors', referrerPolicy: 'unsafe-url'});
    </script>
<script type="text/javascript" src="/wro/62b81afc18538b179f72ac9fd0c18fbc83a2b493~full-text-analytics.js" nonce="9a020aaf9539530c-SJC"></script>
<script type="application/javascript" nonce="9a020aaf9539530c-SJC">
        document.querySelectorAll('.pericles-input-class').forEach(function (element) {
            element.addEventListener('change', function () {
                element.removeAttribute('aria-invalid')
            });
        });
    </script>

    


            <script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'9a020aaf5fcee888',t:'MTc2MzQxMjAxMS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><script defer src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon='{"rayId":"9a020aaf5fcee888","serverTiming":{"name":{"cfExtPri":true,"cfEdge":true,"cfOrigin":true,"cfL4":true,"cfSpeedBrain":true,"cfCacheStatus":true}},"version":"2025.9.1","token":"f3dd54df2ac243bab2129483f57546d1"}' crossorigin="anonymous"></script>
</body>
        
        
    

</html>